[debug] [2025-09-20T21:16:59.030Z] [Firebase Plugin] Activating Firebase extension.
[debug] [2025-09-20T21:17:11.466Z] [Firebase Plugin] requireAuthWrapper
[debug] [2025-09-20T21:17:11.469Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform","https://www.googleapis.com/auth/userinfo.email"]
[debug] [2025-09-20T21:17:11.499Z] [Firebase Plugin] VSCode notification server listening on port 40001
[info] [Firebase Plugin] Starting Data Connect toolkit (version 2.13.0) on port 50001
[info] i  dataconnect: downloading dataconnect-emulator-2.13.0... {"metadata":{"emulator":{"name":"dataconnect"},"message":"downloading dataconnect-emulator-2.13.0..."}}
[debug] [2025-09-20T21:17:13.932Z] >>> [apiv2][query] GET https://storage.googleapis.com/firemat-preview-drop/emulator/dataconnect-emulator-linux-v2.13.0 
[debug] [2025-09-20T21:17:14.358Z] <<< [apiv2][status] GET https://storage.googleapis.com/firemat-preview-drop/emulator/dataconnect-emulator-linux-v2.13.0 200
[debug] [2025-09-20T21:17:14.366Z] <<< [apiv2][body] GET https://storage.googleapis.com/firemat-preview-drop/emulator/dataconnect-emulator-linux-v2.13.0 [stream]
[debug] [2025-09-20T21:17:16.435Z] 'fdc build' failed with error: Unable to build your Data Connect schema and connectors (exit code 1): E0920 21:17:16.432069   14987 cmd.go:43] Error loading data connect config: could not find dataconnect.yaml in `/home/user/studio/dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"'fdc build' failed with error: Unable to build your Data Connect schema and connectors (exit code 1): E0920 21:17:16.432069   14987 cmd.go:43] Error loading data connect config: could not find dataconnect.yaml in `/home/user/studio/dataconnect`\n"}}
[debug] [2025-09-20T21:17:16.439Z] Ignoring unsupported arg: auto_download {"metadata":{"emulator":{"name":"dataconnect"},"message":"Ignoring unsupported arg: auto_download"}}
[debug] [2025-09-20T21:17:16.439Z] Ignoring unsupported arg: port {"metadata":{"emulator":{"name":"dataconnect"},"message":"Ignoring unsupported arg: port"}}
[debug] [2025-09-20T21:17:16.440Z] Starting Data Connect Emulator with command {"binary":"/home/user/.cache/firebase/emulators/dataconnect-emulator-2.13.0","args":["--logtostderr","-v=2","dev","--listen=localhost:50001","--config_dir=/home/user/studio/dataconnect","--enable_output_schema_extensions=true","--enable_output_generated_sdk=true"],"optionalArgs":["listen","config_dir","enable_output_schema_extensions","enable_output_generated_sdk"],"joinArgs":true,"shell":false,"port":9399} {"metadata":{"emulator":{"name":"dataconnect"},"message":"Starting Data Connect Emulator with command {\"binary\":\"/home/user/.cache/firebase/emulators/dataconnect-emulator-2.13.0\",\"args\":[\"--logtostderr\",\"-v=2\",\"dev\",\"--listen=localhost:50001\",\"--config_dir=/home/user/studio/dataconnect\",\"--enable_output_schema_extensions=true\",\"--enable_output_generated_sdk=true\"],\"optionalArgs\":[\"listen\",\"config_dir\",\"enable_output_schema_extensions\",\"enable_output_generated_sdk\"],\"joinArgs\":true,\"shell\":false,\"port\":9399}"}}
[info] i  dataconnect: Data Connect Emulator logging to dataconnect-debug.log {"metadata":{"emulator":{"name":"dataconnect"},"message":"Data Connect Emulator logging to dataconnect-debug.log"}}
[debug] [2025-09-20T21:17:16.501Z] E0920 21:17:16.500983   14992 dev.go:79] Failed to load initial config from "/home/user/studio/dataconnect": could not find dataconnect.yaml in `/home/user/studio/dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0920 21:17:16.500983   14992 dev.go:79] Failed to load initial config from \"/home/user/studio/dataconnect\": could not find dataconnect.yaml in `/home/user/studio/dataconnect`\n"}}
[debug] [2025-09-20T21:17:16.509Z] E0920 21:17:16.502233   14992 watch.go:102] watchAllDir unable to walk dir `/home/user/studio/dataconnect`: lstat /home/user/studio/dataconnect: no such file or directory
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0920 21:17:16.502233   14992 watch.go:102] watchAllDir unable to walk dir `/home/user/studio/dataconnect`: lstat /home/user/studio/dataconnect: no such file or directory\n"}}
[debug] [2025-09-20T21:17:16.529Z] I0920 21:17:16.527979   14992 dev.go:102] Listening on address (HTTP + gRPC): 127.0.0.1:50001
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0920 21:17:16.527979   14992 dev.go:102] Listening on address (HTTP + gRPC): 127.0.0.1:50001\n"}}
[error] [Firebase Plugin] error:  {}
[debug] [2025-09-20T21:17:26.507Z] E0920 21:17:26.506017   14992 watch.go:31] Failed to load config from "/home/user/studio/dataconnect": could not find dataconnect.yaml in `/home/user/studio/dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0920 21:17:26.506017   14992 watch.go:31] Failed to load config from \"/home/user/studio/dataconnect\": could not find dataconnect.yaml in `/home/user/studio/dataconnect`\n"}}
[debug] [2025-09-20T21:17:36.503Z] E0920 21:17:36.502706   14992 watch.go:31] Failed to load config from "/home/user/studio/dataconnect": could not find dataconnect.yaml in `/home/user/studio/dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0920 21:17:36.502706   14992 watch.go:31] Failed to load config from \"/home/user/studio/dataconnect\": could not find dataconnect.yaml in `/home/user/studio/dataconnect`\n"}}
[debug] [2025-09-20T21:17:46.503Z] E0920 21:17:46.503049   14992 watch.go:31] Failed to load config from "/home/user/studio/dataconnect": could not find dataconnect.yaml in `/home/user/studio/dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0920 21:17:46.503049   14992 watch.go:31] Failed to load config from \"/home/user/studio/dataconnect\": could not find dataconnect.yaml in `/home/user/studio/dataconnect`\n"}}
[debug] [2025-09-20T21:17:50.408Z] Running auto auth
[debug] [2025-09-20T21:17:50.460Z] No OAuth tokens found
[debug] [2025-09-20T21:17:50.466Z] [Firebase Plugin] User found:  innovativeenterprisesoman@gmail.com
[info] [Firebase Plugin] (Core:Project) New user detected, fetching projects
[debug] [2025-09-20T21:17:50.474Z] No OAuth tokens found
[debug] [2025-09-20T21:17:50.482Z] >>> [apiv2][query] GET https://firebase.googleapis.com/v1beta1/projects pageSize=1000
[debug] [2025-09-20T21:17:51.275Z] <<< [apiv2][status] GET https://firebase.googleapis.com/v1beta1/projects 200
[debug] [2025-09-20T21:17:51.279Z] <<< [apiv2][body] GET https://firebase.googleapis.com/v1beta1/projects [omitted]
[debug] [2025-09-20T21:17:56.514Z] E0920 21:17:56.514033   14992 watch.go:31] Failed to load config from "/home/user/studio/dataconnect": could not find dataconnect.yaml in `/home/user/studio/dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0920 21:17:56.514033   14992 watch.go:31] Failed to load config from \"/home/user/studio/dataconnect\": could not find dataconnect.yaml in `/home/user/studio/dataconnect`\n"}}
[debug] [2025-09-20T21:18:06.513Z] E0920 21:18:06.512028   14992 watch.go:31] Failed to load config from "/home/user/studio/dataconnect": could not find dataconnect.yaml in `/home/user/studio/dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0920 21:18:06.512028   14992 watch.go:31] Failed to load config from \"/home/user/studio/dataconnect\": could not find dataconnect.yaml in `/home/user/studio/dataconnect`\n"}}
[debug] [2025-09-20T21:18:16.504Z] E0920 21:18:16.503269   14992 watch.go:31] Failed to load config from "/home/user/studio/dataconnect": could not find dataconnect.yaml in `/home/user/studio/dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0920 21:18:16.503269   14992 watch.go:31] Failed to load config from \"/home/user/studio/dataconnect\": could not find dataconnect.yaml in `/home/user/studio/dataconnect`\n"}}
[debug] [2025-09-20T21:18:26.514Z] E0920 21:18:26.512160   14992 watch.go:31] Failed to load config from "/home/user/studio/dataconnect": could not find dataconnect.yaml in `/home/user/studio/dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0920 21:18:26.512160   14992 watch.go:31] Failed to load config from \"/home/user/studio/dataconnect\": could not find dataconnect.yaml in `/home/user/studio/dataconnect`\n"}}
[debug] [2025-09-20T21:18:36.511Z] E0920 21:18:36.511058   14992 watch.go:31] Failed to load config from "/home/user/studio/dataconnect": could not find dataconnect.yaml in `/home/user/studio/dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0920 21:18:36.511058   14992 watch.go:31] Failed to load config from \"/home/user/studio/dataconnect\": could not find dataconnect.yaml in `/home/user/studio/dataconnect`\n"}}
[debug] [2025-09-20T21:18:46.503Z] E0920 21:18:46.503430   14992 watch.go:31] Failed to load config from "/home/user/studio/dataconnect": could not find dataconnect.yaml in `/home/user/studio/dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0920 21:18:46.503430   14992 watch.go:31] Failed to load config from \"/home/user/studio/dataconnect\": could not find dataconnect.yaml in `/home/user/studio/dataconnect`\n"}}
[debug] [2025-09-20T21:18:56.510Z] E0920 21:18:56.507802   14992 watch.go:31] Failed to load config from "/home/user/studio/dataconnect": could not find dataconnect.yaml in `/home/user/studio/dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0920 21:18:56.507802   14992 watch.go:31] Failed to load config from \"/home/user/studio/dataconnect\": could not find dataconnect.yaml in `/home/user/studio/dataconnect`\n"}}
[debug] [2025-09-20T21:18:58.645Z] [Firebase Plugin] Value of process.env.MONOSPACE_ENV: true
[debug] [2025-09-20T21:18:58.646Z] [Firebase Plugin] requireAuthWrapper
[debug] [2025-09-20T21:18:58.647Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform","https://www.googleapis.com/auth/userinfo.email"]
[debug] [2025-09-20T21:18:58.648Z] > authorizing via signed-in user (innovativeenterprisesoman@gmail.com)
[debug] [2025-09-20T21:18:58.650Z] [Firebase Plugin] Value of process.env.MONOSPACE_ENV: true
[debug] [2025-09-20T21:18:58.651Z] [Firebase Plugin] requireAuthWrapper
[debug] [2025-09-20T21:18:58.652Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform","https://www.googleapis.com/auth/userinfo.email"]
[debug] [2025-09-20T21:18:58.652Z] > authorizing via signed-in user (innovativeenterprisesoman@gmail.com)
[debug] [2025-09-20T21:18:58.653Z] Emulator locator file path: /tmp/hub-innovative-enterprises.json
[debug] [2025-09-20T21:18:58.656Z] No OAuth tokens found
[debug] [2025-09-20T21:18:58.656Z] No OAuth tokens found
[debug] [2025-09-20T21:18:58.656Z] [Firebase Plugin] User found:  innovativeenterprisesoman@gmail.com
[debug] [2025-09-20T21:18:58.657Z] [Firebase Plugin] User found:  innovativeenterprisesoman@gmail.com
[info] [Firebase Plugin] (Core:Project) New user detected, fetching projects
[debug] [2025-09-20T21:18:58.658Z] No OAuth tokens found
[info] [Firebase Plugin] (Core:Project) New user detected, fetching projects
[debug] [2025-09-20T21:18:58.659Z] No OAuth tokens found
[debug] [2025-09-20T21:18:58.660Z] >>> [apiv2][query] GET https://firebase.googleapis.com/v1beta1/projects pageSize=1000
[debug] [2025-09-20T21:18:58.663Z] >>> [apiv2][query] GET https://firebase.googleapis.com/v1beta1/projects pageSize=1000
[debug] [2025-09-20T21:18:59.154Z] <<< [apiv2][status] GET https://firebase.googleapis.com/v1beta1/projects 200
[debug] [2025-09-20T21:18:59.158Z] <<< [apiv2][body] GET https://firebase.googleapis.com/v1beta1/projects [omitted]
[debug] [2025-09-20T21:18:59.164Z] <<< [apiv2][status] GET https://firebase.googleapis.com/v1beta1/projects 200
[debug] [2025-09-20T21:18:59.167Z] <<< [apiv2][body] GET https://firebase.googleapis.com/v1beta1/projects [omitted]
[debug] [2025-09-20T21:19:06.511Z] E0920 21:19:06.511182   14992 watch.go:31] Failed to load config from "/home/user/studio/dataconnect": could not find dataconnect.yaml in `/home/user/studio/dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0920 21:19:06.511182   14992 watch.go:31] Failed to load config from \"/home/user/studio/dataconnect\": could not find dataconnect.yaml in `/home/user/studio/dataconnect`\n"}}
[debug] [2025-09-20T21:19:16.506Z] E0920 21:19:16.503250   14992 watch.go:31] Failed to load config from "/home/user/studio/dataconnect": could not find dataconnect.yaml in `/home/user/studio/dataconnect`
E0920 21:19:16.503315   14992 watch.go:102] watchAllDir unable to walk dir `/home/user/studio/dataconnect`: lstat /home/user/studio/dataconnect: no such file or directory
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0920 21:19:16.503250   14992 watch.go:31] Failed to load config from \"/home/user/studio/dataconnect\": could not find dataconnect.yaml in `/home/user/studio/dataconnect`\nE0920 21:19:16.503315   14992 watch.go:102] watchAllDir unable to walk dir `/home/user/studio/dataconnect`: lstat /home/user/studio/dataconnect: no such file or directory\n"}}
[debug] [2025-09-20T21:19:26.503Z] E0920 21:19:26.503200   14992 watch.go:31] Failed to load config from "/home/user/studio/dataconnect": could not find dataconnect.yaml in `/home/user/studio/dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0920 21:19:26.503200   14992 watch.go:31] Failed to load config from \"/home/user/studio/dataconnect\": could not find dataconnect.yaml in `/home/user/studio/dataconnect`\n"}}
[debug] [2025-09-20T21:19:36.512Z] E0920 21:19:36.512135   14992 watch.go:31] Failed to load config from "/home/user/studio/dataconnect": could not find dataconnect.yaml in `/home/user/studio/dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0920 21:19:36.512135   14992 watch.go:31] Failed to load config from \"/home/user/studio/dataconnect\": could not find dataconnect.yaml in `/home/user/studio/dataconnect`\n"}}
[debug] [2025-09-20T21:19:46.504Z] E0920 21:19:46.503556   14992 watch.go:31] Failed to load config from "/home/user/studio/dataconnect": could not find dataconnect.yaml in `/home/user/studio/dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0920 21:19:46.503556   14992 watch.go:31] Failed to load config from \"/home/user/studio/dataconnect\": could not find dataconnect.yaml in `/home/user/studio/dataconnect`\n"}}
[debug] [2025-09-20T21:19:56.513Z] E0920 21:19:56.513050   14992 watch.go:31] Failed to load config from "/home/user/studio/dataconnect": could not find dataconnect.yaml in `/home/user/studio/dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0920 21:19:56.513050   14992 watch.go:31] Failed to load config from \"/home/user/studio/dataconnect\": could not find dataconnect.yaml in `/home/user/studio/dataconnect`\n"}}
[debug] [2025-09-20T21:20:06.508Z] E0920 21:20:06.508157   14992 watch.go:31] Failed to load config from "/home/user/studio/dataconnect": could not find dataconnect.yaml in `/home/user/studio/dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0920 21:20:06.508157   14992 watch.go:31] Failed to load config from \"/home/user/studio/dataconnect\": could not find dataconnect.yaml in `/home/user/studio/dataconnect`\n"}}
[debug] [2025-09-20T21:20:16.503Z] E0920 21:20:16.503504   14992 watch.go:31] Failed to load config from "/home/user/studio/dataconnect": could not find dataconnect.yaml in `/home/user/studio/dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0920 21:20:16.503504   14992 watch.go:31] Failed to load config from \"/home/user/studio/dataconnect\": could not find dataconnect.yaml in `/home/user/studio/dataconnect`\n"}}
[debug] [2025-09-20T21:20:26.507Z] E0920 21:20:26.507149   14992 watch.go:31] Failed to load config from "/home/user/studio/dataconnect": could not find dataconnect.yaml in `/home/user/studio/dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0920 21:20:26.507149   14992 watch.go:31] Failed to load config from \"/home/user/studio/dataconnect\": could not find dataconnect.yaml in `/home/user/studio/dataconnect`\n"}}
[debug] [2025-09-20T21:20:36.502Z] E0920 21:20:36.502647   14992 watch.go:31] Failed to load config from "/home/user/studio/dataconnect": could not find dataconnect.yaml in `/home/user/studio/dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0920 21:20:36.502647   14992 watch.go:31] Failed to load config from \"/home/user/studio/dataconnect\": could not find dataconnect.yaml in `/home/user/studio/dataconnect`\n"}}
[debug] [2025-09-20T21:20:46.503Z] E0920 21:20:46.503366   14992 watch.go:31] Failed to load config from "/home/user/studio/dataconnect": could not find dataconnect.yaml in `/home/user/studio/dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0920 21:20:46.503366   14992 watch.go:31] Failed to load config from \"/home/user/studio/dataconnect\": could not find dataconnect.yaml in `/home/user/studio/dataconnect`\n"}}
[debug] [2025-09-20T21:20:56.502Z] E0920 21:20:56.502648   14992 watch.go:31] Failed to load config from "/home/user/studio/dataconnect": could not find dataconnect.yaml in `/home/user/studio/dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0920 21:20:56.502648   14992 watch.go:31] Failed to load config from \"/home/user/studio/dataconnect\": could not find dataconnect.yaml in `/home/user/studio/dataconnect`\n"}}
[debug] [2025-09-20T21:21:06.503Z] E0920 21:21:06.502662   14992 watch.go:31] Failed to load config from "/home/user/studio/dataconnect": could not find dataconnect.yaml in `/home/user/studio/dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0920 21:21:06.502662   14992 watch.go:31] Failed to load config from \"/home/user/studio/dataconnect\": could not find dataconnect.yaml in `/home/user/studio/dataconnect`\n"}}
[debug] [2025-09-20T21:21:16.503Z] E0920 21:21:16.503043   14992 watch.go:31] Failed to load config from "/home/user/studio/dataconnect": could not find dataconnect.yaml in `/home/user/studio/dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0920 21:21:16.503043   14992 watch.go:31] Failed to load config from \"/home/user/studio/dataconnect\": could not find dataconnect.yaml in `/home/user/studio/dataconnect`\n"}}
[debug] [2025-09-20T21:21:16.504Z] E0920 21:21:16.503096   14992 watch.go:102] watchAllDir unable to walk dir `/home/user/studio/dataconnect`: lstat /home/user/studio/dataconnect: no such file or directory
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0920 21:21:16.503096   14992 watch.go:102] watchAllDir unable to walk dir `/home/user/studio/dataconnect`: lstat /home/user/studio/dataconnect: no such file or directory\n"}}
[debug] [2025-09-20T21:21:26.508Z] E0920 21:21:26.508419   14992 watch.go:31] Failed to load config from "/home/user/studio/dataconnect": could not find dataconnect.yaml in `/home/user/studio/dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0920 21:21:26.508419   14992 watch.go:31] Failed to load config from \"/home/user/studio/dataconnect\": could not find dataconnect.yaml in `/home/user/studio/dataconnect`\n"}}
[debug] [2025-09-20T21:21:36.503Z] E0920 21:21:36.503047   14992 watch.go:31] Failed to load config from "/home/user/studio/dataconnect": could not find dataconnect.yaml in `/home/user/studio/dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0920 21:21:36.503047   14992 watch.go:31] Failed to load config from \"/home/user/studio/dataconnect\": could not find dataconnect.yaml in `/home/user/studio/dataconnect`\n"}}
[debug] [2025-09-20T21:21:46.503Z] E0920 21:21:46.503087   14992 watch.go:31] Failed to load config from "/home/user/studio/dataconnect": could not find dataconnect.yaml in `/home/user/studio/dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0920 21:21:46.503087   14992 watch.go:31] Failed to load config from \"/home/user/studio/dataconnect\": could not find dataconnect.yaml in `/home/user/studio/dataconnect`\n"}}
[debug] [2025-09-20T21:21:56.507Z] E0920 21:21:56.507156   14992 watch.go:31] Failed to load config from "/home/user/studio/dataconnect": could not find dataconnect.yaml in `/home/user/studio/dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0920 21:21:56.507156   14992 watch.go:31] Failed to load config from \"/home/user/studio/dataconnect\": could not find dataconnect.yaml in `/home/user/studio/dataconnect`\n"}}
[debug] [2025-09-20T21:22:06.508Z] E0920 21:22:06.508116   14992 watch.go:31] Failed to load config from "/home/user/studio/dataconnect": could not find dataconnect.yaml in `/home/user/studio/dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0920 21:22:06.508116   14992 watch.go:31] Failed to load config from \"/home/user/studio/dataconnect\": could not find dataconnect.yaml in `/home/user/studio/dataconnect`\n"}}
[debug] [2025-09-20T21:22:16.502Z] E0920 21:22:16.502504   14992 watch.go:31] Failed to load config from "/home/user/studio/dataconnect": could not find dataconnect.yaml in `/home/user/studio/dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0920 21:22:16.502504   14992 watch.go:31] Failed to load config from \"/home/user/studio/dataconnect\": could not find dataconnect.yaml in `/home/user/studio/dataconnect`\n"}}
[debug] [2025-09-20T21:22:26.503Z] E0920 21:22:26.502678   14992 watch.go:31] Failed to load config from "/home/user/studio/dataconnect": could not find dataconnect.yaml in `/home/user/studio/dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0920 21:22:26.502678   14992 watch.go:31] Failed to load config from \"/home/user/studio/dataconnect\": could not find dataconnect.yaml in `/home/user/studio/dataconnect`\n"}}
[debug] [2025-09-20T21:22:36.513Z] E0920 21:22:36.513014   14992 watch.go:31] Failed to load config from "/home/user/studio/dataconnect": could not find dataconnect.yaml in `/home/user/studio/dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0920 21:22:36.513014   14992 watch.go:31] Failed to load config from \"/home/user/studio/dataconnect\": could not find dataconnect.yaml in `/home/user/studio/dataconnect`\n"}}
[debug] [2025-09-20T21:22:46.503Z] E0920 21:22:46.503337   14992 watch.go:31] Failed to load config from "/home/user/studio/dataconnect": could not find dataconnect.yaml in `/home/user/studio/dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0920 21:22:46.503337   14992 watch.go:31] Failed to load config from \"/home/user/studio/dataconnect\": could not find dataconnect.yaml in `/home/user/studio/dataconnect`\n"}}
[debug] [2025-09-20T21:22:56.511Z] E0920 21:22:56.511097   14992 watch.go:31] Failed to load config from "/home/user/studio/dataconnect": could not find dataconnect.yaml in `/home/user/studio/dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0920 21:22:56.511097   14992 watch.go:31] Failed to load config from \"/home/user/studio/dataconnect\": could not find dataconnect.yaml in `/home/user/studio/dataconnect`\n"}}
[debug] [2025-09-20T21:23:06.512Z] E0920 21:23:06.512040   14992 watch.go:31] Failed to load config from "/home/user/studio/dataconnect": could not find dataconnect.yaml in `/home/user/studio/dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0920 21:23:06.512040   14992 watch.go:31] Failed to load config from \"/home/user/studio/dataconnect\": could not find dataconnect.yaml in `/home/user/studio/dataconnect`\n"}}
[debug] [2025-09-20T21:23:16.504Z] E0920 21:23:16.504017   14992 watch.go:31] Failed to load config from "/home/user/studio/dataconnect": could not find dataconnect.yaml in `/home/user/studio/dataconnect`
E0920 21:23:16.504072   14992 watch.go:102] watchAllDir unable to walk dir `/home/user/studio/dataconnect`: lstat /home/user/studio/dataconnect: no such file or directory
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0920 21:23:16.504017   14992 watch.go:31] Failed to load config from \"/home/user/studio/dataconnect\": could not find dataconnect.yaml in `/home/user/studio/dataconnect`\nE0920 21:23:16.504072   14992 watch.go:102] watchAllDir unable to walk dir `/home/user/studio/dataconnect`: lstat /home/user/studio/dataconnect: no such file or directory\n"}}
[debug] [2025-09-20T21:23:26.512Z] E0920 21:23:26.512134   14992 watch.go:31] Failed to load config from "/home/user/studio/dataconnect": could not find dataconnect.yaml in `/home/user/studio/dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0920 21:23:26.512134   14992 watch.go:31] Failed to load config from \"/home/user/studio/dataconnect\": could not find dataconnect.yaml in `/home/user/studio/dataconnect`\n"}}
[debug] [2025-09-20T21:23:36.512Z] E0920 21:23:36.512054   14992 watch.go:31] Failed to load config from "/home/user/studio/dataconnect": could not find dataconnect.yaml in `/home/user/studio/dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0920 21:23:36.512054   14992 watch.go:31] Failed to load config from \"/home/user/studio/dataconnect\": could not find dataconnect.yaml in `/home/user/studio/dataconnect`\n"}}
[debug] [2025-09-20T21:23:46.503Z] E0920 21:23:46.502873   14992 watch.go:31] Failed to load config from "/home/user/studio/dataconnect": could not find dataconnect.yaml in `/home/user/studio/dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0920 21:23:46.502873   14992 watch.go:31] Failed to load config from \"/home/user/studio/dataconnect\": could not find dataconnect.yaml in `/home/user/studio/dataconnect`\n"}}
[debug] [2025-09-20T21:23:56.503Z] E0920 21:23:56.502758   14992 watch.go:31] Failed to load config from "/home/user/studio/dataconnect": could not find dataconnect.yaml in `/home/user/studio/dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0920 21:23:56.502758   14992 watch.go:31] Failed to load config from \"/home/user/studio/dataconnect\": could not find dataconnect.yaml in `/home/user/studio/dataconnect`\n"}}
[debug] [2025-09-20T21:24:06.512Z] E0920 21:24:06.512286   14992 watch.go:31] Failed to load config from "/home/user/studio/dataconnect": could not find dataconnect.yaml in `/home/user/studio/dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0920 21:24:06.512286   14992 watch.go:31] Failed to load config from \"/home/user/studio/dataconnect\": could not find dataconnect.yaml in `/home/user/studio/dataconnect`\n"}}
[debug] [2025-09-20T21:24:16.503Z] E0920 21:24:16.503403   14992 watch.go:31] Failed to load config from "/home/user/studio/dataconnect": could not find dataconnect.yaml in `/home/user/studio/dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0920 21:24:16.503403   14992 watch.go:31] Failed to load config from \"/home/user/studio/dataconnect\": could not find dataconnect.yaml in `/home/user/studio/dataconnect`\n"}}
[debug] [2025-09-20T21:24:26.503Z] E0920 21:24:26.503253   14992 watch.go:31] Failed to load config from "/home/user/studio/dataconnect": could not find dataconnect.yaml in `/home/user/studio/dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0920 21:24:26.503253   14992 watch.go:31] Failed to load config from \"/home/user/studio/dataconnect\": could not find dataconnect.yaml in `/home/user/studio/dataconnect`\n"}}
[debug] [2025-09-20T21:24:36.514Z] E0920 21:24:36.512637   14992 watch.go:31] Failed to load config from "/home/user/studio/dataconnect": could not find dataconnect.yaml in `/home/user/studio/dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0920 21:24:36.512637   14992 watch.go:31] Failed to load config from \"/home/user/studio/dataconnect\": could not find dataconnect.yaml in `/home/user/studio/dataconnect`\n"}}
[debug] [2025-09-20T21:24:46.503Z] E0920 21:24:46.503288   14992 watch.go:31] Failed to load config from "/home/user/studio/dataconnect": could not find dataconnect.yaml in `/home/user/studio/dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0920 21:24:46.503288   14992 watch.go:31] Failed to load config from \"/home/user/studio/dataconnect\": could not find dataconnect.yaml in `/home/user/studio/dataconnect`\n"}}
[debug] [2025-09-20T21:24:56.512Z] E0920 21:24:56.512075   14992 watch.go:31] Failed to load config from "/home/user/studio/dataconnect": could not find dataconnect.yaml in `/home/user/studio/dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0920 21:24:56.512075   14992 watch.go:31] Failed to load config from \"/home/user/studio/dataconnect\": could not find dataconnect.yaml in `/home/user/studio/dataconnect`\n"}}
[debug] [2025-09-20T21:25:06.513Z] E0920 21:25:06.513273   14992 watch.go:31] Failed to load config from "/home/user/studio/dataconnect": could not find dataconnect.yaml in `/home/user/studio/dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0920 21:25:06.513273   14992 watch.go:31] Failed to load config from \"/home/user/studio/dataconnect\": could not find dataconnect.yaml in `/home/user/studio/dataconnect`\n"}}
[debug] [2025-09-20T21:25:16.502Z] E0920 21:25:16.502465   14992 watch.go:31] Failed to load config from "/home/user/studio/dataconnect": could not find dataconnect.yaml in `/home/user/studio/dataconnect`
E0920 21:25:16.502522   14992 watch.go:102] watchAllDir unable to walk dir `/home/user/studio/dataconnect`: lstat /home/user/studio/dataconnect: no such file or directory
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0920 21:25:16.502465   14992 watch.go:31] Failed to load config from \"/home/user/studio/dataconnect\": could not find dataconnect.yaml in `/home/user/studio/dataconnect`\nE0920 21:25:16.502522   14992 watch.go:102] watchAllDir unable to walk dir `/home/user/studio/dataconnect`: lstat /home/user/studio/dataconnect: no such file or directory\n"}}
[debug] [2025-09-20T21:25:26.513Z] E0920 21:25:26.511034   14992 watch.go:31] Failed to load config from "/home/user/studio/dataconnect": could not find dataconnect.yaml in `/home/user/studio/dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0920 21:25:26.511034   14992 watch.go:31] Failed to load config from \"/home/user/studio/dataconnect\": could not find dataconnect.yaml in `/home/user/studio/dataconnect`\n"}}
[debug] [2025-09-20T21:25:36.513Z] E0920 21:25:36.512583   14992 watch.go:31] Failed to load config from "/home/user/studio/dataconnect": could not find dataconnect.yaml in `/home/user/studio/dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0920 21:25:36.512583   14992 watch.go:31] Failed to load config from \"/home/user/studio/dataconnect\": could not find dataconnect.yaml in `/home/user/studio/dataconnect`\n"}}
[debug] [2025-09-20T21:25:46.503Z] E0920 21:25:46.503079   14992 watch.go:31] Failed to load config from "/home/user/studio/dataconnect": could not find dataconnect.yaml in `/home/user/studio/dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0920 21:25:46.503079   14992 watch.go:31] Failed to load config from \"/home/user/studio/dataconnect\": could not find dataconnect.yaml in `/home/user/studio/dataconnect`\n"}}
[debug] [2025-09-20T21:25:56.502Z] E0920 21:25:56.502672   14992 watch.go:31] Failed to load config from "/home/user/studio/dataconnect": could not find dataconnect.yaml in `/home/user/studio/dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0920 21:25:56.502672   14992 watch.go:31] Failed to load config from \"/home/user/studio/dataconnect\": could not find dataconnect.yaml in `/home/user/studio/dataconnect`\n"}}
[debug] [2025-09-20T21:26:06.514Z] E0920 21:26:06.514052   14992 watch.go:31] Failed to load config from "/home/user/studio/dataconnect": could not find dataconnect.yaml in `/home/user/studio/dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0920 21:26:06.514052   14992 watch.go:31] Failed to load config from \"/home/user/studio/dataconnect\": could not find dataconnect.yaml in `/home/user/studio/dataconnect`\n"}}
[debug] [2025-09-20T21:26:16.503Z] E0920 21:26:16.502970   14992 watch.go:31] Failed to load config from "/home/user/studio/dataconnect": could not find dataconnect.yaml in `/home/user/studio/dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0920 21:26:16.502970   14992 watch.go:31] Failed to load config from \"/home/user/studio/dataconnect\": could not find dataconnect.yaml in `/home/user/studio/dataconnect`\n"}}
[debug] [2025-09-20T21:26:26.512Z] E0920 21:26:26.512002   14992 watch.go:31] Failed to load config from "/home/user/studio/dataconnect": could not find dataconnect.yaml in `/home/user/studio/dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0920 21:26:26.512002   14992 watch.go:31] Failed to load config from \"/home/user/studio/dataconnect\": could not find dataconnect.yaml in `/home/user/studio/dataconnect`\n"}}
[debug] [2025-09-20T21:26:36.516Z] I0920 21:26:36.513741   14992 load.go:31] Reloading schema and connectors...
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0920 21:26:36.513741   14992 load.go:31] Reloading schema and connectors...\n"}}
[debug] [2025-09-20T21:26:37.010Z] W0920 21:26:37.004397   14992 control.go:68] [/emulator/studio 373f] connector "example" has errors: example/queries.gql:2:3: mutation.InsertOrganization.organization_insert (data: Organization_Data): column createdAt is required but is missing in data

sources: example/queries.gql [530B] 
I0920 21:26:37.008082   14992 control.go:94] [/emulator/studio 373f] UpdateResources(): done
Schema: sources: schema/schema.gql [1207B] 
Connector "example": sources: example/queries.gql [530B] 
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"W0920 21:26:37.004397   14992 control.go:68] [/emulator/studio 373f] connector \"example\" has errors: example/queries.gql:2:3: mutation.InsertOrganization.organization_insert (data: Organization_Data): column createdAt is required but is missing in data\n\nsources: example/queries.gql [530B] \nI0920 21:26:37.008082   14992 control.go:94] [/emulator/studio 373f] UpdateResources(): done\nSchema: sources: schema/schema.gql [1207B] \nConnector \"example\": sources: example/queries.gql [530B] \n"}}
[debug] [2025-09-20T21:26:37.017Z] E0920 21:26:37.017095   14992 load.go:73] Could not load sources: 
Errors: example/queries.gql:2:3: mutation.InsertOrganization.organization_insert (data: Organization_Data): column createdAt is required but is missing in data
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0920 21:26:37.017095   14992 load.go:73] Could not load sources: \nErrors: example/queries.gql:2:3: mutation.InsertOrganization.organization_insert (data: Organization_Data): column createdAt is required but is missing in data\n"}}
[debug] [2025-09-20T21:26:37.026Z] I0920 21:26:37.019880   14992 collector.go:107] schema extensions wrote into "/home/user/studio/dataconnect/.dataconnect/schema"
Generated sources: main/input.gql [68752B] main/relation.gql [33941B] prelude.gql [73251B] main/query.gql [7604B] main/mutation.gql [20515B] main/implicit.gql [3592B] 
I0920 21:26:37.024056   14992 load.go:74] Finished reloading
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0920 21:26:37.019880   14992 collector.go:107] schema extensions wrote into \"/home/user/studio/dataconnect/.dataconnect/schema\"\nGenerated sources: main/input.gql [68752B] main/relation.gql [33941B] prelude.gql [73251B] main/query.gql [7604B] main/mutation.gql [20515B] main/implicit.gql [3592B] \nI0920 21:26:37.024056   14992 load.go:74] Finished reloading\n"}}
[info] [Firebase Plugin] Stopping Data Connect toolkit
[warn] ⚠  Data Connect Emulator has exited upon receiving signal: SIGINT 
[debug] [2025-09-20T22:10:33.697Z] [Firebase Plugin] Activating Firebase extension.
[debug] [2025-09-20T22:11:00.992Z] [Firebase Plugin] requireAuthWrapper
[debug] [2025-09-20T22:11:00.995Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform","https://www.googleapis.com/auth/userinfo.email"]
[debug] [2025-09-20T22:11:01.015Z] [Firebase Plugin] VSCode notification server listening on port 40001
[info] [Firebase Plugin] Starting Data Connect toolkit (version 2.13.0) on port 50001
[debug] [2025-09-20T22:11:08.009Z] W0920 22:11:07.971772   18781 control.go:68] [/emulator/studio 67be] connector "example" has errors: example/queries.gql:2:3: mutation.InsertOrganization.organization_insert (data: Organization_Data): column createdAt is required but is missing in data

sources: example/queries.gql [530B] 
I0920 22:11:08.006034   18781 control.go:94] [/emulator/studio 67be] UpdateResources(): done
Schema: sources: schema/schema.gql [1207B] 
Connector "example": sources: example/queries.gql [530B] 
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"W0920 22:11:07.971772   18781 control.go:68] [/emulator/studio 67be] connector \"example\" has errors: example/queries.gql:2:3: mutation.InsertOrganization.organization_insert (data: Organization_Data): column createdAt is required but is missing in data\n\nsources: example/queries.gql [530B] \nI0920 22:11:08.006034   18781 control.go:94] [/emulator/studio 67be] UpdateResources(): done\nSchema: sources: schema/schema.gql [1207B] \nConnector \"example\": sources: example/queries.gql [530B] \n"}}
[debug] [2025-09-20T22:11:08.012Z] Ignoring unsupported arg: auto_download {"metadata":{"emulator":{"name":"dataconnect"},"message":"Ignoring unsupported arg: auto_download"}}
[debug] [2025-09-20T22:11:08.012Z] Ignoring unsupported arg: port {"metadata":{"emulator":{"name":"dataconnect"},"message":"Ignoring unsupported arg: port"}}
[debug] [2025-09-20T22:11:08.013Z] Starting Data Connect Emulator with command {"binary":"/home/user/.cache/firebase/emulators/dataconnect-emulator-2.13.0","args":["--logtostderr","-v=2","dev","--listen=localhost:50001","--config_dir=/home/user/studio/dataconnect","--enable_output_schema_extensions=true","--enable_output_generated_sdk=true"],"optionalArgs":["listen","config_dir","enable_output_schema_extensions","enable_output_generated_sdk"],"joinArgs":true,"shell":false,"port":9399} {"metadata":{"emulator":{"name":"dataconnect"},"message":"Starting Data Connect Emulator with command {\"binary\":\"/home/user/.cache/firebase/emulators/dataconnect-emulator-2.13.0\",\"args\":[\"--logtostderr\",\"-v=2\",\"dev\",\"--listen=localhost:50001\",\"--config_dir=/home/user/studio/dataconnect\",\"--enable_output_schema_extensions=true\",\"--enable_output_generated_sdk=true\"],\"optionalArgs\":[\"listen\",\"config_dir\",\"enable_output_schema_extensions\",\"enable_output_generated_sdk\"],\"joinArgs\":true,\"shell\":false,\"port\":9399}"}}
[info] i  dataconnect: Data Connect Emulator logging to dataconnect-debug.log {"metadata":{"emulator":{"name":"dataconnect"},"message":"Data Connect Emulator logging to dataconnect-debug.log"}}
[error] [Firebase Plugin] requireAuth error: Could not refresh access token: Unsuccessful response status code. Request failed with status code 500
[debug] [2025-09-20T22:11:09.001Z] I0920 22:11:08.133822   18787 load.go:31] Reloading schema and connectors...
W0920 22:11:08.471823   18787 control.go:68] [/emulator/studio bab8] connector "example" has errors: example/queries.gql:2:3: mutation.InsertOrganization.organization_insert (data: Organization_Data): column createdAt is required but is missing in data

sources: example/queries.gql [530B] 
I0920 22:11:08.471925   18787 control.go:94] [/emulator/studio bab8] UpdateResources(): done
Schema: sources: schema/schema.gql [1207B] 
Connector "example": sources: example/queries.gql [530B] 
E0920 22:11:08.481228   18787 load.go:73] Could not load sources: 
Errors: example/queries.gql:2:3: mutation.InsertOrganization.organization_insert (data: Organization_Data): column createdAt is required but is missing in data
I0920 22:11:08.483144   18787 collector.go:107] schema extensions wrote into "/home/user/studio/dataconnect/.dataconnect/schema"
Generated sources: prelude.gql [73251B] main/implicit.gql [3592B] main/input.gql [68752B] main/relation.gql [33941B] main/query.gql [7604B] main/mutation.gql [20515B] 
I0920 22:11:08.483196   18787 load.go:74] Finished reloading
I0920 22:11:08.620891   18787 dev.go:102] Listening on address (HTTP + gRPC): 127.0.0.1:50001
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0920 22:11:08.133822   18787 load.go:31] Reloading schema and connectors...\nW0920 22:11:08.471823   18787 control.go:68] [/emulator/studio bab8] connector \"example\" has errors: example/queries.gql:2:3: mutation.InsertOrganization.organization_insert (data: Organization_Data): column createdAt is required but is missing in data\n\nsources: example/queries.gql [530B] \nI0920 22:11:08.471925   18787 control.go:94] [/emulator/studio bab8] UpdateResources(): done\nSchema: sources: schema/schema.gql [1207B] \nConnector \"example\": sources: example/queries.gql [530B] \nE0920 22:11:08.481228   18787 load.go:73] Could not load sources: \nErrors: example/queries.gql:2:3: mutation.InsertOrganization.organization_insert (data: Organization_Data): column createdAt is required but is missing in data\nI0920 22:11:08.483144   18787 collector.go:107] schema extensions wrote into \"/home/user/studio/dataconnect/.dataconnect/schema\"\nGenerated sources: prelude.gql [73251B] main/implicit.gql [3592B] main/input.gql [68752B] main/relation.gql [33941B] main/query.gql [7604B] main/mutation.gql [20515B] \nI0920 22:11:08.483196   18787 load.go:74] Finished reloading\nI0920 22:11:08.620891   18787 dev.go:102] Listening on address (HTTP + gRPC): 127.0.0.1:50001\n"}}
[debug] [2025-09-20T22:11:09.481Z] I0920 22:11:09.481071   18787 engine.go:169] [/emulator/studio bab8] ExecuteGraphqlRead IntrospectionQuery: succeeded. 
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0920 22:11:09.481071   18787 engine.go:169] [/emulator/studio bab8] ExecuteGraphqlRead IntrospectionQuery: succeeded. \n"}}
[info] [Firebase Plugin] Stopping Data Connect toolkit
[warn] ⚠  Data Connect Emulator has exited upon receiving signal: SIGINT 
[debug] [2025-09-20T22:30:02.967Z] [Firebase Plugin] Activating Firebase extension.
[debug] [2025-09-20T22:30:11.832Z] [Firebase Plugin] requireAuthWrapper
[debug] [2025-09-20T22:30:11.834Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform","https://www.googleapis.com/auth/userinfo.email"]
[debug] [2025-09-20T22:30:11.900Z] [Firebase Plugin] VSCode notification server listening on port 40001
[info] [Firebase Plugin] Starting Data Connect toolkit (version 2.13.0) on port 50001
[debug] [2025-09-20T22:30:17.309Z] W0920 22:30:17.298379   20418 control.go:68] [/emulator/studio f33f] connector "example" has errors: example/queries.gql:2:3: mutation.InsertOrganization.organization_insert (data: Organization_Data): column createdAt is required but is missing in data

sources: example/queries.gql [530B] 
I0920 22:30:17.298550   20418 control.go:94] [/emulator/studio f33f] UpdateResources(): done
Schema: sources: schema/schema.gql [1207B] 
Connector "example": sources: example/queries.gql [530B] 
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"W0920 22:30:17.298379   20418 control.go:68] [/emulator/studio f33f] connector \"example\" has errors: example/queries.gql:2:3: mutation.InsertOrganization.organization_insert (data: Organization_Data): column createdAt is required but is missing in data\n\nsources: example/queries.gql [530B] \nI0920 22:30:17.298550   20418 control.go:94] [/emulator/studio f33f] UpdateResources(): done\nSchema: sources: schema/schema.gql [1207B] \nConnector \"example\": sources: example/queries.gql [530B] \n"}}
[debug] [2025-09-20T22:30:17.317Z] Ignoring unsupported arg: auto_download {"metadata":{"emulator":{"name":"dataconnect"},"message":"Ignoring unsupported arg: auto_download"}}
[debug] [2025-09-20T22:30:17.320Z] Ignoring unsupported arg: port {"metadata":{"emulator":{"name":"dataconnect"},"message":"Ignoring unsupported arg: port"}}
[debug] [2025-09-20T22:30:17.321Z] Starting Data Connect Emulator with command {"binary":"/home/user/.cache/firebase/emulators/dataconnect-emulator-2.13.0","args":["--logtostderr","-v=2","dev","--listen=localhost:50001","--config_dir=/home/user/studio/dataconnect","--enable_output_schema_extensions=true","--enable_output_generated_sdk=true"],"optionalArgs":["listen","config_dir","enable_output_schema_extensions","enable_output_generated_sdk"],"joinArgs":true,"shell":false,"port":9399} {"metadata":{"emulator":{"name":"dataconnect"},"message":"Starting Data Connect Emulator with command {\"binary\":\"/home/user/.cache/firebase/emulators/dataconnect-emulator-2.13.0\",\"args\":[\"--logtostderr\",\"-v=2\",\"dev\",\"--listen=localhost:50001\",\"--config_dir=/home/user/studio/dataconnect\",\"--enable_output_schema_extensions=true\",\"--enable_output_generated_sdk=true\"],\"optionalArgs\":[\"listen\",\"config_dir\",\"enable_output_schema_extensions\",\"enable_output_generated_sdk\"],\"joinArgs\":true,\"shell\":false,\"port\":9399}"}}
[info] i  dataconnect: Data Connect Emulator logging to dataconnect-debug.log {"metadata":{"emulator":{"name":"dataconnect"},"message":"Data Connect Emulator logging to dataconnect-debug.log"}}
[debug] [2025-09-20T22:30:17.620Z] I0920 22:30:17.490834   20423 load.go:31] Reloading schema and connectors...
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0920 22:30:17.490834   20423 load.go:31] Reloading schema and connectors...\n"}}
[debug] [2025-09-20T22:30:17.937Z] W0920 22:30:17.837087   20423 control.go:68] [/emulator/studio 0fbf] connector "example" has errors: example/queries.gql:2:3: mutation.InsertOrganization.organization_insert (data: Organization_Data): column createdAt is required but is missing in data

sources: example/queries.gql [530B] 
I0920 22:30:17.837161   20423 control.go:94] [/emulator/studio 0fbf] UpdateResources(): done
Schema: sources: schema/schema.gql [1207B] 
Connector "example": sources: example/queries.gql [530B] 
E0920 22:30:17.842078   20423 load.go:73] Could not load sources: 
Errors: example/queries.gql:2:3: mutation.InsertOrganization.organization_insert (data: Organization_Data): column createdAt is required but is missing in data
I0920 22:30:17.846247   20423 collector.go:107] schema extensions wrote into "/home/user/studio/dataconnect/.dataconnect/schema"
Generated sources: prelude.gql [73251B] main/query.gql [7604B] main/mutation.gql [20515B] main/implicit.gql [3592B] main/input.gql [68752B] main/relation.gql [33941B] 
I0920 22:30:17.848226   20423 load.go:74] Finished reloading
I0920 22:30:17.911486   20423 dev.go:102] Listening on address (HTTP + gRPC): 127.0.0.1:50001
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"W0920 22:30:17.837087   20423 control.go:68] [/emulator/studio 0fbf] connector \"example\" has errors: example/queries.gql:2:3: mutation.InsertOrganization.organization_insert (data: Organization_Data): column createdAt is required but is missing in data\n\nsources: example/queries.gql [530B] \nI0920 22:30:17.837161   20423 control.go:94] [/emulator/studio 0fbf] UpdateResources(): done\nSchema: sources: schema/schema.gql [1207B] \nConnector \"example\": sources: example/queries.gql [530B] \nE0920 22:30:17.842078   20423 load.go:73] Could not load sources: \nErrors: example/queries.gql:2:3: mutation.InsertOrganization.organization_insert (data: Organization_Data): column createdAt is required but is missing in data\nI0920 22:30:17.846247   20423 collector.go:107] schema extensions wrote into \"/home/user/studio/dataconnect/.dataconnect/schema\"\nGenerated sources: prelude.gql [73251B] main/query.gql [7604B] main/mutation.gql [20515B] main/implicit.gql [3592B] main/input.gql [68752B] main/relation.gql [33941B] \nI0920 22:30:17.848226   20423 load.go:74] Finished reloading\nI0920 22:30:17.911486   20423 dev.go:102] Listening on address (HTTP + gRPC): 127.0.0.1:50001\n"}}
[debug] [2025-09-20T22:30:19.827Z] I0920 22:30:19.165697   20423 engine.go:169] [/emulator/studio 0fbf] ExecuteGraphqlRead IntrospectionQuery: succeeded. 
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0920 22:30:19.165697   20423 engine.go:169] [/emulator/studio 0fbf] ExecuteGraphqlRead IntrospectionQuery: succeeded. \n"}}
[debug] [2025-09-20T22:30:37.059Z] Running auto auth
[debug] [2025-09-20T22:30:37.478Z] No OAuth tokens found
[debug] [2025-09-20T22:30:37.482Z] [Firebase Plugin] User found:  innovativeenterprisesoman@gmail.com
[info] [Firebase Plugin] (Core:Project) New user detected, fetching projects
[debug] [2025-09-20T22:30:37.486Z] No OAuth tokens found
[debug] [2025-09-20T22:30:37.499Z] >>> [apiv2][query] GET https://firebase.googleapis.com/v1beta1/projects pageSize=1000
[debug] [2025-09-20T22:30:38.279Z] <<< [apiv2][status] GET https://firebase.googleapis.com/v1beta1/projects 200
[debug] [2025-09-20T22:30:38.281Z] <<< [apiv2][body] GET https://firebase.googleapis.com/v1beta1/projects [omitted]
[info] [Firebase Plugin] Stopping Data Connect toolkit
[warn] ⚠  Data Connect Emulator has exited upon receiving signal: SIGINT 
[debug] [2025-09-20T22:45:34.470Z] [Firebase Plugin] Activating Firebase extension.
[debug] [2025-09-20T22:46:01.626Z] [Firebase Plugin] requireAuthWrapper
[debug] [2025-09-20T22:46:01.630Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform","https://www.googleapis.com/auth/userinfo.email"]
[debug] [2025-09-20T22:46:01.632Z] > authorizing via signed-in user (innovativeenterprisesoman@gmail.com)
[debug] [2025-09-20T22:46:01.644Z] No OAuth tokens found
[debug] [2025-09-20T22:46:01.646Z] No OAuth tokens found
[debug] [2025-09-20T22:46:01.648Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform","https://www.googleapis.com/auth/userinfo.email"]
[debug] [2025-09-20T22:46:01.649Z] > authorizing via signed-in user (innovativeenterprisesoman@gmail.com)
[debug] [2025-09-20T22:46:01.650Z] [Firebase Plugin] User found:  innovativeenterprisesoman@gmail.com
[info] [Firebase Plugin] (Core:Project) New user detected, fetching projects
[debug] [2025-09-20T22:46:01.657Z] No OAuth tokens found
[debug] [2025-09-20T22:46:01.662Z] >>> [apiv2][query] GET https://firebase.googleapis.com/v1beta1/projects pageSize=1000
[debug] [2025-09-20T22:46:01.775Z] [Firebase Plugin] VSCode notification server listening on port 40001
[debug] [2025-09-20T22:46:03.473Z] <<< [apiv2][status] GET https://firebase.googleapis.com/v1beta1/projects 200
[debug] [2025-09-20T22:46:03.474Z] <<< [apiv2][body] GET https://firebase.googleapis.com/v1beta1/projects [omitted]
[info] [Firebase Plugin] Starting Data Connect toolkit (version 2.13.0) on port 50001
[debug] [2025-09-20T22:46:06.263Z] W0920 22:46:06.258927   23940 control.go:68] [/emulator/studio e2c9] connector "example" has errors: example/queries.gql:2:3: mutation.InsertOrganization.organization_insert (data: Organization_Data): column createdAt is required but is missing in data

sources: example/queries.gql [530B] 
I0920 22:46:06.259074   23940 control.go:94] [/emulator/studio e2c9] UpdateResources(): done
Schema: sources: schema/schema.gql [1207B] 
Connector "example": sources: example/queries.gql [530B] 
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"W0920 22:46:06.258927   23940 control.go:68] [/emulator/studio e2c9] connector \"example\" has errors: example/queries.gql:2:3: mutation.InsertOrganization.organization_insert (data: Organization_Data): column createdAt is required but is missing in data\n\nsources: example/queries.gql [530B] \nI0920 22:46:06.259074   23940 control.go:94] [/emulator/studio e2c9] UpdateResources(): done\nSchema: sources: schema/schema.gql [1207B] \nConnector \"example\": sources: example/queries.gql [530B] \n"}}
[debug] [2025-09-20T22:46:06.265Z] Ignoring unsupported arg: auto_download {"metadata":{"emulator":{"name":"dataconnect"},"message":"Ignoring unsupported arg: auto_download"}}
[debug] [2025-09-20T22:46:06.266Z] Ignoring unsupported arg: port {"metadata":{"emulator":{"name":"dataconnect"},"message":"Ignoring unsupported arg: port"}}
[debug] [2025-09-20T22:46:06.267Z] Starting Data Connect Emulator with command {"binary":"/home/user/.cache/firebase/emulators/dataconnect-emulator-2.13.0","args":["--logtostderr","-v=2","dev","--listen=localhost:50001","--config_dir=/home/user/studio/dataconnect","--enable_output_schema_extensions=true","--enable_output_generated_sdk=true"],"optionalArgs":["listen","config_dir","enable_output_schema_extensions","enable_output_generated_sdk"],"joinArgs":true,"shell":false,"port":9399} {"metadata":{"emulator":{"name":"dataconnect"},"message":"Starting Data Connect Emulator with command {\"binary\":\"/home/user/.cache/firebase/emulators/dataconnect-emulator-2.13.0\",\"args\":[\"--logtostderr\",\"-v=2\",\"dev\",\"--listen=localhost:50001\",\"--config_dir=/home/user/studio/dataconnect\",\"--enable_output_schema_extensions=true\",\"--enable_output_generated_sdk=true\"],\"optionalArgs\":[\"listen\",\"config_dir\",\"enable_output_schema_extensions\",\"enable_output_generated_sdk\"],\"joinArgs\":true,\"shell\":false,\"port\":9399}"}}
[info] i  dataconnect: Data Connect Emulator logging to dataconnect-debug.log {"metadata":{"emulator":{"name":"dataconnect"},"message":"Data Connect Emulator logging to dataconnect-debug.log"}}
[debug] [2025-09-20T22:46:06.395Z] I0920 22:46:06.382745   23945 load.go:31] Reloading schema and connectors...
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0920 22:46:06.382745   23945 load.go:31] Reloading schema and connectors...\n"}}
[debug] [2025-09-20T22:46:06.636Z] W0920 22:46:06.635769   23945 control.go:68] [/emulator/studio b0b5] connector "example" has errors: example/queries.gql:2:3: mutation.InsertOrganization.organization_insert (data: Organization_Data): column createdAt is required but is missing in data

sources: example/queries.gql [530B] 
I0920 22:46:06.635848   23945 control.go:94] [/emulator/studio b0b5] UpdateResources(): done
Schema: sources: schema/schema.gql [1207B] 
Connector "example": sources: example/queries.gql [530B] 
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"W0920 22:46:06.635769   23945 control.go:68] [/emulator/studio b0b5] connector \"example\" has errors: example/queries.gql:2:3: mutation.InsertOrganization.organization_insert (data: Organization_Data): column createdAt is required but is missing in data\n\nsources: example/queries.gql [530B] \nI0920 22:46:06.635848   23945 control.go:94] [/emulator/studio b0b5] UpdateResources(): done\nSchema: sources: schema/schema.gql [1207B] \nConnector \"example\": sources: example/queries.gql [530B] \n"}}
[debug] [2025-09-20T22:46:06.648Z] E0920 22:46:06.647444   23945 load.go:73] Could not load sources: 
Errors: example/queries.gql:2:3: mutation.InsertOrganization.organization_insert (data: Organization_Data): column createdAt is required but is missing in data
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0920 22:46:06.647444   23945 load.go:73] Could not load sources: \nErrors: example/queries.gql:2:3: mutation.InsertOrganization.organization_insert (data: Organization_Data): column createdAt is required but is missing in data\n"}}
[debug] [2025-09-20T22:46:06.661Z] I0920 22:46:06.658281   23945 collector.go:107] schema extensions wrote into "/home/user/studio/dataconnect/.dataconnect/schema"
Generated sources: main/input.gql [68752B] main/relation.gql [33941B] main/query.gql [7604B] main/mutation.gql [20515B] prelude.gql [73251B] main/implicit.gql [3592B] 
I0920 22:46:06.658351   23945 load.go:74] Finished reloading
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0920 22:46:06.658281   23945 collector.go:107] schema extensions wrote into \"/home/user/studio/dataconnect/.dataconnect/schema\"\nGenerated sources: main/input.gql [68752B] main/relation.gql [33941B] main/query.gql [7604B] main/mutation.gql [20515B] prelude.gql [73251B] main/implicit.gql [3592B] \nI0920 22:46:06.658351   23945 load.go:74] Finished reloading\n"}}
[debug] [2025-09-20T22:46:06.675Z] I0920 22:46:06.673763   23945 dev.go:102] Listening on address (HTTP + gRPC): 127.0.0.1:50001
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0920 22:46:06.673763   23945 dev.go:102] Listening on address (HTTP + gRPC): 127.0.0.1:50001\n"}}
[debug] [2025-09-20T22:46:06.890Z] I0920 22:46:06.889515   23945 engine.go:169] [/emulator/studio b0b5] ExecuteGraphqlRead IntrospectionQuery: succeeded. 
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0920 22:46:06.889515   23945 engine.go:169] [/emulator/studio b0b5] ExecuteGraphqlRead IntrospectionQuery: succeeded. \n"}}
[info] [Firebase Plugin] Stopping Data Connect toolkit
[warn] ⚠  Data Connect Emulator has exited upon receiving signal: SIGINT 
[debug] [2025-09-20T23:15:49.032Z] [Firebase Plugin] Activating Firebase extension.
[debug] [2025-09-20T23:16:43.553Z] [Firebase Plugin] requireAuthWrapper
[debug] [2025-09-20T23:16:43.561Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform","https://www.googleapis.com/auth/userinfo.email"]
[debug] [2025-09-20T23:16:43.648Z] [Firebase Plugin] VSCode notification server listening on port 40001
[info] [Firebase Plugin] Starting Data Connect toolkit (version 2.13.0) on port 50001
[debug] [2025-09-20T23:16:48.275Z] W0920 23:16:48.258155   30322 control.go:68] [/emulator/studio 2951] connector "example" has errors: example/queries.gql:2:3: mutation.InsertOrganization.organization_insert (data: Organization_Data): column createdAt is required but is missing in data

sources: example/queries.gql [530B] 
I0920 23:16:48.261626   30322 control.go:94] [/emulator/studio 2951] UpdateResources(): done
Schema: sources: schema/schema.gql [1207B] 
Connector "example": sources: example/queries.gql [530B] 
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"W0920 23:16:48.258155   30322 control.go:68] [/emulator/studio 2951] connector \"example\" has errors: example/queries.gql:2:3: mutation.InsertOrganization.organization_insert (data: Organization_Data): column createdAt is required but is missing in data\n\nsources: example/queries.gql [530B] \nI0920 23:16:48.261626   30322 control.go:94] [/emulator/studio 2951] UpdateResources(): done\nSchema: sources: schema/schema.gql [1207B] \nConnector \"example\": sources: example/queries.gql [530B] \n"}}
[debug] [2025-09-20T23:16:48.283Z] Ignoring unsupported arg: auto_download {"metadata":{"emulator":{"name":"dataconnect"},"message":"Ignoring unsupported arg: auto_download"}}
[debug] [2025-09-20T23:16:48.284Z] Ignoring unsupported arg: port {"metadata":{"emulator":{"name":"dataconnect"},"message":"Ignoring unsupported arg: port"}}
[debug] [2025-09-20T23:16:48.288Z] Starting Data Connect Emulator with command {"binary":"/home/user/.cache/firebase/emulators/dataconnect-emulator-2.13.0","args":["--logtostderr","-v=2","dev","--listen=localhost:50001","--config_dir=/home/user/studio/dataconnect","--enable_output_schema_extensions=true","--enable_output_generated_sdk=true"],"optionalArgs":["listen","config_dir","enable_output_schema_extensions","enable_output_generated_sdk"],"joinArgs":true,"shell":false,"port":9399} {"metadata":{"emulator":{"name":"dataconnect"},"message":"Starting Data Connect Emulator with command {\"binary\":\"/home/user/.cache/firebase/emulators/dataconnect-emulator-2.13.0\",\"args\":[\"--logtostderr\",\"-v=2\",\"dev\",\"--listen=localhost:50001\",\"--config_dir=/home/user/studio/dataconnect\",\"--enable_output_schema_extensions=true\",\"--enable_output_generated_sdk=true\"],\"optionalArgs\":[\"listen\",\"config_dir\",\"enable_output_schema_extensions\",\"enable_output_generated_sdk\"],\"joinArgs\":true,\"shell\":false,\"port\":9399}"}}
[info] i  dataconnect: Data Connect Emulator logging to dataconnect-debug.log {"metadata":{"emulator":{"name":"dataconnect"},"message":"Data Connect Emulator logging to dataconnect-debug.log"}}
[debug] [2025-09-20T23:16:48.533Z] I0920 23:16:48.430105   30328 load.go:31] Reloading schema and connectors...
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0920 23:16:48.430105   30328 load.go:31] Reloading schema and connectors...\n"}}
[debug] [2025-09-20T23:16:48.947Z] W0920 23:16:48.832173   30328 control.go:68] [/emulator/studio cb6a] connector "example" has errors: example/queries.gql:2:3: mutation.InsertOrganization.organization_insert (data: Organization_Data): column createdAt is required but is missing in data

sources: example/queries.gql [530B] 
I0920 23:16:48.832261   30328 control.go:94] [/emulator/studio cb6a] UpdateResources(): done
Schema: sources: schema/schema.gql [1207B] 
Connector "example": sources: example/queries.gql [530B] 
E0920 23:16:48.858322   30328 load.go:73] Could not load sources: 
Errors: example/queries.gql:2:3: mutation.InsertOrganization.organization_insert (data: Organization_Data): column createdAt is required but is missing in data
I0920 23:16:48.879580   30328 collector.go:107] schema extensions wrote into "/home/user/studio/dataconnect/.dataconnect/schema"
Generated sources: prelude.gql [73251B] main/implicit.gql [3592B] main/input.gql [68752B] main/relation.gql [33941B] main/query.gql [7604B] main/mutation.gql [20515B] 
I0920 23:16:48.879657   30328 load.go:74] Finished reloading
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"W0920 23:16:48.832173   30328 control.go:68] [/emulator/studio cb6a] connector \"example\" has errors: example/queries.gql:2:3: mutation.InsertOrganization.organization_insert (data: Organization_Data): column createdAt is required but is missing in data\n\nsources: example/queries.gql [530B] \nI0920 23:16:48.832261   30328 control.go:94] [/emulator/studio cb6a] UpdateResources(): done\nSchema: sources: schema/schema.gql [1207B] \nConnector \"example\": sources: example/queries.gql [530B] \nE0920 23:16:48.858322   30328 load.go:73] Could not load sources: \nErrors: example/queries.gql:2:3: mutation.InsertOrganization.organization_insert (data: Organization_Data): column createdAt is required but is missing in data\nI0920 23:16:48.879580   30328 collector.go:107] schema extensions wrote into \"/home/user/studio/dataconnect/.dataconnect/schema\"\nGenerated sources: prelude.gql [73251B] main/implicit.gql [3592B] main/input.gql [68752B] main/relation.gql [33941B] main/query.gql [7604B] main/mutation.gql [20515B] \nI0920 23:16:48.879657   30328 load.go:74] Finished reloading\n"}}
[debug] [2025-09-20T23:16:48.991Z] I0920 23:16:48.949505   30328 dev.go:102] Listening on address (HTTP + gRPC): 127.0.0.1:50001
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0920 23:16:48.949505   30328 dev.go:102] Listening on address (HTTP + gRPC): 127.0.0.1:50001\n"}}
[debug] [2025-09-20T23:16:49.510Z] I0920 23:16:49.510028   30328 engine.go:169] [/emulator/studio cb6a] ExecuteGraphqlRead IntrospectionQuery: succeeded. 
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0920 23:16:49.510028   30328 engine.go:169] [/emulator/studio cb6a] ExecuteGraphqlRead IntrospectionQuery: succeeded. \n"}}
[debug] [2025-09-20T23:17:11.020Z] Running auto auth
[debug] [2025-09-20T23:17:11.087Z] No OAuth tokens found
[debug] [2025-09-20T23:17:11.089Z] [Firebase Plugin] User found:  innovativeenterprisesoman@gmail.com
[info] [Firebase Plugin] (Core:Project) New user detected, fetching projects
[debug] [2025-09-20T23:17:11.095Z] No OAuth tokens found
[debug] [2025-09-20T23:17:11.101Z] >>> [apiv2][query] GET https://firebase.googleapis.com/v1beta1/projects pageSize=1000
[debug] [2025-09-20T23:17:11.702Z] <<< [apiv2][status] GET https://firebase.googleapis.com/v1beta1/projects 200
[debug] [2025-09-20T23:17:11.703Z] <<< [apiv2][body] GET https://firebase.googleapis.com/v1beta1/projects [omitted]
[info] [Firebase Plugin] Stopping Data Connect toolkit
[warn] ⚠  Data Connect Emulator has exited upon receiving signal: SIGINT 
[debug] [2025-09-21T02:19:16.576Z] [Firebase Plugin] Activating Firebase extension.
[debug] [2025-09-21T02:20:21.232Z] [Firebase Plugin] requireAuthWrapper
[debug] [2025-09-21T02:20:21.249Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform","https://www.googleapis.com/auth/userinfo.email"]
[debug] [2025-09-21T02:20:21.346Z] [Firebase Plugin] VSCode notification server listening on port 40001
[info] [Firebase Plugin] Starting Data Connect toolkit (version 2.13.0) on port 50001
[debug] [2025-09-21T02:20:32.725Z] W0921 02:20:32.720878   33077 control.go:68] [/emulator/studio 4679] connector "example" has errors: example/queries.gql:2:3: mutation.InsertOrganization.organization_insert (data: Organization_Data): column createdAt is required but is missing in data

sources: example/queries.gql [530B] 
I0921 02:20:32.721255   33077 control.go:94] [/emulator/studio 4679] UpdateResources(): done
Schema: sources: schema/schema.gql [1207B] 
Connector "example": sources: example/queries.gql [530B] 
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"W0921 02:20:32.720878   33077 control.go:68] [/emulator/studio 4679] connector \"example\" has errors: example/queries.gql:2:3: mutation.InsertOrganization.organization_insert (data: Organization_Data): column createdAt is required but is missing in data\n\nsources: example/queries.gql [530B] \nI0921 02:20:32.721255   33077 control.go:94] [/emulator/studio 4679] UpdateResources(): done\nSchema: sources: schema/schema.gql [1207B] \nConnector \"example\": sources: example/queries.gql [530B] \n"}}
[debug] [2025-09-21T02:20:32.729Z] Ignoring unsupported arg: auto_download {"metadata":{"emulator":{"name":"dataconnect"},"message":"Ignoring unsupported arg: auto_download"}}
[debug] [2025-09-21T02:20:32.729Z] Ignoring unsupported arg: port {"metadata":{"emulator":{"name":"dataconnect"},"message":"Ignoring unsupported arg: port"}}
[debug] [2025-09-21T02:20:32.730Z] Starting Data Connect Emulator with command {"binary":"/home/user/.cache/firebase/emulators/dataconnect-emulator-2.13.0","args":["--logtostderr","-v=2","dev","--listen=localhost:50001","--config_dir=/home/user/studio/dataconnect","--enable_output_schema_extensions=true","--enable_output_generated_sdk=true"],"optionalArgs":["listen","config_dir","enable_output_schema_extensions","enable_output_generated_sdk"],"joinArgs":true,"shell":false,"port":9399} {"metadata":{"emulator":{"name":"dataconnect"},"message":"Starting Data Connect Emulator with command {\"binary\":\"/home/user/.cache/firebase/emulators/dataconnect-emulator-2.13.0\",\"args\":[\"--logtostderr\",\"-v=2\",\"dev\",\"--listen=localhost:50001\",\"--config_dir=/home/user/studio/dataconnect\",\"--enable_output_schema_extensions=true\",\"--enable_output_generated_sdk=true\"],\"optionalArgs\":[\"listen\",\"config_dir\",\"enable_output_schema_extensions\",\"enable_output_generated_sdk\"],\"joinArgs\":true,\"shell\":false,\"port\":9399}"}}
[info] i  dataconnect: Data Connect Emulator logging to dataconnect-debug.log {"metadata":{"emulator":{"name":"dataconnect"},"message":"Data Connect Emulator logging to dataconnect-debug.log"}}
[debug] [2025-09-21T02:20:32.993Z] I0921 02:20:32.832148   33082 load.go:31] Reloading schema and connectors...
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0921 02:20:32.832148   33082 load.go:31] Reloading schema and connectors...\n"}}
[debug] [2025-09-21T02:20:33.296Z] W0921 02:20:33.266195   33082 control.go:68] [/emulator/studio 5533] connector "example" has errors: example/queries.gql:2:3: mutation.InsertOrganization.organization_insert (data: Organization_Data): column createdAt is required but is missing in data

sources: example/queries.gql [530B] 
I0921 02:20:33.266382   33082 control.go:94] [/emulator/studio 5533] UpdateResources(): done
Schema: sources: schema/schema.gql [1207B] 
Connector "example": sources: example/queries.gql [530B] 
E0921 02:20:33.274981   33082 load.go:73] Could not load sources: 
Errors: example/queries.gql:2:3: mutation.InsertOrganization.organization_insert (data: Organization_Data): column createdAt is required but is missing in data
I0921 02:20:33.281197   33082 collector.go:107] schema extensions wrote into "/home/user/studio/dataconnect/.dataconnect/schema"
Generated sources: prelude.gql [73251B] main/input.gql [68752B] main/relation.gql [33941B] main/query.gql [7604B] main/mutation.gql [20515B] main/implicit.gql [3592B] 
I0921 02:20:33.281273   33082 load.go:74] Finished reloading
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"W0921 02:20:33.266195   33082 control.go:68] [/emulator/studio 5533] connector \"example\" has errors: example/queries.gql:2:3: mutation.InsertOrganization.organization_insert (data: Organization_Data): column createdAt is required but is missing in data\n\nsources: example/queries.gql [530B] \nI0921 02:20:33.266382   33082 control.go:94] [/emulator/studio 5533] UpdateResources(): done\nSchema: sources: schema/schema.gql [1207B] \nConnector \"example\": sources: example/queries.gql [530B] \nE0921 02:20:33.274981   33082 load.go:73] Could not load sources: \nErrors: example/queries.gql:2:3: mutation.InsertOrganization.organization_insert (data: Organization_Data): column createdAt is required but is missing in data\nI0921 02:20:33.281197   33082 collector.go:107] schema extensions wrote into \"/home/user/studio/dataconnect/.dataconnect/schema\"\nGenerated sources: prelude.gql [73251B] main/input.gql [68752B] main/relation.gql [33941B] main/query.gql [7604B] main/mutation.gql [20515B] main/implicit.gql [3592B] \nI0921 02:20:33.281273   33082 load.go:74] Finished reloading\n"}}
[debug] [2025-09-21T02:20:33.509Z] I0921 02:20:33.336207   33082 dev.go:102] Listening on address (HTTP + gRPC): 127.0.0.1:50001
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0921 02:20:33.336207   33082 dev.go:102] Listening on address (HTTP + gRPC): 127.0.0.1:50001\n"}}
[debug] [2025-09-21T02:20:34.219Z] I0921 02:20:34.218490   33082 engine.go:169] [/emulator/studio 5533] ExecuteGraphqlRead IntrospectionQuery: succeeded. 
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0921 02:20:34.218490   33082 engine.go:169] [/emulator/studio 5533] ExecuteGraphqlRead IntrospectionQuery: succeeded. \n"}}
[error] [Firebase Plugin] requireAuth error: Could not refresh access token: Unsuccessful response status code. Request failed with status code 500
[debug] [2025-09-21T19:50:12.501Z] [Firebase Plugin] Activating Firebase extension.
[debug] [2025-09-21T19:50:21.837Z] [Firebase Plugin] requireAuthWrapper
[debug] [2025-09-21T19:50:21.839Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform","https://www.googleapis.com/auth/userinfo.email"]
[debug] [2025-09-21T19:50:21.861Z] [Firebase Plugin] VSCode notification server listening on port 40001
[info] [Firebase Plugin] Starting Data Connect toolkit (version 2.13.0) on port 50001
[debug] [2025-09-21T19:50:25.271Z] W0921 19:50:25.266279    2399 control.go:68] [/emulator/studio 3b02] connector "example" has errors: example/queries.gql:2:3: mutation.InsertOrganization.organization_insert (data: Organization_Data): column createdAt is required but is missing in data

sources: example/queries.gql [530B] 
I0921 19:50:25.266494    2399 control.go:94] [/emulator/studio 3b02] UpdateResources(): done
Schema: sources: schema/schema.gql [1207B] 
Connector "example": sources: example/queries.gql [530B] 
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"W0921 19:50:25.266279    2399 control.go:68] [/emulator/studio 3b02] connector \"example\" has errors: example/queries.gql:2:3: mutation.InsertOrganization.organization_insert (data: Organization_Data): column createdAt is required but is missing in data\n\nsources: example/queries.gql [530B] \nI0921 19:50:25.266494    2399 control.go:94] [/emulator/studio 3b02] UpdateResources(): done\nSchema: sources: schema/schema.gql [1207B] \nConnector \"example\": sources: example/queries.gql [530B] \n"}}
[debug] [2025-09-21T19:50:25.277Z] Ignoring unsupported arg: auto_download {"metadata":{"emulator":{"name":"dataconnect"},"message":"Ignoring unsupported arg: auto_download"}}
[debug] [2025-09-21T19:50:25.279Z] Ignoring unsupported arg: port {"metadata":{"emulator":{"name":"dataconnect"},"message":"Ignoring unsupported arg: port"}}
[debug] [2025-09-21T19:50:25.280Z] Starting Data Connect Emulator with command {"binary":"/home/user/.cache/firebase/emulators/dataconnect-emulator-2.13.0","args":["--logtostderr","-v=2","dev","--listen=localhost:50001","--config_dir=/home/user/studio/dataconnect","--enable_output_schema_extensions=true","--enable_output_generated_sdk=true"],"optionalArgs":["listen","config_dir","enable_output_schema_extensions","enable_output_generated_sdk"],"joinArgs":true,"shell":false,"port":9399} {"metadata":{"emulator":{"name":"dataconnect"},"message":"Starting Data Connect Emulator with command {\"binary\":\"/home/user/.cache/firebase/emulators/dataconnect-emulator-2.13.0\",\"args\":[\"--logtostderr\",\"-v=2\",\"dev\",\"--listen=localhost:50001\",\"--config_dir=/home/user/studio/dataconnect\",\"--enable_output_schema_extensions=true\",\"--enable_output_generated_sdk=true\"],\"optionalArgs\":[\"listen\",\"config_dir\",\"enable_output_schema_extensions\",\"enable_output_generated_sdk\"],\"joinArgs\":true,\"shell\":false,\"port\":9399}"}}
[info] i  dataconnect: Data Connect Emulator logging to dataconnect-debug.log {"metadata":{"emulator":{"name":"dataconnect"},"message":"Data Connect Emulator logging to dataconnect-debug.log"}}
[debug] [2025-09-21T19:50:25.466Z] I0921 19:50:25.353886    2404 load.go:31] Reloading schema and connectors...
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0921 19:50:25.353886    2404 load.go:31] Reloading schema and connectors...\n"}}
[debug] [2025-09-21T19:50:26.082Z] W0921 19:50:25.716121    2404 control.go:68] [/emulator/studio 1ca9] connector "example" has errors: example/queries.gql:2:3: mutation.InsertOrganization.organization_insert (data: Organization_Data): column createdAt is required but is missing in data

sources: example/queries.gql [530B] 
I0921 19:50:25.716208    2404 control.go:94] [/emulator/studio 1ca9] UpdateResources(): done
Schema: sources: schema/schema.gql [1207B] 
Connector "example": sources: example/queries.gql [530B] 
E0921 19:50:25.731848    2404 load.go:73] Could not load sources: 
Errors: example/queries.gql:2:3: mutation.InsertOrganization.organization_insert (data: Organization_Data): column createdAt is required but is missing in data
I0921 19:50:25.734879    2404 collector.go:107] schema extensions wrote into "/home/user/studio/dataconnect/.dataconnect/schema"
Generated sources: main/mutation.gql [20515B] prelude.gql [73251B] main/implicit.gql [3592B] main/input.gql [68752B] main/relation.gql [33941B] main/query.gql [7604B] 
I0921 19:50:25.735542    2404 load.go:74] Finished reloading
I0921 19:50:25.755622    2404 dev.go:102] Listening on address (HTTP + gRPC): 127.0.0.1:50001
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"W0921 19:50:25.716121    2404 control.go:68] [/emulator/studio 1ca9] connector \"example\" has errors: example/queries.gql:2:3: mutation.InsertOrganization.organization_insert (data: Organization_Data): column createdAt is required but is missing in data\n\nsources: example/queries.gql [530B] \nI0921 19:50:25.716208    2404 control.go:94] [/emulator/studio 1ca9] UpdateResources(): done\nSchema: sources: schema/schema.gql [1207B] \nConnector \"example\": sources: example/queries.gql [530B] \nE0921 19:50:25.731848    2404 load.go:73] Could not load sources: \nErrors: example/queries.gql:2:3: mutation.InsertOrganization.organization_insert (data: Organization_Data): column createdAt is required but is missing in data\nI0921 19:50:25.734879    2404 collector.go:107] schema extensions wrote into \"/home/user/studio/dataconnect/.dataconnect/schema\"\nGenerated sources: main/mutation.gql [20515B] prelude.gql [73251B] main/implicit.gql [3592B] main/input.gql [68752B] main/relation.gql [33941B] main/query.gql [7604B] \nI0921 19:50:25.735542    2404 load.go:74] Finished reloading\nI0921 19:50:25.755622    2404 dev.go:102] Listening on address (HTTP + gRPC): 127.0.0.1:50001\n"}}
[debug] [2025-09-21T19:50:27.105Z] I0921 19:50:27.102624    2404 engine.go:169] [/emulator/studio 1ca9] ExecuteGraphqlRead IntrospectionQuery: succeeded. 
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0921 19:50:27.102624    2404 engine.go:169] [/emulator/studio 1ca9] ExecuteGraphqlRead IntrospectionQuery: succeeded. \n"}}
[error] [Firebase Plugin] requireAuth error: Could not refresh access token: Unsuccessful response status code. Request failed with status code 500
[info] [Firebase Plugin] Stopping Data Connect toolkit
[warn] ⚠  Data Connect Emulator has exited upon receiving signal: SIGINT 
[debug] [2025-09-21T23:54:45.527Z] [Firebase Plugin] Activating Firebase extension.
[debug] [2025-09-21T23:55:32.444Z] [Firebase Plugin] requireAuthWrapper
[debug] [2025-09-21T23:55:32.453Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform","https://www.googleapis.com/auth/userinfo.email"]
[debug] [2025-09-21T23:55:32.514Z] [Firebase Plugin] VSCode notification server listening on port 40001
[info] [Firebase Plugin] Starting Data Connect toolkit (version 2.13.0) on port 50001
[debug] [2025-09-21T23:55:40.244Z] W0921 23:55:40.238629   21698 control.go:68] [/emulator/studio 5123] connector "example" has errors: example/queries.gql:2:3: mutation.InsertOrganization.organization_insert (data: Organization_Data): column createdAt is required but is missing in data

sources: example/queries.gql [530B] 
I0921 23:55:40.238761   21698 control.go:94] [/emulator/studio 5123] UpdateResources(): done
Schema: sources: schema/schema.gql [1207B] 
Connector "example": sources: example/queries.gql [530B] 
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"W0921 23:55:40.238629   21698 control.go:68] [/emulator/studio 5123] connector \"example\" has errors: example/queries.gql:2:3: mutation.InsertOrganization.organization_insert (data: Organization_Data): column createdAt is required but is missing in data\n\nsources: example/queries.gql [530B] \nI0921 23:55:40.238761   21698 control.go:94] [/emulator/studio 5123] UpdateResources(): done\nSchema: sources: schema/schema.gql [1207B] \nConnector \"example\": sources: example/queries.gql [530B] \n"}}
[debug] [2025-09-21T23:55:40.248Z] Ignoring unsupported arg: auto_download {"metadata":{"emulator":{"name":"dataconnect"},"message":"Ignoring unsupported arg: auto_download"}}
[debug] [2025-09-21T23:55:40.251Z] Ignoring unsupported arg: port {"metadata":{"emulator":{"name":"dataconnect"},"message":"Ignoring unsupported arg: port"}}
[debug] [2025-09-21T23:55:40.254Z] Starting Data Connect Emulator with command {"binary":"/home/user/.cache/firebase/emulators/dataconnect-emulator-2.13.0","args":["--logtostderr","-v=2","dev","--listen=localhost:50001","--config_dir=/home/user/studio/dataconnect","--enable_output_schema_extensions=true","--enable_output_generated_sdk=true"],"optionalArgs":["listen","config_dir","enable_output_schema_extensions","enable_output_generated_sdk"],"joinArgs":true,"shell":false,"port":9399} {"metadata":{"emulator":{"name":"dataconnect"},"message":"Starting Data Connect Emulator with command {\"binary\":\"/home/user/.cache/firebase/emulators/dataconnect-emulator-2.13.0\",\"args\":[\"--logtostderr\",\"-v=2\",\"dev\",\"--listen=localhost:50001\",\"--config_dir=/home/user/studio/dataconnect\",\"--enable_output_schema_extensions=true\",\"--enable_output_generated_sdk=true\"],\"optionalArgs\":[\"listen\",\"config_dir\",\"enable_output_schema_extensions\",\"enable_output_generated_sdk\"],\"joinArgs\":true,\"shell\":false,\"port\":9399}"}}
[info] i  dataconnect: Data Connect Emulator logging to dataconnect-debug.log {"metadata":{"emulator":{"name":"dataconnect"},"message":"Data Connect Emulator logging to dataconnect-debug.log"}}
[debug] [2025-09-21T23:55:40.522Z] I0921 23:55:40.435041   21703 load.go:31] Reloading schema and connectors...
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0921 23:55:40.435041   21703 load.go:31] Reloading schema and connectors...\n"}}
[debug] [2025-09-21T23:55:40.879Z] W0921 23:55:40.819449   21703 control.go:68] [/emulator/studio 536a] connector "example" has errors: example/queries.gql:2:3: mutation.InsertOrganization.organization_insert (data: Organization_Data): column createdAt is required but is missing in data

sources: example/queries.gql [530B] 
I0921 23:55:40.819534   21703 control.go:94] [/emulator/studio 536a] UpdateResources(): done
Schema: sources: schema/schema.gql [1207B] 
Connector "example": sources: example/queries.gql [530B] 
E0921 23:55:40.858878   21703 load.go:73] Could not load sources: 
Errors: example/queries.gql:2:3: mutation.InsertOrganization.organization_insert (data: Organization_Data): column createdAt is required but is missing in data
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"W0921 23:55:40.819449   21703 control.go:68] [/emulator/studio 536a] connector \"example\" has errors: example/queries.gql:2:3: mutation.InsertOrganization.organization_insert (data: Organization_Data): column createdAt is required but is missing in data\n\nsources: example/queries.gql [530B] \nI0921 23:55:40.819534   21703 control.go:94] [/emulator/studio 536a] UpdateResources(): done\nSchema: sources: schema/schema.gql [1207B] \nConnector \"example\": sources: example/queries.gql [530B] \nE0921 23:55:40.858878   21703 load.go:73] Could not load sources: \nErrors: example/queries.gql:2:3: mutation.InsertOrganization.organization_insert (data: Organization_Data): column createdAt is required but is missing in data\n"}}
[debug] [2025-09-21T23:55:40.949Z] I0921 23:55:40.883638   21703 collector.go:107] schema extensions wrote into "/home/user/studio/dataconnect/.dataconnect/schema"
Generated sources: main/relation.gql [33941B] main/query.gql [7604B] main/mutation.gql [20515B] prelude.gql [73251B] main/implicit.gql [3592B] main/input.gql [68752B] 
I0921 23:55:40.883716   21703 load.go:74] Finished reloading
I0921 23:55:40.927631   21703 dev.go:102] Listening on address (HTTP + gRPC): 127.0.0.1:50001
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0921 23:55:40.883638   21703 collector.go:107] schema extensions wrote into \"/home/user/studio/dataconnect/.dataconnect/schema\"\nGenerated sources: main/relation.gql [33941B] main/query.gql [7604B] main/mutation.gql [20515B] prelude.gql [73251B] main/implicit.gql [3592B] main/input.gql [68752B] \nI0921 23:55:40.883716   21703 load.go:74] Finished reloading\nI0921 23:55:40.927631   21703 dev.go:102] Listening on address (HTTP + gRPC): 127.0.0.1:50001\n"}}
[debug] [2025-09-21T23:55:41.641Z] I0921 23:55:41.565447   21703 engine.go:169] [/emulator/studio 536a] ExecuteGraphqlRead IntrospectionQuery: succeeded. 
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0921 23:55:41.565447   21703 engine.go:169] [/emulator/studio 536a] ExecuteGraphqlRead IntrospectionQuery: succeeded. \n"}}
[error] [Firebase Plugin] requireAuth error: Could not refresh access token: Unsuccessful response status code. Request failed with status code 500
[info] [Firebase Plugin] Stopping Data Connect toolkit
[warn] ⚠  Data Connect Emulator has exited upon receiving signal: SIGINT 
[debug] [2025-09-22T19:12:31.362Z] [Firebase Plugin] Activating Firebase extension.
[debug] [2025-09-22T19:13:11.448Z] [Firebase Plugin] requireAuthWrapper
[debug] [2025-09-22T19:13:11.450Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform","https://www.googleapis.com/auth/userinfo.email"]
[debug] [2025-09-22T19:13:11.549Z] [Firebase Plugin] VSCode notification server listening on port 40001
[info] [Firebase Plugin] Starting Data Connect toolkit (version 2.13.0) on port 50001
[debug] [2025-09-22T19:13:18.828Z] W0922 19:13:18.822510    3396 control.go:68] [/emulator/studio e40e] connector "example" has errors: example/queries.gql:2:3: mutation.InsertOrganization.organization_insert (data: Organization_Data): column createdAt is required but is missing in data

sources: example/queries.gql [530B] 
I0922 19:13:18.822817    3396 control.go:94] [/emulator/studio e40e] UpdateResources(): done
Schema: sources: schema/schema.gql [1207B] 
Connector "example": sources: example/queries.gql [530B] 
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"W0922 19:13:18.822510    3396 control.go:68] [/emulator/studio e40e] connector \"example\" has errors: example/queries.gql:2:3: mutation.InsertOrganization.organization_insert (data: Organization_Data): column createdAt is required but is missing in data\n\nsources: example/queries.gql [530B] \nI0922 19:13:18.822817    3396 control.go:94] [/emulator/studio e40e] UpdateResources(): done\nSchema: sources: schema/schema.gql [1207B] \nConnector \"example\": sources: example/queries.gql [530B] \n"}}
[debug] [2025-09-22T19:13:18.833Z] Ignoring unsupported arg: auto_download {"metadata":{"emulator":{"name":"dataconnect"},"message":"Ignoring unsupported arg: auto_download"}}
[debug] [2025-09-22T19:13:18.835Z] Ignoring unsupported arg: port {"metadata":{"emulator":{"name":"dataconnect"},"message":"Ignoring unsupported arg: port"}}
[debug] [2025-09-22T19:13:18.836Z] Starting Data Connect Emulator with command {"binary":"/home/user/.cache/firebase/emulators/dataconnect-emulator-2.13.0","args":["--logtostderr","-v=2","dev","--listen=localhost:50001","--config_dir=/home/user/studio/dataconnect","--enable_output_schema_extensions=true","--enable_output_generated_sdk=true"],"optionalArgs":["listen","config_dir","enable_output_schema_extensions","enable_output_generated_sdk"],"joinArgs":true,"shell":false,"port":9399} {"metadata":{"emulator":{"name":"dataconnect"},"message":"Starting Data Connect Emulator with command {\"binary\":\"/home/user/.cache/firebase/emulators/dataconnect-emulator-2.13.0\",\"args\":[\"--logtostderr\",\"-v=2\",\"dev\",\"--listen=localhost:50001\",\"--config_dir=/home/user/studio/dataconnect\",\"--enable_output_schema_extensions=true\",\"--enable_output_generated_sdk=true\"],\"optionalArgs\":[\"listen\",\"config_dir\",\"enable_output_schema_extensions\",\"enable_output_generated_sdk\"],\"joinArgs\":true,\"shell\":false,\"port\":9399}"}}
[info] i  dataconnect: Data Connect Emulator logging to dataconnect-debug.log {"metadata":{"emulator":{"name":"dataconnect"},"message":"Data Connect Emulator logging to dataconnect-debug.log"}}
[debug] [2025-09-22T19:13:19.287Z] I0922 19:13:19.065049    3401 load.go:31] Reloading schema and connectors...
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0922 19:13:19.065049    3401 load.go:31] Reloading schema and connectors...\n"}}
[debug] [2025-09-22T19:13:21.526Z] W0922 19:13:19.663873    3401 control.go:68] [/emulator/studio 1e39] connector "example" has errors: example/queries.gql:2:3: mutation.InsertOrganization.organization_insert (data: Organization_Data): column createdAt is required but is missing in data

sources: example/queries.gql [530B] 
I0922 19:13:19.663959    3401 control.go:94] [/emulator/studio 1e39] UpdateResources(): done
Schema: sources: schema/schema.gql [1207B] 
Connector "example": sources: example/queries.gql [530B] 
E0922 19:13:19.690562    3401 load.go:73] Could not load sources: 
Errors: example/queries.gql:2:3: mutation.InsertOrganization.organization_insert (data: Organization_Data): column createdAt is required but is missing in data
I0922 19:13:19.708113    3401 collector.go:107] schema extensions wrote into "/home/user/studio/dataconnect/.dataconnect/schema"
Generated sources: main/relation.gql [33941B] main/query.gql [7604B] main/mutation.gql [20515B] prelude.gql [73251B] main/implicit.gql [3592B] main/input.gql [68752B] 
I0922 19:13:19.712720    3401 load.go:74] Finished reloading
I0922 19:13:19.751154    3401 dev.go:102] Listening on address (HTTP + gRPC): 127.0.0.1:50001
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"W0922 19:13:19.663873    3401 control.go:68] [/emulator/studio 1e39] connector \"example\" has errors: example/queries.gql:2:3: mutation.InsertOrganization.organization_insert (data: Organization_Data): column createdAt is required but is missing in data\n\nsources: example/queries.gql [530B] \nI0922 19:13:19.663959    3401 control.go:94] [/emulator/studio 1e39] UpdateResources(): done\nSchema: sources: schema/schema.gql [1207B] \nConnector \"example\": sources: example/queries.gql [530B] \nE0922 19:13:19.690562    3401 load.go:73] Could not load sources: \nErrors: example/queries.gql:2:3: mutation.InsertOrganization.organization_insert (data: Organization_Data): column createdAt is required but is missing in data\nI0922 19:13:19.708113    3401 collector.go:107] schema extensions wrote into \"/home/user/studio/dataconnect/.dataconnect/schema\"\nGenerated sources: main/relation.gql [33941B] main/query.gql [7604B] main/mutation.gql [20515B] prelude.gql [73251B] main/implicit.gql [3592B] main/input.gql [68752B] \nI0922 19:13:19.712720    3401 load.go:74] Finished reloading\nI0922 19:13:19.751154    3401 dev.go:102] Listening on address (HTTP + gRPC): 127.0.0.1:50001\n"}}
[debug] [2025-09-22T19:13:23.646Z] I0922 19:13:22.879862    3401 engine.go:169] [/emulator/studio 1e39] ExecuteGraphqlRead IntrospectionQuery: succeeded. 
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0922 19:13:22.879862    3401 engine.go:169] [/emulator/studio 1e39] ExecuteGraphqlRead IntrospectionQuery: succeeded. \n"}}
[error] [Firebase Plugin] requireAuth error: Could not refresh access token: Unsuccessful response status code. Request failed with status code 500
[info] [Firebase Plugin] Stopping Data Connect toolkit
[warn] ⚠  Data Connect Emulator has exited upon receiving signal: SIGINT 
[debug] [2025-09-24T01:37:53.927Z] [Firebase Plugin] Activating Firebase extension.
[debug] [2025-09-24T01:38:09.304Z] [Firebase Plugin] requireAuthWrapper
[debug] [2025-09-24T01:38:09.306Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform","https://www.googleapis.com/auth/userinfo.email"]
[debug] [2025-09-24T01:38:09.326Z] [Firebase Plugin] VSCode notification server listening on port 40001
[info] [Firebase Plugin] Starting Data Connect toolkit (version 2.13.0) on port 50001
[debug] [2025-09-24T01:38:13.588Z] W0924 01:38:13.582728    5720 control.go:68] [/emulator/studio dbd6] connector "example" has errors: example/queries.gql:2:3: mutation.InsertOrganization.organization_insert (data: Organization_Data): column createdAt is required but is missing in data

sources: example/queries.gql [530B] 
I0924 01:38:13.584812    5720 control.go:94] [/emulator/studio dbd6] UpdateResources(): done
Schema: sources: schema/schema.gql [1207B] 
Connector "example": sources: example/queries.gql [530B] 
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"W0924 01:38:13.582728    5720 control.go:68] [/emulator/studio dbd6] connector \"example\" has errors: example/queries.gql:2:3: mutation.InsertOrganization.organization_insert (data: Organization_Data): column createdAt is required but is missing in data\n\nsources: example/queries.gql [530B] \nI0924 01:38:13.584812    5720 control.go:94] [/emulator/studio dbd6] UpdateResources(): done\nSchema: sources: schema/schema.gql [1207B] \nConnector \"example\": sources: example/queries.gql [530B] \n"}}
[debug] [2025-09-24T01:38:13.594Z] Ignoring unsupported arg: auto_download {"metadata":{"emulator":{"name":"dataconnect"},"message":"Ignoring unsupported arg: auto_download"}}
[debug] [2025-09-24T01:38:13.595Z] Ignoring unsupported arg: port {"metadata":{"emulator":{"name":"dataconnect"},"message":"Ignoring unsupported arg: port"}}
[debug] [2025-09-24T01:38:13.596Z] Starting Data Connect Emulator with command {"binary":"/home/user/.cache/firebase/emulators/dataconnect-emulator-2.13.0","args":["--logtostderr","-v=2","dev","--listen=localhost:50001","--config_dir=/home/user/studio/dataconnect","--enable_output_schema_extensions=true","--enable_output_generated_sdk=true"],"optionalArgs":["listen","config_dir","enable_output_schema_extensions","enable_output_generated_sdk"],"joinArgs":true,"shell":false,"port":9399} {"metadata":{"emulator":{"name":"dataconnect"},"message":"Starting Data Connect Emulator with command {\"binary\":\"/home/user/.cache/firebase/emulators/dataconnect-emulator-2.13.0\",\"args\":[\"--logtostderr\",\"-v=2\",\"dev\",\"--listen=localhost:50001\",\"--config_dir=/home/user/studio/dataconnect\",\"--enable_output_schema_extensions=true\",\"--enable_output_generated_sdk=true\"],\"optionalArgs\":[\"listen\",\"config_dir\",\"enable_output_schema_extensions\",\"enable_output_generated_sdk\"],\"joinArgs\":true,\"shell\":false,\"port\":9399}"}}
[info] i  dataconnect: Data Connect Emulator logging to dataconnect-debug.log {"metadata":{"emulator":{"name":"dataconnect"},"message":"Data Connect Emulator logging to dataconnect-debug.log"}}
[debug] [2025-09-24T01:38:13.986Z] I0924 01:38:13.753341    5725 load.go:31] Reloading schema and connectors...
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0924 01:38:13.753341    5725 load.go:31] Reloading schema and connectors...\n"}}
[debug] [2025-09-24T01:38:14.372Z] W0924 01:38:14.166131    5725 control.go:68] [/emulator/studio 006a] connector "example" has errors: example/queries.gql:2:3: mutation.InsertOrganization.organization_insert (data: Organization_Data): column createdAt is required but is missing in data

sources: example/queries.gql [530B] 
I0924 01:38:14.166206    5725 control.go:94] [/emulator/studio 006a] UpdateResources(): done
Schema: sources: schema/schema.gql [1207B] 
Connector "example": sources: example/queries.gql [530B] 
E0924 01:38:14.185696    5725 load.go:73] Could not load sources: 
Errors: example/queries.gql:2:3: mutation.InsertOrganization.organization_insert (data: Organization_Data): column createdAt is required but is missing in data
I0924 01:38:14.213141    5725 collector.go:107] schema extensions wrote into "/home/user/studio/dataconnect/.dataconnect/schema"
Generated sources: main/query.gql [7604B] main/mutation.gql [20515B] main/implicit.gql [3592B] prelude.gql [73251B] main/input.gql [68752B] main/relation.gql [33941B] 
I0924 01:38:14.213241    5725 load.go:74] Finished reloading
I0924 01:38:14.242059    5725 dev.go:102] Listening on address (HTTP + gRPC): 127.0.0.1:50001
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"W0924 01:38:14.166131    5725 control.go:68] [/emulator/studio 006a] connector \"example\" has errors: example/queries.gql:2:3: mutation.InsertOrganization.organization_insert (data: Organization_Data): column createdAt is required but is missing in data\n\nsources: example/queries.gql [530B] \nI0924 01:38:14.166206    5725 control.go:94] [/emulator/studio 006a] UpdateResources(): done\nSchema: sources: schema/schema.gql [1207B] \nConnector \"example\": sources: example/queries.gql [530B] \nE0924 01:38:14.185696    5725 load.go:73] Could not load sources: \nErrors: example/queries.gql:2:3: mutation.InsertOrganization.organization_insert (data: Organization_Data): column createdAt is required but is missing in data\nI0924 01:38:14.213141    5725 collector.go:107] schema extensions wrote into \"/home/user/studio/dataconnect/.dataconnect/schema\"\nGenerated sources: main/query.gql [7604B] main/mutation.gql [20515B] main/implicit.gql [3592B] prelude.gql [73251B] main/input.gql [68752B] main/relation.gql [33941B] \nI0924 01:38:14.213241    5725 load.go:74] Finished reloading\nI0924 01:38:14.242059    5725 dev.go:102] Listening on address (HTTP + gRPC): 127.0.0.1:50001\n"}}
[debug] [2025-09-24T01:38:15.752Z] I0924 01:38:15.242704    5725 engine.go:169] [/emulator/studio 006a] ExecuteGraphqlRead IntrospectionQuery: succeeded. 
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0924 01:38:15.242704    5725 engine.go:169] [/emulator/studio 006a] ExecuteGraphqlRead IntrospectionQuery: succeeded. \n"}}
[error] [Firebase Plugin] requireAuth error: Could not refresh access token: Unsuccessful response status code. Request failed with status code 500
[info] [Firebase Plugin] Stopping Data Connect toolkit
[warn] ⚠  Data Connect Emulator has exited upon receiving signal: SIGINT 
[debug] [2025-09-26T00:30:30.746Z] [Firebase Plugin] Activating Firebase extension.
[debug] [2025-09-26T00:30:54.455Z] [Firebase Plugin] requireAuthWrapper
[debug] [2025-09-26T00:30:54.458Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform","https://www.googleapis.com/auth/userinfo.email"]
[debug] [2025-09-26T00:30:54.481Z] [Firebase Plugin] VSCode notification server listening on port 40001
[info] [Firebase Plugin] Starting Data Connect toolkit (version 2.13.0) on port 50001
[debug] [2025-09-26T00:30:57.167Z] W0926 00:30:57.163307     517 control.go:68] [/emulator/studio 7599] connector "example" has errors: example/queries.gql:2:3: mutation.InsertOrganization.organization_insert (data: Organization_Data): column createdAt is required but is missing in data

sources: example/queries.gql [530B] 
I0926 00:30:57.163445     517 control.go:94] [/emulator/studio 7599] UpdateResources(): done
Schema: sources: schema/schema.gql [1207B] 
Connector "example": sources: example/queries.gql [530B] 
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"W0926 00:30:57.163307     517 control.go:68] [/emulator/studio 7599] connector \"example\" has errors: example/queries.gql:2:3: mutation.InsertOrganization.organization_insert (data: Organization_Data): column createdAt is required but is missing in data\n\nsources: example/queries.gql [530B] \nI0926 00:30:57.163445     517 control.go:94] [/emulator/studio 7599] UpdateResources(): done\nSchema: sources: schema/schema.gql [1207B] \nConnector \"example\": sources: example/queries.gql [530B] \n"}}
[debug] [2025-09-26T00:30:57.171Z] Ignoring unsupported arg: auto_download {"metadata":{"emulator":{"name":"dataconnect"},"message":"Ignoring unsupported arg: auto_download"}}
[debug] [2025-09-26T00:30:57.172Z] Ignoring unsupported arg: port {"metadata":{"emulator":{"name":"dataconnect"},"message":"Ignoring unsupported arg: port"}}
[debug] [2025-09-26T00:30:57.173Z] Starting Data Connect Emulator with command {"binary":"/home/user/.cache/firebase/emulators/dataconnect-emulator-2.13.0","args":["--logtostderr","-v=2","dev","--listen=localhost:50001","--config_dir=/home/user/studio/dataconnect","--enable_output_schema_extensions=true","--enable_output_generated_sdk=true"],"optionalArgs":["listen","config_dir","enable_output_schema_extensions","enable_output_generated_sdk"],"joinArgs":true,"shell":false,"port":9399} {"metadata":{"emulator":{"name":"dataconnect"},"message":"Starting Data Connect Emulator with command {\"binary\":\"/home/user/.cache/firebase/emulators/dataconnect-emulator-2.13.0\",\"args\":[\"--logtostderr\",\"-v=2\",\"dev\",\"--listen=localhost:50001\",\"--config_dir=/home/user/studio/dataconnect\",\"--enable_output_schema_extensions=true\",\"--enable_output_generated_sdk=true\"],\"optionalArgs\":[\"listen\",\"config_dir\",\"enable_output_schema_extensions\",\"enable_output_generated_sdk\"],\"joinArgs\":true,\"shell\":false,\"port\":9399}"}}
[info] i  dataconnect: Data Connect Emulator logging to dataconnect-debug.log {"metadata":{"emulator":{"name":"dataconnect"},"message":"Data Connect Emulator logging to dataconnect-debug.log"}}
[debug] [2025-09-26T00:30:57.344Z] I0926 00:30:57.222741     523 load.go:31] Reloading schema and connectors...
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0926 00:30:57.222741     523 load.go:31] Reloading schema and connectors...\n"}}
[debug] [2025-09-26T00:30:57.659Z] W0926 00:30:57.495645     523 control.go:68] [/emulator/studio 03ad] connector "example" has errors: example/queries.gql:2:3: mutation.InsertOrganization.organization_insert (data: Organization_Data): column createdAt is required but is missing in data

sources: example/queries.gql [530B] 
I0926 00:30:57.495722     523 control.go:94] [/emulator/studio 03ad] UpdateResources(): done
Schema: sources: schema/schema.gql [1207B] 
Connector "example": sources: example/queries.gql [530B] 
E0926 00:30:57.511307     523 load.go:73] Could not load sources: 
Errors: example/queries.gql:2:3: mutation.InsertOrganization.organization_insert (data: Organization_Data): column createdAt is required but is missing in data
I0926 00:30:57.515910     523 collector.go:107] schema extensions wrote into "/home/user/studio/dataconnect/.dataconnect/schema"
Generated sources: prelude.gql [73251B] main/implicit.gql [3592B] main/input.gql [68752B] main/relation.gql [33941B] main/query.gql [7604B] main/mutation.gql [20515B] 
I0926 00:30:57.515999     523 load.go:74] Finished reloading
I0926 00:30:57.532291     523 dev.go:102] Listening on address (HTTP + gRPC): 127.0.0.1:50001
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"W0926 00:30:57.495645     523 control.go:68] [/emulator/studio 03ad] connector \"example\" has errors: example/queries.gql:2:3: mutation.InsertOrganization.organization_insert (data: Organization_Data): column createdAt is required but is missing in data\n\nsources: example/queries.gql [530B] \nI0926 00:30:57.495722     523 control.go:94] [/emulator/studio 03ad] UpdateResources(): done\nSchema: sources: schema/schema.gql [1207B] \nConnector \"example\": sources: example/queries.gql [530B] \nE0926 00:30:57.511307     523 load.go:73] Could not load sources: \nErrors: example/queries.gql:2:3: mutation.InsertOrganization.organization_insert (data: Organization_Data): column createdAt is required but is missing in data\nI0926 00:30:57.515910     523 collector.go:107] schema extensions wrote into \"/home/user/studio/dataconnect/.dataconnect/schema\"\nGenerated sources: prelude.gql [73251B] main/implicit.gql [3592B] main/input.gql [68752B] main/relation.gql [33941B] main/query.gql [7604B] main/mutation.gql [20515B] \nI0926 00:30:57.515999     523 load.go:74] Finished reloading\nI0926 00:30:57.532291     523 dev.go:102] Listening on address (HTTP + gRPC): 127.0.0.1:50001\n"}}
[debug] [2025-09-26T00:30:59.394Z] I0926 00:30:58.525987     523 engine.go:169] [/emulator/studio 03ad] ExecuteGraphqlRead IntrospectionQuery: succeeded. 
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0926 00:30:58.525987     523 engine.go:169] [/emulator/studio 03ad] ExecuteGraphqlRead IntrospectionQuery: succeeded. \n"}}
[debug] [2025-09-26T00:31:27.079Z] Running auto auth
[debug] [2025-09-26T00:31:27.273Z] No OAuth tokens found
[debug] [2025-09-26T00:31:27.280Z] [Firebase Plugin] User found:  innovativeenterprisesoman@gmail.com
[info] [Firebase Plugin] (Core:Project) New user detected, fetching projects
[debug] [2025-09-26T00:31:27.290Z] No OAuth tokens found
[debug] [2025-09-26T00:31:27.312Z] >>> [apiv2][query] GET https://firebase.googleapis.com/v1beta1/projects pageSize=1000
[debug] [2025-09-26T00:31:28.250Z] <<< [apiv2][status] GET https://firebase.googleapis.com/v1beta1/projects 200
[debug] [2025-09-26T00:31:28.252Z] <<< [apiv2][body] GET https://firebase.googleapis.com/v1beta1/projects [omitted]
[debug] [2025-09-26T00:40:17.517Z] I0926 00:40:17.516944     523 load.go:31] Reloading schema and connectors...
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0926 00:40:17.516944     523 load.go:31] Reloading schema and connectors...\n"}}
[debug] [2025-09-26T00:40:17.567Z] W0926 00:40:17.566905     523 control.go:68] [/emulator/studio 03ad] connector "example" has errors: example/queries.gql:5:5: Cannot query field "id" on type "Organization_KeyOutput".
example/queries.gql:2:3: Field "organization_insert" must not have a selection since type "Organization_KeyOutput" has no subfields.
example/queries.gql:12:30: Field "last_updated" is not defined by type "Task_Data".
example/queries.gql:14:5: Cannot query field "id" on type "Task_KeyOutput".
example/queries.gql:10:3: Field "task_update" must not have a selection since type "Task_KeyOutput" has no subfields.

sources: example/queries.gql [505B] 
I0926 00:40:17.566993     523 control.go:94] [/emulator/studio 03ad] UpdateResources(): done
Connector "example": sources: example/queries.gql [505B] 
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"W0926 00:40:17.566905     523 control.go:68] [/emulator/studio 03ad] connector \"example\" has errors: example/queries.gql:5:5: Cannot query field \"id\" on type \"Organization_KeyOutput\".\nexample/queries.gql:2:3: Field \"organization_insert\" must not have a selection since type \"Organization_KeyOutput\" has no subfields.\nexample/queries.gql:12:30: Field \"last_updated\" is not defined by type \"Task_Data\".\nexample/queries.gql:14:5: Cannot query field \"id\" on type \"Task_KeyOutput\".\nexample/queries.gql:10:3: Field \"task_update\" must not have a selection since type \"Task_KeyOutput\" has no subfields.\n\nsources: example/queries.gql [505B] \nI0926 00:40:17.566993     523 control.go:94] [/emulator/studio 03ad] UpdateResources(): done\nConnector \"example\": sources: example/queries.gql [505B] \n"}}
[debug] [2025-09-26T00:40:17.574Z] E0926 00:40:17.572913     523 load.go:73] Could not load sources: 
Errors: example/queries.gql:5:5: Cannot query field "id" on type "Organization_KeyOutput".
example/queries.gql:2:3: Field "organization_insert" must not have a selection since type "Organization_KeyOutput" has no subfields.
example/queries.gql:12:30: Field "last_updated" is not defined by type "Task_Data".
example/queries.gql:14:5: Cannot query field "id" on type "Task_KeyOutput".
example/queries.gql:10:3: Field "task_update" must not have a selection since type "Task_KeyOutput" has no subfields.
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0926 00:40:17.572913     523 load.go:73] Could not load sources: \nErrors: example/queries.gql:5:5: Cannot query field \"id\" on type \"Organization_KeyOutput\".\nexample/queries.gql:2:3: Field \"organization_insert\" must not have a selection since type \"Organization_KeyOutput\" has no subfields.\nexample/queries.gql:12:30: Field \"last_updated\" is not defined by type \"Task_Data\".\nexample/queries.gql:14:5: Cannot query field \"id\" on type \"Task_KeyOutput\".\nexample/queries.gql:10:3: Field \"task_update\" must not have a selection since type \"Task_KeyOutput\" has no subfields.\n"}}
[debug] [2025-09-26T00:40:17.576Z] I0926 00:40:17.576365     523 collector.go:107] schema extensions wrote into "/home/user/studio/dataconnect/.dataconnect/schema"
Generated sources: main/relation.gql [33941B] main/query.gql [7604B] main/mutation.gql [20515B] prelude.gql [73251B] main/implicit.gql [3592B] main/input.gql [68752B] 
I0926 00:40:17.576409     523 load.go:74] Finished reloading
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0926 00:40:17.576365     523 collector.go:107] schema extensions wrote into \"/home/user/studio/dataconnect/.dataconnect/schema\"\nGenerated sources: main/relation.gql [33941B] main/query.gql [7604B] main/mutation.gql [20515B] prelude.gql [73251B] main/implicit.gql [3592B] main/input.gql [68752B] \nI0926 00:40:17.576409     523 load.go:74] Finished reloading\n"}}
[debug] [2025-09-26T00:41:57.517Z] I0926 00:41:57.517453     523 load.go:31] Reloading schema and connectors...
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0926 00:41:57.517453     523 load.go:31] Reloading schema and connectors...\n"}}
[debug] [2025-09-26T00:41:57.565Z] W0926 00:41:57.565506     523 control.go:68] [/emulator/studio 03ad] connector "example" has errors: example/queries.gql:3:45: Expected Name, found <Invalid>

sources: example/queries.gql [493B] 
I0926 00:41:57.565597     523 control.go:94] [/emulator/studio 03ad] UpdateResources(): done
Connector "example": sources: example/queries.gql [493B] 
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"W0926 00:41:57.565506     523 control.go:68] [/emulator/studio 03ad] connector \"example\" has errors: example/queries.gql:3:45: Expected Name, found <Invalid>\n\nsources: example/queries.gql [493B] \nI0926 00:41:57.565597     523 control.go:94] [/emulator/studio 03ad] UpdateResources(): done\nConnector \"example\": sources: example/queries.gql [493B] \n"}}
[debug] [2025-09-26T00:41:57.605Z] E0926 00:41:57.605388     523 load.go:73] Could not load sources: 
Errors: example/queries.gql:3:45: Expected Name, found <Invalid>
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0926 00:41:57.605388     523 load.go:73] Could not load sources: \nErrors: example/queries.gql:3:45: Expected Name, found <Invalid>\n"}}
[debug] [2025-09-26T00:41:57.613Z] I0926 00:41:57.611138     523 collector.go:107] schema extensions wrote into "/home/user/studio/dataconnect/.dataconnect/schema"
Generated sources: main/mutation.gql [20515B] prelude.gql [73251B] main/implicit.gql [3592B] main/input.gql [68752B] main/relation.gql [33941B] main/query.gql [7604B] 
I0926 00:41:57.612001     523 load.go:74] Finished reloading
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0926 00:41:57.611138     523 collector.go:107] schema extensions wrote into \"/home/user/studio/dataconnect/.dataconnect/schema\"\nGenerated sources: main/mutation.gql [20515B] prelude.gql [73251B] main/implicit.gql [3592B] main/input.gql [68752B] main/relation.gql [33941B] main/query.gql [7604B] \nI0926 00:41:57.612001     523 load.go:74] Finished reloading\n"}}
[debug] [2025-09-26T00:42:57.518Z] I0926 00:42:57.517744     523 load.go:31] Reloading schema and connectors...
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0926 00:42:57.517744     523 load.go:31] Reloading schema and connectors...\n"}}
[debug] [2025-09-26T00:42:57.643Z] W0926 00:42:57.628237     523 control.go:68] [/emulator/studio 03ad] connector "example" has errors: example/queries.gql:10:36: Field "_eq" is not defined by type "UUID_Filter". Did you mean "eq" or "ne"?
example/queries.gql:26:30: Field "updatedAt" is not defined by type "Task_Data". Did you mean "createdAt" or "dueDate_date"?

sources: example/queries.gql [477B] 
I0926 00:42:57.628356     523 control.go:94] [/emulator/studio 03ad] UpdateResources(): done
Connector "example": sources: example/queries.gql [477B] 
E0926 00:42:57.639888     523 load.go:73] Could not load sources: 
Errors: example/queries.gql:10:36: Field "_eq" is not defined by type "UUID_Filter". Did you mean "eq" or "ne"?
example/queries.gql:26:30: Field "updatedAt" is not defined by type "Task_Data". Did you mean "createdAt" or "dueDate_date"?
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"W0926 00:42:57.628237     523 control.go:68] [/emulator/studio 03ad] connector \"example\" has errors: example/queries.gql:10:36: Field \"_eq\" is not defined by type \"UUID_Filter\". Did you mean \"eq\" or \"ne\"?\nexample/queries.gql:26:30: Field \"updatedAt\" is not defined by type \"Task_Data\". Did you mean \"createdAt\" or \"dueDate_date\"?\n\nsources: example/queries.gql [477B] \nI0926 00:42:57.628356     523 control.go:94] [/emulator/studio 03ad] UpdateResources(): done\nConnector \"example\": sources: example/queries.gql [477B] \nE0926 00:42:57.639888     523 load.go:73] Could not load sources: \nErrors: example/queries.gql:10:36: Field \"_eq\" is not defined by type \"UUID_Filter\". Did you mean \"eq\" or \"ne\"?\nexample/queries.gql:26:30: Field \"updatedAt\" is not defined by type \"Task_Data\". Did you mean \"createdAt\" or \"dueDate_date\"?\n"}}
[debug] [2025-09-26T00:42:57.659Z] I0926 00:42:57.658794     523 collector.go:107] schema extensions wrote into "/home/user/studio/dataconnect/.dataconnect/schema"
Generated sources: main/input.gql [68752B] main/relation.gql [33941B] main/query.gql [7604B] main/mutation.gql [20515B] prelude.gql [73251B] main/implicit.gql [3592B] 
I0926 00:42:57.658844     523 load.go:74] Finished reloading
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0926 00:42:57.658794     523 collector.go:107] schema extensions wrote into \"/home/user/studio/dataconnect/.dataconnect/schema\"\nGenerated sources: main/input.gql [68752B] main/relation.gql [33941B] main/query.gql [7604B] main/mutation.gql [20515B] prelude.gql [73251B] main/implicit.gql [3592B] \nI0926 00:42:57.658844     523 load.go:74] Finished reloading\n"}}
[debug] [2025-09-26T00:45:37.528Z] I0926 00:45:37.525960     523 load.go:31] Reloading schema and connectors...
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0926 00:45:37.525960     523 load.go:31] Reloading schema and connectors...\n"}}
[debug] [2025-09-26T00:45:37.612Z] W0926 00:45:37.611877     523 control.go:68] [/emulator/studio 03ad] connector "example" has errors: example/queries.gql:5:5: Cannot query field "id" on type "Organization_KeyOutput".
example/queries.gql:2:3: Field "organization_insert" must not have a selection since type "Organization_KeyOutput" has no subfields.
example/queries.gql:10:14: Field "organization_id" is not defined by type "User_Filter". Did you mean "organizationId" or "organization"?
example/queries.gql:30:5: Cannot query field "id" on type "Task_KeyOutput".
example/queries.gql:26:3: Unknown argument "where" on field "Mutation.task_update".
example/queries.gql:26:3: Field "task_update" must not have a selection since type "Task_KeyOutput" has no subfields.

sources: example/queries.gql [533B] 
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"W0926 00:45:37.611877     523 control.go:68] [/emulator/studio 03ad] connector \"example\" has errors: example/queries.gql:5:5: Cannot query field \"id\" on type \"Organization_KeyOutput\".\nexample/queries.gql:2:3: Field \"organization_insert\" must not have a selection since type \"Organization_KeyOutput\" has no subfields.\nexample/queries.gql:10:14: Field \"organization_id\" is not defined by type \"User_Filter\". Did you mean \"organizationId\" or \"organization\"?\nexample/queries.gql:30:5: Cannot query field \"id\" on type \"Task_KeyOutput\".\nexample/queries.gql:26:3: Unknown argument \"where\" on field \"Mutation.task_update\".\nexample/queries.gql:26:3: Field \"task_update\" must not have a selection since type \"Task_KeyOutput\" has no subfields.\n\nsources: example/queries.gql [533B] \n"}}
[debug] [2025-09-26T00:45:37.618Z] I0926 00:45:37.613081     523 control.go:94] [/emulator/studio 03ad] UpdateResources(): done
Connector "example": sources: example/queries.gql [533B] 
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0926 00:45:37.613081     523 control.go:94] [/emulator/studio 03ad] UpdateResources(): done\nConnector \"example\": sources: example/queries.gql [533B] \n"}}
[debug] [2025-09-26T00:45:37.624Z] E0926 00:45:37.622927     523 load.go:73] Could not load sources: 
Errors: example/queries.gql:5:5: Cannot query field "id" on type "Organization_KeyOutput".
example/queries.gql:2:3: Field "organization_insert" must not have a selection since type "Organization_KeyOutput" has no subfields.
example/queries.gql:10:14: Field "organization_id" is not defined by type "User_Filter". Did you mean "organizationId" or "organization"?
example/queries.gql:30:5: Cannot query field "id" on type "Task_KeyOutput".
example/queries.gql:26:3: Unknown argument "where" on field "Mutation.task_update".
example/queries.gql:26:3: Field "task_update" must not have a selection since type "Task_KeyOutput" has no subfields.
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0926 00:45:37.622927     523 load.go:73] Could not load sources: \nErrors: example/queries.gql:5:5: Cannot query field \"id\" on type \"Organization_KeyOutput\".\nexample/queries.gql:2:3: Field \"organization_insert\" must not have a selection since type \"Organization_KeyOutput\" has no subfields.\nexample/queries.gql:10:14: Field \"organization_id\" is not defined by type \"User_Filter\". Did you mean \"organizationId\" or \"organization\"?\nexample/queries.gql:30:5: Cannot query field \"id\" on type \"Task_KeyOutput\".\nexample/queries.gql:26:3: Unknown argument \"where\" on field \"Mutation.task_update\".\nexample/queries.gql:26:3: Field \"task_update\" must not have a selection since type \"Task_KeyOutput\" has no subfields.\n"}}
[debug] [2025-09-26T00:45:37.632Z] I0926 00:45:37.630664     523 collector.go:107] schema extensions wrote into "/home/user/studio/dataconnect/.dataconnect/schema"
Generated sources: prelude.gql [73251B] main/query.gql [7604B] main/mutation.gql [20515B] main/implicit.gql [3592B] main/input.gql [68752B] main/relation.gql [33941B] 
I0926 00:45:37.630715     523 load.go:74] Finished reloading
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0926 00:45:37.630664     523 collector.go:107] schema extensions wrote into \"/home/user/studio/dataconnect/.dataconnect/schema\"\nGenerated sources: prelude.gql [73251B] main/query.gql [7604B] main/mutation.gql [20515B] main/implicit.gql [3592B] main/input.gql [68752B] main/relation.gql [33941B] \nI0926 00:45:37.630715     523 load.go:74] Finished reloading\n"}}
[debug] [2025-09-26T00:48:47.548Z] I0926 00:48:47.547579     523 load.go:31] Reloading schema and connectors...
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0926 00:48:47.547579     523 load.go:31] Reloading schema and connectors...\n"}}
[debug] [2025-09-26T00:48:47.738Z] W0926 00:48:47.729593     523 control.go:68] [/emulator/studio 03ad] connector "example" has errors: example/queries.gql:14:3: Unknown argument "where" on field "Mutation.task_update".

sources: example/queries.gql [455B] 
I0926 00:48:47.729711     523 control.go:94] [/emulator/studio 03ad] UpdateResources(): done
Connector "example": sources: example/queries.gql [455B] 
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"W0926 00:48:47.729593     523 control.go:68] [/emulator/studio 03ad] connector \"example\" has errors: example/queries.gql:14:3: Unknown argument \"where\" on field \"Mutation.task_update\".\n\nsources: example/queries.gql [455B] \nI0926 00:48:47.729711     523 control.go:94] [/emulator/studio 03ad] UpdateResources(): done\nConnector \"example\": sources: example/queries.gql [455B] \n"}}
[debug] [2025-09-26T00:48:47.754Z] E0926 00:48:47.745006     523 load.go:73] Could not load sources: 
Errors: example/queries.gql:14:3: Unknown argument "where" on field "Mutation.task_update".
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0926 00:48:47.745006     523 load.go:73] Could not load sources: \nErrors: example/queries.gql:14:3: Unknown argument \"where\" on field \"Mutation.task_update\".\n"}}
[debug] [2025-09-26T00:48:47.765Z] I0926 00:48:47.760478     523 collector.go:107] schema extensions wrote into "/home/user/studio/dataconnect/.dataconnect/schema"
Generated sources: main/implicit.gql [3592B] main/input.gql [68752B] main/relation.gql [33941B] main/query.gql [7604B] main/mutation.gql [20515B] prelude.gql [73251B] 
I0926 00:48:47.760818     523 load.go:74] Finished reloading
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0926 00:48:47.760478     523 collector.go:107] schema extensions wrote into \"/home/user/studio/dataconnect/.dataconnect/schema\"\nGenerated sources: main/implicit.gql [3592B] main/input.gql [68752B] main/relation.gql [33941B] main/query.gql [7604B] main/mutation.gql [20515B] prelude.gql [73251B] \nI0926 00:48:47.760818     523 load.go:74] Finished reloading\n"}}
[debug] [2025-09-26T00:49:37.527Z] I0926 00:49:37.526877     523 load.go:31] Reloading schema and connectors...
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0926 00:49:37.526877     523 load.go:31] Reloading schema and connectors...\n"}}
[debug] [2025-09-26T00:49:37.632Z] W0926 00:49:37.626996     523 control.go:68] [/emulator/studio 03ad] connector "example" has errors: example/queries.gql:2:3: mutation.InsertOrganization.organization_insert (data.createdAt: Timestamp): parsing time "now()" as "2006-01-02T15:04:05.999999999Z07:00": cannot parse "now()" as "2006"

sources: example/queries.gql [447B] 
I0926 00:49:37.627087     523 control.go:94] [/emulator/studio 03ad] UpdateResources(): done
Connector "example": sources: example/queries.gql [447B] 
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"W0926 00:49:37.626996     523 control.go:68] [/emulator/studio 03ad] connector \"example\" has errors: example/queries.gql:2:3: mutation.InsertOrganization.organization_insert (data.createdAt: Timestamp): parsing time \"now()\" as \"2006-01-02T15:04:05.999999999Z07:00\": cannot parse \"now()\" as \"2006\"\n\nsources: example/queries.gql [447B] \nI0926 00:49:37.627087     523 control.go:94] [/emulator/studio 03ad] UpdateResources(): done\nConnector \"example\": sources: example/queries.gql [447B] \n"}}
[debug] [2025-09-26T00:49:37.640Z] E0926 00:49:37.636886     523 load.go:73] Could not load sources: 
Errors: example/queries.gql:2:3: mutation.InsertOrganization.organization_insert (data.createdAt: Timestamp): parsing time "now()" as "2006-01-02T15:04:05.999999999Z07:00": cannot parse "now()" as "2006"
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0926 00:49:37.636886     523 load.go:73] Could not load sources: \nErrors: example/queries.gql:2:3: mutation.InsertOrganization.organization_insert (data.createdAt: Timestamp): parsing time \"now()\" as \"2006-01-02T15:04:05.999999999Z07:00\": cannot parse \"now()\" as \"2006\"\n"}}
[debug] [2025-09-26T00:49:37.648Z] I0926 00:49:37.648422     523 collector.go:107] schema extensions wrote into "/home/user/studio/dataconnect/.dataconnect/schema"
Generated sources: prelude.gql [73251B] main/query.gql [7604B] main/mutation.gql [20515B] main/implicit.gql [3592B] main/input.gql [68752B] main/relation.gql [33941B] 
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0926 00:49:37.648422     523 collector.go:107] schema extensions wrote into \"/home/user/studio/dataconnect/.dataconnect/schema\"\nGenerated sources: prelude.gql [73251B] main/query.gql [7604B] main/mutation.gql [20515B] main/implicit.gql [3592B] main/input.gql [68752B] main/relation.gql [33941B] \n"}}
[debug] [2025-09-26T00:49:37.653Z] I0926 00:49:37.651230     523 load.go:74] Finished reloading
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0926 00:49:37.651230     523 load.go:74] Finished reloading\n"}}
[debug] [2025-09-26T00:50:47.525Z] I0926 00:50:47.524764     523 load.go:31] Reloading schema and connectors...
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0926 00:50:47.524764     523 load.go:31] Reloading schema and connectors...\n"}}
[debug] [2025-09-26T00:50:47.599Z] W0926 00:50:47.599444     523 control.go:68] [/emulator/studio 03ad] connector "example" has errors: example/queries.gql:3:45: Expected Name, found (

sources: example/queries.gql [456B] 
I0926 00:50:47.599513     523 control.go:94] [/emulator/studio 03ad] UpdateResources(): done
Connector "example": sources: example/queries.gql [456B] 
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"W0926 00:50:47.599444     523 control.go:68] [/emulator/studio 03ad] connector \"example\" has errors: example/queries.gql:3:45: Expected Name, found (\n\nsources: example/queries.gql [456B] \nI0926 00:50:47.599513     523 control.go:94] [/emulator/studio 03ad] UpdateResources(): done\nConnector \"example\": sources: example/queries.gql [456B] \n"}}
[debug] [2025-09-26T00:50:47.613Z] E0926 00:50:47.610454     523 load.go:73] Could not load sources: 
Errors: example/queries.gql:3:45: Expected Name, found (
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0926 00:50:47.610454     523 load.go:73] Could not load sources: \nErrors: example/queries.gql:3:45: Expected Name, found (\n"}}
[debug] [2025-09-26T00:50:47.621Z] I0926 00:50:47.621539     523 collector.go:107] schema extensions wrote into "/home/user/studio/dataconnect/.dataconnect/schema"
Generated sources: main/query.gql [7604B] main/mutation.gql [20515B] main/implicit.gql [3592B] main/input.gql [68752B] main/relation.gql [33941B] prelude.gql [73251B] 
I0926 00:50:47.621590     523 load.go:74] Finished reloading
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0926 00:50:47.621539     523 collector.go:107] schema extensions wrote into \"/home/user/studio/dataconnect/.dataconnect/schema\"\nGenerated sources: main/query.gql [7604B] main/mutation.gql [20515B] main/implicit.gql [3592B] main/input.gql [68752B] main/relation.gql [33941B] prelude.gql [73251B] \nI0926 00:50:47.621590     523 load.go:74] Finished reloading\n"}}
[info] [Firebase Plugin] Stopping Data Connect toolkit
[warn] ⚠  Data Connect Emulator has exited upon receiving signal: SIGINT 
[debug] [2025-09-26T00:52:09.393Z] [Firebase Plugin] Activating Firebase extension.
[debug] [2025-09-26T00:52:21.412Z] [Firebase Plugin] requireAuthWrapper
[debug] [2025-09-26T00:52:21.414Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform","https://www.googleapis.com/auth/userinfo.email"]
[debug] [2025-09-26T00:52:21.415Z] > authorizing via signed-in user (innovativeenterprisesoman@gmail.com)
[debug] [2025-09-26T00:52:21.422Z] No OAuth tokens found
[debug] [2025-09-26T00:52:21.423Z] No OAuth tokens found
[debug] [2025-09-26T00:52:21.424Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform","https://www.googleapis.com/auth/userinfo.email"]
[debug] [2025-09-26T00:52:21.425Z] > authorizing via signed-in user (innovativeenterprisesoman@gmail.com)
[debug] [2025-09-26T00:52:21.426Z] [Firebase Plugin] User found:  innovativeenterprisesoman@gmail.com
[info] [Firebase Plugin] (Core:Project) New user detected, fetching projects
[debug] [2025-09-26T00:52:21.429Z] No OAuth tokens found
[debug] [2025-09-26T00:52:21.432Z] >>> [apiv2][query] GET https://firebase.googleapis.com/v1beta1/projects pageSize=1000
[debug] [2025-09-26T00:52:21.488Z] [Firebase Plugin] VSCode notification server listening on port 40001
[info] [Firebase Plugin] Starting Data Connect toolkit (version 2.13.0) on port 50001
[debug] [2025-09-26T00:52:25.815Z] I0926 00:52:25.529689    5988 control.go:94] [/emulator/studio a98d] UpdateResources(): done
Schema: sources: schema/schema.gql [1207B] 
Connector "example": sources: example/queries.gql [340B] 
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0926 00:52:25.529689    5988 control.go:94] [/emulator/studio a98d] UpdateResources(): done\nSchema: sources: schema/schema.gql [1207B] \nConnector \"example\": sources: example/queries.gql [340B] \n"}}
[debug] [2025-09-26T00:52:25.820Z] Ignoring unsupported arg: auto_download {"metadata":{"emulator":{"name":"dataconnect"},"message":"Ignoring unsupported arg: auto_download"}}
[debug] [2025-09-26T00:52:25.825Z] Ignoring unsupported arg: port {"metadata":{"emulator":{"name":"dataconnect"},"message":"Ignoring unsupported arg: port"}}
[debug] [2025-09-26T00:52:25.825Z] Starting Data Connect Emulator with command {"binary":"/home/user/.cache/firebase/emulators/dataconnect-emulator-2.13.0","args":["--logtostderr","-v=2","dev","--listen=localhost:50001","--config_dir=/home/user/studio/dataconnect","--enable_output_schema_extensions=true","--enable_output_generated_sdk=true"],"optionalArgs":["listen","config_dir","enable_output_schema_extensions","enable_output_generated_sdk"],"joinArgs":true,"shell":false,"port":9399} {"metadata":{"emulator":{"name":"dataconnect"},"message":"Starting Data Connect Emulator with command {\"binary\":\"/home/user/.cache/firebase/emulators/dataconnect-emulator-2.13.0\",\"args\":[\"--logtostderr\",\"-v=2\",\"dev\",\"--listen=localhost:50001\",\"--config_dir=/home/user/studio/dataconnect\",\"--enable_output_schema_extensions=true\",\"--enable_output_generated_sdk=true\"],\"optionalArgs\":[\"listen\",\"config_dir\",\"enable_output_schema_extensions\",\"enable_output_generated_sdk\"],\"joinArgs\":true,\"shell\":false,\"port\":9399}"}}
[info] i  dataconnect: Data Connect Emulator logging to dataconnect-debug.log {"metadata":{"emulator":{"name":"dataconnect"},"message":"Data Connect Emulator logging to dataconnect-debug.log"}}
[debug] [2025-09-26T00:52:26.091Z] I0926 00:52:25.989612    5993 load.go:31] Reloading schema and connectors...
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0926 00:52:25.989612    5993 load.go:31] Reloading schema and connectors...\n"}}
[debug] [2025-09-26T00:52:26.414Z] I0926 00:52:26.396012    5993 control.go:94] [/emulator/studio e9e9] UpdateResources(): done
Schema: sources: schema/schema.gql [1207B] 
Connector "example": sources: example/queries.gql [340B] 
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0926 00:52:26.396012    5993 control.go:94] [/emulator/studio e9e9] UpdateResources(): done\nSchema: sources: schema/schema.gql [1207B] \nConnector \"example\": sources: example/queries.gql [340B] \n"}}
[debug] [2025-09-26T00:52:28.073Z] I0926 00:52:26.974234    5993 codegen.go:82] [connector "example" javascriptSdk] Generating sources into "/home/user/studio/src/dataconnect-generated"
I0926 00:52:27.106097    5993 collector.go:107] schema extensions wrote into "/home/user/studio/dataconnect/.dataconnect/schema"
Generated sources: main/mutation.gql [20515B] main/implicit.gql [3592B] main/input.gql [68752B] main/relation.gql [33941B] prelude.gql [73251B] main/query.gql [7604B] 
I0926 00:52:27.126727    5993 collector.go:107] connector "example" javascriptSdk wrote into "/home/user/studio/src/dataconnect-generated"
Generated sources: esm/package.json [17B] index.cjs.js [1629B] react/index.cjs.js [1379B] package.json [863B] README.md [17362B] react/index.d.ts [1699B] react/package.json [460B] react/esm/package.json [17B] react/esm/index.esm.js [1305B] react/README.md [23414B] index.d.ts [3317B] esm/index.esm.js [1403B] 
I0926 00:52:27.141677    5993 installsdk.go:55] Package @dataconnect/generated is already installed. Skipping installation
I0926 00:52:27.142338    5993 load.go:109] Finished reloading
I0926 00:52:27.171698    5993 dev.go:102] Listening on address (HTTP + gRPC): 127.0.0.1:50001
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0926 00:52:26.974234    5993 codegen.go:82] [connector \"example\" javascriptSdk] Generating sources into \"/home/user/studio/src/dataconnect-generated\"\nI0926 00:52:27.106097    5993 collector.go:107] schema extensions wrote into \"/home/user/studio/dataconnect/.dataconnect/schema\"\nGenerated sources: main/mutation.gql [20515B] main/implicit.gql [3592B] main/input.gql [68752B] main/relation.gql [33941B] prelude.gql [73251B] main/query.gql [7604B] \nI0926 00:52:27.126727    5993 collector.go:107] connector \"example\" javascriptSdk wrote into \"/home/user/studio/src/dataconnect-generated\"\nGenerated sources: esm/package.json [17B] index.cjs.js [1629B] react/index.cjs.js [1379B] package.json [863B] README.md [17362B] react/index.d.ts [1699B] react/package.json [460B] react/esm/package.json [17B] react/esm/index.esm.js [1305B] react/README.md [23414B] index.d.ts [3317B] esm/index.esm.js [1403B] \nI0926 00:52:27.141677    5993 installsdk.go:55] Package @dataconnect/generated is already installed. Skipping installation\nI0926 00:52:27.142338    5993 load.go:109] Finished reloading\nI0926 00:52:27.171698    5993 dev.go:102] Listening on address (HTTP + gRPC): 127.0.0.1:50001\n"}}
[debug] [2025-09-26T00:52:28.524Z] <<< [apiv2][status] GET https://firebase.googleapis.com/v1beta1/projects 200
[debug] [2025-09-26T00:52:28.531Z] <<< [apiv2][body] GET https://firebase.googleapis.com/v1beta1/projects [omitted]
[debug] [2025-09-26T00:52:28.694Z] I0926 00:52:28.637807    5993 engine.go:169] [/emulator/studio e9e9] ExecuteGraphqlRead IntrospectionQuery: succeeded. 
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0926 00:52:28.637807    5993 engine.go:169] [/emulator/studio e9e9] ExecuteGraphqlRead IntrospectionQuery: succeeded. \n"}}
[debug] [2025-09-26T00:53:37.154Z] I0926 00:53:37.152871    5993 load.go:31] Reloading schema and connectors...
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0926 00:53:37.152871    5993 load.go:31] Reloading schema and connectors...\n"}}
[debug] [2025-09-26T00:53:37.362Z] W0926 00:53:37.361755    5993 control.go:68] [/emulator/studio e9e9] connector "example" has errors: example/queries.gql:1:21: Unknown argument "rules" on directive "@auth".
example/queries.gql:9:56: Unknown argument "rules" on directive "@auth".
example/queries.gql:17:52: Unknown argument "rules" on directive "@auth".

sources: example/queries.gql [457B] 
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"W0926 00:53:37.361755    5993 control.go:68] [/emulator/studio e9e9] connector \"example\" has errors: example/queries.gql:1:21: Unknown argument \"rules\" on directive \"@auth\".\nexample/queries.gql:9:56: Unknown argument \"rules\" on directive \"@auth\".\nexample/queries.gql:17:52: Unknown argument \"rules\" on directive \"@auth\".\n\nsources: example/queries.gql [457B] \n"}}
[debug] [2025-09-26T00:53:37.384Z] I0926 00:53:37.377501    5993 control.go:94] [/emulator/studio e9e9] UpdateResources(): done
Connector "example": sources: example/queries.gql [457B] 
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0926 00:53:37.377501    5993 control.go:94] [/emulator/studio e9e9] UpdateResources(): done\nConnector \"example\": sources: example/queries.gql [457B] \n"}}
[debug] [2025-09-26T00:53:37.415Z] E0926 00:53:37.413330    5993 load.go:73] Could not load sources: 
Errors: example/queries.gql:1:21: Unknown argument "rules" on directive "@auth".
example/queries.gql:9:56: Unknown argument "rules" on directive "@auth".
example/queries.gql:17:52: Unknown argument "rules" on directive "@auth".
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0926 00:53:37.413330    5993 load.go:73] Could not load sources: \nErrors: example/queries.gql:1:21: Unknown argument \"rules\" on directive \"@auth\".\nexample/queries.gql:9:56: Unknown argument \"rules\" on directive \"@auth\".\nexample/queries.gql:17:52: Unknown argument \"rules\" on directive \"@auth\".\n"}}
[debug] [2025-09-26T00:53:37.437Z] I0926 00:53:37.435746    5993 collector.go:107] schema extensions wrote into "/home/user/studio/dataconnect/.dataconnect/schema"
Generated sources: prelude.gql [73251B] main/mutation.gql [20515B] main/implicit.gql [3592B] main/input.gql [68752B] main/relation.gql [33941B] main/query.gql [7604B] 
I0926 00:53:37.435796    5993 load.go:74] Finished reloading
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0926 00:53:37.435746    5993 collector.go:107] schema extensions wrote into \"/home/user/studio/dataconnect/.dataconnect/schema\"\nGenerated sources: prelude.gql [73251B] main/mutation.gql [20515B] main/implicit.gql [3592B] main/input.gql [68752B] main/relation.gql [33941B] main/query.gql [7604B] \nI0926 00:53:37.435796    5993 load.go:74] Finished reloading\n"}}
[info] [Firebase Plugin] Stopping Data Connect toolkit
[warn] ⚠  Data Connect Emulator has exited upon receiving signal: SIGINT 
[debug] [2025-09-26T00:55:05.991Z] [Firebase Plugin] Activating Firebase extension.
[debug] [2025-09-26T00:55:14.245Z] [Firebase Plugin] requireAuthWrapper
[debug] [2025-09-26T00:55:14.248Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform","https://www.googleapis.com/auth/userinfo.email"]
[debug] [2025-09-26T00:55:14.249Z] > authorizing via signed-in user (innovativeenterprisesoman@gmail.com)
[debug] [2025-09-26T00:55:14.255Z] No OAuth tokens found
[debug] [2025-09-26T00:55:14.256Z] No OAuth tokens found
[debug] [2025-09-26T00:55:14.256Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform","https://www.googleapis.com/auth/userinfo.email"]
[debug] [2025-09-26T00:55:14.257Z] > authorizing via signed-in user (innovativeenterprisesoman@gmail.com)
[debug] [2025-09-26T00:55:14.258Z] [Firebase Plugin] User found:  innovativeenterprisesoman@gmail.com
[info] [Firebase Plugin] (Core:Project) New user detected, fetching projects
[debug] [2025-09-26T00:55:14.260Z] No OAuth tokens found
[debug] [2025-09-26T00:55:14.263Z] >>> [apiv2][query] GET https://firebase.googleapis.com/v1beta1/projects pageSize=1000
[debug] [2025-09-26T00:55:14.328Z] [Firebase Plugin] VSCode notification server listening on port 40001
[debug] [2025-09-26T00:55:18.849Z] <<< [apiv2][status] GET https://firebase.googleapis.com/v1beta1/projects 200
[debug] [2025-09-26T00:55:18.850Z] <<< [apiv2][body] GET https://firebase.googleapis.com/v1beta1/projects [omitted]
[info] [Firebase Plugin] Starting Data Connect toolkit (version 2.13.0) on port 50001
[debug] [2025-09-26T00:55:34.257Z] W0926 00:55:34.254283    6554 control.go:68] [/emulator/studio 2e81] connector "example" has errors: example/queries.gql:1:21: Unknown argument "rules" on directive "@auth".
example/queries.gql:9:56: Unknown argument "rules" on directive "@auth".
example/queries.gql:17:52: Unknown argument "rules" on directive "@auth".

sources: example/queries.gql [457B] 
I0926 00:55:34.254422    6554 control.go:94] [/emulator/studio 2e81] UpdateResources(): done
Schema: sources: schema/schema.gql [1207B] 
Connector "example": sources: example/queries.gql [457B] 
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"W0926 00:55:34.254283    6554 control.go:68] [/emulator/studio 2e81] connector \"example\" has errors: example/queries.gql:1:21: Unknown argument \"rules\" on directive \"@auth\".\nexample/queries.gql:9:56: Unknown argument \"rules\" on directive \"@auth\".\nexample/queries.gql:17:52: Unknown argument \"rules\" on directive \"@auth\".\n\nsources: example/queries.gql [457B] \nI0926 00:55:34.254422    6554 control.go:94] [/emulator/studio 2e81] UpdateResources(): done\nSchema: sources: schema/schema.gql [1207B] \nConnector \"example\": sources: example/queries.gql [457B] \n"}}
[debug] [2025-09-26T00:55:34.259Z] Ignoring unsupported arg: auto_download {"metadata":{"emulator":{"name":"dataconnect"},"message":"Ignoring unsupported arg: auto_download"}}
[debug] [2025-09-26T00:55:34.260Z] Ignoring unsupported arg: port {"metadata":{"emulator":{"name":"dataconnect"},"message":"Ignoring unsupported arg: port"}}
[debug] [2025-09-26T00:55:34.261Z] Starting Data Connect Emulator with command {"binary":"/home/user/.cache/firebase/emulators/dataconnect-emulator-2.13.0","args":["--logtostderr","-v=2","dev","--listen=localhost:50001","--config_dir=/home/user/studio/dataconnect","--enable_output_schema_extensions=true","--enable_output_generated_sdk=true"],"optionalArgs":["listen","config_dir","enable_output_schema_extensions","enable_output_generated_sdk"],"joinArgs":true,"shell":false,"port":9399} {"metadata":{"emulator":{"name":"dataconnect"},"message":"Starting Data Connect Emulator with command {\"binary\":\"/home/user/.cache/firebase/emulators/dataconnect-emulator-2.13.0\",\"args\":[\"--logtostderr\",\"-v=2\",\"dev\",\"--listen=localhost:50001\",\"--config_dir=/home/user/studio/dataconnect\",\"--enable_output_schema_extensions=true\",\"--enable_output_generated_sdk=true\"],\"optionalArgs\":[\"listen\",\"config_dir\",\"enable_output_schema_extensions\",\"enable_output_generated_sdk\"],\"joinArgs\":true,\"shell\":false,\"port\":9399}"}}
[info] i  dataconnect: Data Connect Emulator logging to dataconnect-debug.log {"metadata":{"emulator":{"name":"dataconnect"},"message":"Data Connect Emulator logging to dataconnect-debug.log"}}
[debug] [2025-09-26T00:55:34.408Z] I0926 00:55:34.394929    6559 load.go:31] Reloading schema and connectors...
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0926 00:55:34.394929    6559 load.go:31] Reloading schema and connectors...\n"}}
[debug] [2025-09-26T00:55:34.814Z] W0926 00:55:34.812552    6559 control.go:68] [/emulator/studio d9b2] connector "example" has errors: example/queries.gql:1:21: Unknown argument "rules" on directive "@auth".
example/queries.gql:9:56: Unknown argument "rules" on directive "@auth".
example/queries.gql:17:52: Unknown argument "rules" on directive "@auth".

sources: example/queries.gql [457B] 
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"W0926 00:55:34.812552    6559 control.go:68] [/emulator/studio d9b2] connector \"example\" has errors: example/queries.gql:1:21: Unknown argument \"rules\" on directive \"@auth\".\nexample/queries.gql:9:56: Unknown argument \"rules\" on directive \"@auth\".\nexample/queries.gql:17:52: Unknown argument \"rules\" on directive \"@auth\".\n\nsources: example/queries.gql [457B] \n"}}
[debug] [2025-09-26T00:55:34.829Z] I0926 00:55:34.815606    6559 control.go:94] [/emulator/studio d9b2] UpdateResources(): done
Schema: sources: schema/schema.gql [1207B] 
Connector "example": sources: example/queries.gql [457B] 
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0926 00:55:34.815606    6559 control.go:94] [/emulator/studio d9b2] UpdateResources(): done\nSchema: sources: schema/schema.gql [1207B] \nConnector \"example\": sources: example/queries.gql [457B] \n"}}
[debug] [2025-09-26T00:55:34.893Z] E0926 00:55:34.852115    6559 load.go:73] Could not load sources: 
Errors: example/queries.gql:1:21: Unknown argument "rules" on directive "@auth".
example/queries.gql:9:56: Unknown argument "rules" on directive "@auth".
example/queries.gql:17:52: Unknown argument "rules" on directive "@auth".
I0926 00:55:34.864302    6559 collector.go:107] schema extensions wrote into "/home/user/studio/dataconnect/.dataconnect/schema"
Generated sources: main/input.gql [68752B] main/relation.gql [33941B] main/query.gql [7604B] main/mutation.gql [20515B] prelude.gql [73251B] main/implicit.gql [3592B] 
I0926 00:55:34.864380    6559 load.go:74] Finished reloading
I0926 00:55:34.888239    6559 dev.go:102] Listening on address (HTTP + gRPC): 127.0.0.1:50001
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0926 00:55:34.852115    6559 load.go:73] Could not load sources: \nErrors: example/queries.gql:1:21: Unknown argument \"rules\" on directive \"@auth\".\nexample/queries.gql:9:56: Unknown argument \"rules\" on directive \"@auth\".\nexample/queries.gql:17:52: Unknown argument \"rules\" on directive \"@auth\".\nI0926 00:55:34.864302    6559 collector.go:107] schema extensions wrote into \"/home/user/studio/dataconnect/.dataconnect/schema\"\nGenerated sources: main/input.gql [68752B] main/relation.gql [33941B] main/query.gql [7604B] main/mutation.gql [20515B] prelude.gql [73251B] main/implicit.gql [3592B] \nI0926 00:55:34.864380    6559 load.go:74] Finished reloading\nI0926 00:55:34.888239    6559 dev.go:102] Listening on address (HTTP + gRPC): 127.0.0.1:50001\n"}}
[debug] [2025-09-26T00:55:35.209Z] I0926 00:55:35.207611    6559 engine.go:169] [/emulator/studio d9b2] ExecuteGraphqlRead IntrospectionQuery: succeeded. 
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0926 00:55:35.207611    6559 engine.go:169] [/emulator/studio d9b2] ExecuteGraphqlRead IntrospectionQuery: succeeded. \n"}}
[debug] [2025-09-26T01:02:04.214Z] [Firebase Plugin] Value of process.env.MONOSPACE_ENV: true
[debug] [2025-09-26T01:02:04.216Z] [Firebase Plugin] requireAuthWrapper
[debug] [2025-09-26T01:02:04.217Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform","https://www.googleapis.com/auth/userinfo.email"]
[debug] [2025-09-26T01:02:04.218Z] > authorizing via signed-in user (innovativeenterprisesoman@gmail.com)
[debug] [2025-09-26T01:02:04.219Z] [Firebase Plugin] Value of process.env.MONOSPACE_ENV: true
[debug] [2025-09-26T01:02:04.220Z] [Firebase Plugin] requireAuthWrapper
[debug] [2025-09-26T01:02:04.221Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform","https://www.googleapis.com/auth/userinfo.email"]
[debug] [2025-09-26T01:02:04.221Z] > authorizing via signed-in user (innovativeenterprisesoman@gmail.com)
[debug] [2025-09-26T01:02:04.223Z] Emulator locator file path: /tmp/hub-innovative-enterprises.json
[debug] [2025-09-26T01:02:04.224Z] No OAuth tokens found
[debug] [2025-09-26T01:02:04.225Z] No OAuth tokens found
[debug] [2025-09-26T01:02:04.225Z] [Firebase Plugin] User found:  innovativeenterprisesoman@gmail.com
[debug] [2025-09-26T01:02:04.226Z] [Firebase Plugin] User found:  innovativeenterprisesoman@gmail.com
[info] [Firebase Plugin] (Core:Project) New user detected, fetching projects
[debug] [2025-09-26T01:02:04.228Z] No OAuth tokens found
[info] [Firebase Plugin] (Core:Project) New user detected, fetching projects
[debug] [2025-09-26T01:02:04.229Z] No OAuth tokens found
[debug] [2025-09-26T01:02:04.282Z] >>> [apiv2][query] GET https://firebase.googleapis.com/v1beta1/projects pageSize=1000
[debug] [2025-09-26T01:02:04.286Z] >>> [apiv2][query] GET https://firebase.googleapis.com/v1beta1/projects pageSize=1000
[debug] [2025-09-26T01:02:04.739Z] <<< [apiv2][status] GET https://firebase.googleapis.com/v1beta1/projects 200
[debug] [2025-09-26T01:02:04.740Z] <<< [apiv2][body] GET https://firebase.googleapis.com/v1beta1/projects [omitted]
[debug] [2025-09-26T01:02:04.797Z] <<< [apiv2][status] GET https://firebase.googleapis.com/v1beta1/projects 200
[debug] [2025-09-26T01:02:04.798Z] <<< [apiv2][body] GET https://firebase.googleapis.com/v1beta1/projects [omitted]
[debug] [2025-09-26T01:03:44.899Z] I0926 01:03:44.898402    6559 load.go:31] Reloading schema and connectors...
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0926 01:03:44.898402    6559 load.go:31] Reloading schema and connectors...\n"}}
[debug] [2025-09-26T01:03:45.010Z] W0926 01:03:45.010282    6559 control.go:68] [/emulator/studio d9b2] connector "example" has errors: example/queries.gql:1:21: Unknown argument "allow" on directive "@auth".
example/queries.gql:9:56: Unknown argument "allow" on directive "@auth".
example/queries.gql:23:52: Unknown argument "allow" on directive "@auth".

sources: example/queries.gql [455B] 
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"W0926 01:03:45.010282    6559 control.go:68] [/emulator/studio d9b2] connector \"example\" has errors: example/queries.gql:1:21: Unknown argument \"allow\" on directive \"@auth\".\nexample/queries.gql:9:56: Unknown argument \"allow\" on directive \"@auth\".\nexample/queries.gql:23:52: Unknown argument \"allow\" on directive \"@auth\".\n\nsources: example/queries.gql [455B] \n"}}
[debug] [2025-09-26T01:03:45.015Z] I0926 01:03:45.012597    6559 control.go:94] [/emulator/studio d9b2] UpdateResources(): done
Connector "example": sources: example/queries.gql [455B] 
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0926 01:03:45.012597    6559 control.go:94] [/emulator/studio d9b2] UpdateResources(): done\nConnector \"example\": sources: example/queries.gql [455B] \n"}}
[debug] [2025-09-26T01:03:45.028Z] E0926 01:03:45.026251    6559 load.go:73] Could not load sources: 
Errors: example/queries.gql:1:21: Unknown argument "allow" on directive "@auth".
example/queries.gql:9:56: Unknown argument "allow" on directive "@auth".
example/queries.gql:23:52: Unknown argument "allow" on directive "@auth".
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0926 01:03:45.026251    6559 load.go:73] Could not load sources: \nErrors: example/queries.gql:1:21: Unknown argument \"allow\" on directive \"@auth\".\nexample/queries.gql:9:56: Unknown argument \"allow\" on directive \"@auth\".\nexample/queries.gql:23:52: Unknown argument \"allow\" on directive \"@auth\".\n"}}
[debug] [2025-09-26T01:03:45.038Z] I0926 01:03:45.037503    6559 collector.go:107] schema extensions wrote into "/home/user/studio/dataconnect/.dataconnect/schema"
Generated sources: prelude.gql [73251B] main/implicit.gql [3592B] main/input.gql [68752B] main/relation.gql [33941B] main/query.gql [7604B] main/mutation.gql [20515B] 
I0926 01:03:45.037556    6559 load.go:74] Finished reloading
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0926 01:03:45.037503    6559 collector.go:107] schema extensions wrote into \"/home/user/studio/dataconnect/.dataconnect/schema\"\nGenerated sources: prelude.gql [73251B] main/implicit.gql [3592B] main/input.gql [68752B] main/relation.gql [33941B] main/query.gql [7604B] main/mutation.gql [20515B] \nI0926 01:03:45.037556    6559 load.go:74] Finished reloading\n"}}
[info] ✔  Wrote .gemini/extensions/firebase/gemini-extension.json 
[info] ✔  Wrote .gemini/extensions/firebase/contexts/FIREBASE-BASE.md 
[info] ✔  Wrote .gemini/extensions/firebase/FIREBASE.md 
[debug] [2025-09-26T01:07:45.080Z] [Firebase Plugin] Received emulators started notification. Running detection.
[debug] [2025-09-26T01:07:45.083Z] Emulator locator file path: /tmp/hub-innovative-enterprises.json
[debug] [2025-09-26T01:07:45.086Z] Found emulator locator with different version: {"version":"14.17.0","origins":["http://127.0.0.1:4400"]}, CLI_VERSION: 14.16.0
[debug] [2025-09-26T01:07:45.088Z] >>> [apiv2][query] GET http://127.0.0.1:4400/emulators [none]
[debug] [2025-09-26T01:07:45.610Z] <<< [apiv2][status] GET http://127.0.0.1:4400/emulators 200
[debug] [2025-09-26T01:07:45.613Z] <<< [apiv2][body] GET http://127.0.0.1:4400/emulators {"hub":{"listen":[{"address":"127.0.0.1","family":"IPv4","port":4400}],"name":"hub","host":"127.0.0.1","port":4400},"ui":{"listen":[{"address":"127.0.0.1","family":"IPv4","port":4000}],"name":"ui","host":"127.0.0.1","port":4000},"logging":{"listen":[{"address":"127.0.0.1","family":"IPv4","port":4500}],"name":"logging","host":"127.0.0.1","port":4500},"functions":{"name":"functions","host":"127.0.0.1","port":5001},"firestore":{"listen":[{"address":"127.0.0.1","family":"IPv4","port":8080}],"name":"firestore","host":"127.0.0.1","port":8080,"pid":9362,"reservedPorts":[9150],"webSocketHost":"127.0.0.1","webSocketPort":9150},"hosting":{"listen":[{"address":"127.0.0.1","family":"IPv4","port":5000}],"name":"hosting","host":"127.0.0.1","port":5000},"eventarc":{"name":"eventarc","host":"127.0.0.1","port":9299},"dataconnect":{"listen":[{"address":"127.0.0.1","family":"IPv4","port":9399}],"name":"dataconnect","host":"127.0.0.1","port":9399,"pid":9415,"timeout":10000},"tasks":{"name":"tasks","host":"127.0.0.1","port":9499}}
[debug] [2025-09-26T01:07:46.077Z] I0926 01:07:46.077261    6559 engine.go:169] [/emulator/studio d9b2] ExecuteGraphqlRead IntrospectionQuery: succeeded. 
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0926 01:07:46.077261    6559 engine.go:169] [/emulator/studio d9b2] ExecuteGraphqlRead IntrospectionQuery: succeeded. \n"}}
[debug] [2025-09-26T02:42:06.538Z] [Firebase Plugin] Activating Firebase extension.
[debug] [2025-09-26T02:42:23.312Z] [Firebase Plugin] requireAuthWrapper
[debug] [2025-09-26T02:42:23.315Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform","https://www.googleapis.com/auth/userinfo.email"]
[debug] [2025-09-26T02:42:23.317Z] > authorizing via signed-in user (innovativeenterprisesoman@gmail.com)
[debug] [2025-09-26T02:42:23.327Z] Checked if tokens are valid: false, expires at: 1758854070061
[debug] [2025-09-26T02:42:23.329Z] Checked if tokens are valid: false, expires at: 1758854070061
[debug] [2025-09-26T02:42:23.331Z] > refreshing access token with scopes: []
[debug] [2025-09-26T02:42:23.335Z] >>> [apiv2][query] POST https://www.googleapis.com/oauth2/v3/token [none]
[debug] [2025-09-26T02:42:23.337Z] >>> [apiv2][body] POST https://www.googleapis.com/oauth2/v3/token [omitted]
[debug] [2025-09-26T02:42:23.436Z] [Firebase Plugin] VSCode notification server listening on port 40001
[debug] [2025-09-26T02:42:27.746Z] <<< [apiv2][status] POST https://www.googleapis.com/oauth2/v3/token 200
[debug] [2025-09-26T02:42:27.752Z] <<< [apiv2][body] POST https://www.googleapis.com/oauth2/v3/token [omitted]
[debug] [2025-09-26T02:42:27.790Z] [Firebase Plugin] User found:  innovativeenterprisesoman@gmail.com
[info] [Firebase Plugin] (Core:Project) New user detected, fetching projects
[debug] [2025-09-26T02:42:27.797Z] Checked if tokens are valid: true, expires at: 1758858146755
[debug] [2025-09-26T02:42:27.800Z] >>> [apiv2][query] GET https://firebase.googleapis.com/v1beta1/projects pageSize=1000
[debug] [2025-09-26T02:42:28.302Z] [Firebase Plugin] Value of process.env.MONOSPACE_ENV: true
[debug] [2025-09-26T02:42:28.310Z] [Firebase Plugin] requireAuthWrapper
[debug] [2025-09-26T02:42:28.319Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform","https://www.googleapis.com/auth/userinfo.email"]
[debug] [2025-09-26T02:42:28.327Z] > authorizing via signed-in user (innovativeenterprisesoman@gmail.com)
[debug] [2025-09-26T02:42:28.334Z] [Firebase Plugin] Value of process.env.MONOSPACE_ENV: true
[debug] [2025-09-26T02:42:28.339Z] [Firebase Plugin] requireAuthWrapper
[debug] [2025-09-26T02:42:28.346Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform","https://www.googleapis.com/auth/userinfo.email"]
[debug] [2025-09-26T02:42:28.358Z] > authorizing via signed-in user (innovativeenterprisesoman@gmail.com)
[debug] [2025-09-26T02:42:28.391Z] Emulator locator file path: /tmp/hub-demo-no-project.json
[debug] [2025-09-26T02:42:28.421Z] Checked if tokens are valid: true, expires at: 1758858146755
[debug] [2025-09-26T02:42:28.421Z] Checked if tokens are valid: true, expires at: 1758858146755
[debug] [2025-09-26T02:42:28.422Z] [Firebase Plugin] User found:  innovativeenterprisesoman@gmail.com
[debug] [2025-09-26T02:42:28.422Z] [Firebase Plugin] User found:  innovativeenterprisesoman@gmail.com
[info] [Firebase Plugin] (Core:Project) New user detected, fetching projects
[debug] [2025-09-26T02:42:28.423Z] Checked if tokens are valid: true, expires at: 1758858146755
[info] [Firebase Plugin] (Core:Project) New user detected, fetching projects
[debug] [2025-09-26T02:42:28.424Z] Checked if tokens are valid: true, expires at: 1758858146755
[debug] [2025-09-26T02:42:28.425Z] >>> [apiv2][query] GET https://firebase.googleapis.com/v1beta1/projects pageSize=1000
[debug] [2025-09-26T02:42:28.427Z] >>> [apiv2][query] GET https://firebase.googleapis.com/v1beta1/projects pageSize=1000
[info] [Firebase Plugin] Starting Data Connect toolkit (version 2.13.0) on port 50001
[debug] [2025-09-26T02:42:30.250Z] W0926 02:42:30.247971    2582 control.go:68] [/emulator/studio 9387] connector "example" has errors: example/queries.gql:1:21: Unknown argument "allow" on directive "@auth".
example/queries.gql:9:56: Unknown argument "allow" on directive "@auth".
example/queries.gql:23:52: Unknown argument "allow" on directive "@auth".

sources: example/queries.gql [455B] 
I0926 02:42:30.248116    2582 control.go:94] [/emulator/studio 9387] UpdateResources(): done
Schema: sources: schema/schema.gql [1207B] 
Connector "example": sources: example/queries.gql [455B] 
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"W0926 02:42:30.247971    2582 control.go:68] [/emulator/studio 9387] connector \"example\" has errors: example/queries.gql:1:21: Unknown argument \"allow\" on directive \"@auth\".\nexample/queries.gql:9:56: Unknown argument \"allow\" on directive \"@auth\".\nexample/queries.gql:23:52: Unknown argument \"allow\" on directive \"@auth\".\n\nsources: example/queries.gql [455B] \nI0926 02:42:30.248116    2582 control.go:94] [/emulator/studio 9387] UpdateResources(): done\nSchema: sources: schema/schema.gql [1207B] \nConnector \"example\": sources: example/queries.gql [455B] \n"}}
[debug] [2025-09-26T02:42:30.253Z] Ignoring unsupported arg: auto_download {"metadata":{"emulator":{"name":"dataconnect"},"message":"Ignoring unsupported arg: auto_download"}}
[debug] [2025-09-26T02:42:30.253Z] Ignoring unsupported arg: port {"metadata":{"emulator":{"name":"dataconnect"},"message":"Ignoring unsupported arg: port"}}
[debug] [2025-09-26T02:42:30.254Z] Starting Data Connect Emulator with command {"binary":"/home/user/.cache/firebase/emulators/dataconnect-emulator-2.13.0","args":["--logtostderr","-v=2","dev","--listen=localhost:50001","--config_dir=/home/user/studio/dataconnect","--enable_output_schema_extensions=true","--enable_output_generated_sdk=true"],"optionalArgs":["listen","config_dir","enable_output_schema_extensions","enable_output_generated_sdk"],"joinArgs":true,"shell":false,"port":9399} {"metadata":{"emulator":{"name":"dataconnect"},"message":"Starting Data Connect Emulator with command {\"binary\":\"/home/user/.cache/firebase/emulators/dataconnect-emulator-2.13.0\",\"args\":[\"--logtostderr\",\"-v=2\",\"dev\",\"--listen=localhost:50001\",\"--config_dir=/home/user/studio/dataconnect\",\"--enable_output_schema_extensions=true\",\"--enable_output_generated_sdk=true\"],\"optionalArgs\":[\"listen\",\"config_dir\",\"enable_output_schema_extensions\",\"enable_output_generated_sdk\"],\"joinArgs\":true,\"shell\":false,\"port\":9399}"}}
[info] i  dataconnect: Data Connect Emulator logging to dataconnect-debug.log {"metadata":{"emulator":{"name":"dataconnect"},"message":"Data Connect Emulator logging to dataconnect-debug.log"}}
[debug] [2025-09-26T02:42:30.689Z] I0926 02:42:30.326429    2587 load.go:31] Reloading schema and connectors...
W0926 02:42:30.470432    2587 control.go:68] [/emulator/studio 7fc6] connector "example" has errors: example/queries.gql:1:21: Unknown argument "allow" on directive "@auth".
example/queries.gql:9:56: Unknown argument "allow" on directive "@auth".
example/queries.gql:23:52: Unknown argument "allow" on directive "@auth".

sources: example/queries.gql [455B] 
I0926 02:42:30.471509    2587 control.go:94] [/emulator/studio 7fc6] UpdateResources(): done
Schema: sources: schema/schema.gql [1207B] 
Connector "example": sources: example/queries.gql [455B] 
E0926 02:42:30.479020    2587 load.go:73] Could not load sources: 
Errors: example/queries.gql:1:21: Unknown argument "allow" on directive "@auth".
example/queries.gql:9:56: Unknown argument "allow" on directive "@auth".
example/queries.gql:23:52: Unknown argument "allow" on directive "@auth".
I0926 02:42:30.480794    2587 collector.go:107] schema extensions wrote into "/home/user/studio/dataconnect/.dataconnect/schema"
Generated sources: main/relation.gql [33941B] main/query.gql [7604B] main/mutation.gql [20515B] prelude.gql [73251B] main/implicit.gql [3592B] main/input.gql [68752B] 
I0926 02:42:30.480864    2587 load.go:74] Finished reloading
I0926 02:42:30.493794    2587 dev.go:102] Listening on address (HTTP + gRPC): 127.0.0.1:50001
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0926 02:42:30.326429    2587 load.go:31] Reloading schema and connectors...\nW0926 02:42:30.470432    2587 control.go:68] [/emulator/studio 7fc6] connector \"example\" has errors: example/queries.gql:1:21: Unknown argument \"allow\" on directive \"@auth\".\nexample/queries.gql:9:56: Unknown argument \"allow\" on directive \"@auth\".\nexample/queries.gql:23:52: Unknown argument \"allow\" on directive \"@auth\".\n\nsources: example/queries.gql [455B] \nI0926 02:42:30.471509    2587 control.go:94] [/emulator/studio 7fc6] UpdateResources(): done\nSchema: sources: schema/schema.gql [1207B] \nConnector \"example\": sources: example/queries.gql [455B] \nE0926 02:42:30.479020    2587 load.go:73] Could not load sources: \nErrors: example/queries.gql:1:21: Unknown argument \"allow\" on directive \"@auth\".\nexample/queries.gql:9:56: Unknown argument \"allow\" on directive \"@auth\".\nexample/queries.gql:23:52: Unknown argument \"allow\" on directive \"@auth\".\nI0926 02:42:30.480794    2587 collector.go:107] schema extensions wrote into \"/home/user/studio/dataconnect/.dataconnect/schema\"\nGenerated sources: main/relation.gql [33941B] main/query.gql [7604B] main/mutation.gql [20515B] prelude.gql [73251B] main/implicit.gql [3592B] main/input.gql [68752B] \nI0926 02:42:30.480864    2587 load.go:74] Finished reloading\nI0926 02:42:30.493794    2587 dev.go:102] Listening on address (HTTP + gRPC): 127.0.0.1:50001\n"}}
[debug] [2025-09-26T02:42:31.335Z] <<< [apiv2][status] GET https://firebase.googleapis.com/v1beta1/projects 200
[debug] [2025-09-26T02:42:31.343Z] <<< [apiv2][body] GET https://firebase.googleapis.com/v1beta1/projects [omitted]
[debug] [2025-09-26T02:42:31.359Z] I0926 02:42:31.304143    2587 engine.go:169] [/emulator/studio 7fc6] ExecuteGraphqlRead IntrospectionQuery: succeeded. 
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0926 02:42:31.304143    2587 engine.go:169] [/emulator/studio 7fc6] ExecuteGraphqlRead IntrospectionQuery: succeeded. \n"}}
[debug] [2025-09-26T02:42:31.394Z] <<< [apiv2][status] GET https://firebase.googleapis.com/v1beta1/projects 200
[debug] [2025-09-26T02:42:31.394Z] <<< [apiv2][body] GET https://firebase.googleapis.com/v1beta1/projects [omitted]
[debug] [2025-09-26T02:42:31.397Z] <<< [apiv2][status] GET https://firebase.googleapis.com/v1beta1/projects 200
[debug] [2025-09-26T02:42:31.398Z] <<< [apiv2][body] GET https://firebase.googleapis.com/v1beta1/projects [omitted]
[debug] [2025-09-26T16:49:13.352Z] [Firebase Plugin] Activating Firebase extension.
[debug] [2025-09-26T16:50:00.429Z] [Firebase Plugin] requireAuthWrapper
[debug] [2025-09-26T16:50:00.433Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform","https://www.googleapis.com/auth/userinfo.email"]
[debug] [2025-09-26T16:50:00.434Z] > authorizing via signed-in user (innovativeenterprisesoman@gmail.com)
[debug] [2025-09-26T16:50:00.441Z] Checked if tokens are valid: true, expires at: 1758908992460
[debug] [2025-09-26T16:50:00.442Z] Checked if tokens are valid: true, expires at: 1758908992460
[debug] [2025-09-26T16:50:00.443Z] [Firebase Plugin] User found:  innovativeenterprisesoman@gmail.com
[info] [Firebase Plugin] (Core:Project) New user detected, fetching projects
[debug] [2025-09-26T16:50:00.446Z] Checked if tokens are valid: true, expires at: 1758908992460
[debug] [2025-09-26T16:50:00.450Z] >>> [apiv2][query] GET https://firebase.googleapis.com/v1beta1/projects pageSize=1000
[debug] [2025-09-26T16:50:00.570Z] [Firebase Plugin] VSCode notification server listening on port 40001
[debug] [2025-09-26T16:50:01.548Z] <<< [apiv2][status] GET https://firebase.googleapis.com/v1beta1/projects 200
[debug] [2025-09-26T16:50:01.549Z] <<< [apiv2][body] GET https://firebase.googleapis.com/v1beta1/projects [omitted]
[info] [Firebase Plugin] Starting Data Connect toolkit (version 2.13.0) on port 50001
[debug] [2025-09-26T16:50:05.099Z] W0926 16:50:05.088267   14935 control.go:68] [/emulator/studio dce0] connector "example" has errors: example/queries.gql:1:21: Unknown argument "allow" on directive "@auth".
example/queries.gql:9:56: Unknown argument "allow" on directive "@auth".
example/queries.gql:23:52: Unknown argument "allow" on directive "@auth".

sources: example/queries.gql [455B] 
I0926 16:50:05.091740   14935 control.go:94] [/emulator/studio dce0] UpdateResources(): done
Schema: sources: schema/schema.gql [1207B] 
Connector "example": sources: example/queries.gql [455B] 
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"W0926 16:50:05.088267   14935 control.go:68] [/emulator/studio dce0] connector \"example\" has errors: example/queries.gql:1:21: Unknown argument \"allow\" on directive \"@auth\".\nexample/queries.gql:9:56: Unknown argument \"allow\" on directive \"@auth\".\nexample/queries.gql:23:52: Unknown argument \"allow\" on directive \"@auth\".\n\nsources: example/queries.gql [455B] \nI0926 16:50:05.091740   14935 control.go:94] [/emulator/studio dce0] UpdateResources(): done\nSchema: sources: schema/schema.gql [1207B] \nConnector \"example\": sources: example/queries.gql [455B] \n"}}
[debug] [2025-09-26T16:50:05.108Z] Ignoring unsupported arg: auto_download {"metadata":{"emulator":{"name":"dataconnect"},"message":"Ignoring unsupported arg: auto_download"}}
[debug] [2025-09-26T16:50:05.109Z] Ignoring unsupported arg: port {"metadata":{"emulator":{"name":"dataconnect"},"message":"Ignoring unsupported arg: port"}}
[debug] [2025-09-26T16:50:05.111Z] Starting Data Connect Emulator with command {"binary":"/home/user/.cache/firebase/emulators/dataconnect-emulator-2.13.0","args":["--logtostderr","-v=2","dev","--listen=localhost:50001","--config_dir=/home/user/studio/dataconnect","--enable_output_schema_extensions=true","--enable_output_generated_sdk=true"],"optionalArgs":["listen","config_dir","enable_output_schema_extensions","enable_output_generated_sdk"],"joinArgs":true,"shell":false,"port":9399} {"metadata":{"emulator":{"name":"dataconnect"},"message":"Starting Data Connect Emulator with command {\"binary\":\"/home/user/.cache/firebase/emulators/dataconnect-emulator-2.13.0\",\"args\":[\"--logtostderr\",\"-v=2\",\"dev\",\"--listen=localhost:50001\",\"--config_dir=/home/user/studio/dataconnect\",\"--enable_output_schema_extensions=true\",\"--enable_output_generated_sdk=true\"],\"optionalArgs\":[\"listen\",\"config_dir\",\"enable_output_schema_extensions\",\"enable_output_generated_sdk\"],\"joinArgs\":true,\"shell\":false,\"port\":9399}"}}
[info] i  dataconnect: Data Connect Emulator logging to dataconnect-debug.log {"metadata":{"emulator":{"name":"dataconnect"},"message":"Data Connect Emulator logging to dataconnect-debug.log"}}
[debug] [2025-09-26T16:50:05.565Z] I0926 16:50:05.257501   14941 load.go:31] Reloading schema and connectors...
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0926 16:50:05.257501   14941 load.go:31] Reloading schema and connectors...\n"}}
[debug] [2025-09-26T16:50:05.757Z] W0926 16:50:05.740445   14941 control.go:68] [/emulator/studio bbd6] connector "example" has errors: example/queries.gql:1:21: Unknown argument "allow" on directive "@auth".
example/queries.gql:9:56: Unknown argument "allow" on directive "@auth".
example/queries.gql:23:52: Unknown argument "allow" on directive "@auth".

sources: example/queries.gql [455B] 
I0926 16:50:05.742267   14941 control.go:94] [/emulator/studio bbd6] UpdateResources(): done
Schema: sources: schema/schema.gql [1207B] 
Connector "example": sources: example/queries.gql [455B] 
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"W0926 16:50:05.740445   14941 control.go:68] [/emulator/studio bbd6] connector \"example\" has errors: example/queries.gql:1:21: Unknown argument \"allow\" on directive \"@auth\".\nexample/queries.gql:9:56: Unknown argument \"allow\" on directive \"@auth\".\nexample/queries.gql:23:52: Unknown argument \"allow\" on directive \"@auth\".\n\nsources: example/queries.gql [455B] \nI0926 16:50:05.742267   14941 control.go:94] [/emulator/studio bbd6] UpdateResources(): done\nSchema: sources: schema/schema.gql [1207B] \nConnector \"example\": sources: example/queries.gql [455B] \n"}}
[debug] [2025-09-26T16:50:05.827Z] E0926 16:50:05.760012   14941 load.go:73] Could not load sources: 
Errors: example/queries.gql:1:21: Unknown argument "allow" on directive "@auth".
example/queries.gql:9:56: Unknown argument "allow" on directive "@auth".
example/queries.gql:23:52: Unknown argument "allow" on directive "@auth".
I0926 16:50:05.765800   14941 collector.go:107] schema extensions wrote into "/home/user/studio/dataconnect/.dataconnect/schema"
Generated sources: main/implicit.gql [3592B] main/input.gql [68752B] prelude.gql [73251B] main/relation.gql [33941B] main/query.gql [7604B] main/mutation.gql [20515B] 
I0926 16:50:05.765867   14941 load.go:74] Finished reloading
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0926 16:50:05.760012   14941 load.go:73] Could not load sources: \nErrors: example/queries.gql:1:21: Unknown argument \"allow\" on directive \"@auth\".\nexample/queries.gql:9:56: Unknown argument \"allow\" on directive \"@auth\".\nexample/queries.gql:23:52: Unknown argument \"allow\" on directive \"@auth\".\nI0926 16:50:05.765800   14941 collector.go:107] schema extensions wrote into \"/home/user/studio/dataconnect/.dataconnect/schema\"\nGenerated sources: main/implicit.gql [3592B] main/input.gql [68752B] prelude.gql [73251B] main/relation.gql [33941B] main/query.gql [7604B] main/mutation.gql [20515B] \nI0926 16:50:05.765867   14941 load.go:74] Finished reloading\n"}}
[debug] [2025-09-26T16:50:05.870Z] I0926 16:50:05.863974   14941 dev.go:102] Listening on address (HTTP + gRPC): 127.0.0.1:50001
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0926 16:50:05.863974   14941 dev.go:102] Listening on address (HTTP + gRPC): 127.0.0.1:50001\n"}}
[debug] [2025-09-26T16:50:06.671Z] I0926 16:50:06.669744   14941 engine.go:169] [/emulator/studio bbd6] ExecuteGraphqlRead IntrospectionQuery: succeeded. 
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0926 16:50:06.669744   14941 engine.go:169] [/emulator/studio bbd6] ExecuteGraphqlRead IntrospectionQuery: succeeded. \n"}}
[debug] [2025-09-26T16:53:38.819Z] [Firebase Plugin] Received emulators started notification. Running detection.
[debug] [2025-09-26T16:53:38.825Z] Emulator locator file path: /tmp/hub-innovative-enterprises.json
[debug] [2025-09-26T16:53:38.830Z] Found emulator locator with different version: {"version":"14.17.0","origins":["http://127.0.0.1:4400"]}, CLI_VERSION: 14.16.0
[debug] [2025-09-26T16:53:38.850Z] >>> [apiv2][query] GET http://127.0.0.1:4400/emulators [none]
[debug] [2025-09-26T16:53:39.798Z] <<< [apiv2][status] GET http://127.0.0.1:4400/emulators 200
[debug] [2025-09-26T16:53:39.807Z] <<< [apiv2][body] GET http://127.0.0.1:4400/emulators {"hub":{"listen":[{"address":"127.0.0.1","family":"IPv4","port":4400}],"name":"hub","host":"127.0.0.1","port":4400},"ui":{"listen":[{"address":"127.0.0.1","family":"IPv4","port":4000}],"name":"ui","host":"127.0.0.1","port":4000},"logging":{"listen":[{"address":"127.0.0.1","family":"IPv4","port":4500}],"name":"logging","host":"127.0.0.1","port":4500},"functions":{"name":"functions","host":"127.0.0.1","port":5001},"firestore":{"listen":[{"address":"127.0.0.1","family":"IPv4","port":8080}],"name":"firestore","host":"127.0.0.1","port":8080,"pid":15886,"reservedPorts":[9150],"webSocketHost":"127.0.0.1","webSocketPort":9150},"hosting":{"listen":[{"address":"127.0.0.1","family":"IPv4","port":5000}],"name":"hosting","host":"127.0.0.1","port":5000},"eventarc":{"name":"eventarc","host":"127.0.0.1","port":9299},"dataconnect":{"listen":[{"address":"127.0.0.1","family":"IPv4","port":9399}],"name":"dataconnect","host":"127.0.0.1","port":9399,"pid":16052,"timeout":10000},"tasks":{"name":"tasks","host":"127.0.0.1","port":9499}}
[debug] [2025-09-26T16:53:40.694Z] I0926 16:53:40.618273   14941 engine.go:169] [/emulator/studio bbd6] ExecuteGraphqlRead IntrospectionQuery: succeeded. 
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0926 16:53:40.618273   14941 engine.go:169] [/emulator/studio bbd6] ExecuteGraphqlRead IntrospectionQuery: succeeded. \n"}}
[info] [Firebase Plugin] Stopping Data Connect toolkit
[warn] ⚠  Data Connect Emulator has exited upon receiving signal: SIGINT 
[debug] [2025-09-27T05:51:58.440Z] [Firebase Plugin] Activating Firebase extension.
[debug] [2025-09-27T05:52:29.910Z] [Firebase Plugin] requireAuthWrapper
[debug] [2025-09-27T05:52:29.913Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform","https://www.googleapis.com/auth/userinfo.email"]
[debug] [2025-09-27T05:52:29.915Z] > authorizing via signed-in user (innovativeenterprisesoman@gmail.com)
[debug] [2025-09-27T05:52:29.921Z] Checked if tokens are valid: false, expires at: 1758908992460
[debug] [2025-09-27T05:52:29.922Z] Checked if tokens are valid: false, expires at: 1758908992460
[debug] [2025-09-27T05:52:29.923Z] > refreshing access token with scopes: []
[debug] [2025-09-27T05:52:29.927Z] >>> [apiv2][query] POST https://www.googleapis.com/oauth2/v3/token [none]
[debug] [2025-09-27T05:52:29.928Z] >>> [apiv2][body] POST https://www.googleapis.com/oauth2/v3/token [omitted]
[debug] [2025-09-27T05:52:29.990Z] [Firebase Plugin] VSCode notification server listening on port 40001
[debug] [2025-09-27T05:52:30.121Z] <<< [apiv2][status] POST https://www.googleapis.com/oauth2/v3/token 200
[debug] [2025-09-27T05:52:30.122Z] <<< [apiv2][body] POST https://www.googleapis.com/oauth2/v3/token [omitted]
[debug] [2025-09-27T05:52:30.154Z] [Firebase Plugin] User found:  innovativeenterprisesoman@gmail.com
[info] [Firebase Plugin] (Core:Project) New user detected, fetching projects
[debug] [2025-09-27T05:52:30.157Z] Checked if tokens are valid: true, expires at: 1758955949123
[debug] [2025-09-27T05:52:30.158Z] >>> [apiv2][query] GET https://firebase.googleapis.com/v1beta1/projects pageSize=1000
[debug] [2025-09-27T05:52:30.617Z] <<< [apiv2][status] GET https://firebase.googleapis.com/v1beta1/projects 200
[debug] [2025-09-27T05:52:30.618Z] <<< [apiv2][body] GET https://firebase.googleapis.com/v1beta1/projects [omitted]
[info] [Firebase Plugin] Starting Data Connect toolkit (version 2.13.0) on port 50001
[debug] [2025-09-27T05:52:33.041Z] W0927 05:52:33.037386   14053 control.go:68] [/emulator/studio 83b3] connector "example" has errors: example/queries.gql:1:21: Unknown argument "allow" on directive "@auth".
example/queries.gql:9:56: Unknown argument "allow" on directive "@auth".
example/queries.gql:23:52: Unknown argument "allow" on directive "@auth".

sources: example/queries.gql [455B] 
I0927 05:52:33.037552   14053 control.go:94] [/emulator/studio 83b3] UpdateResources(): done
Schema: sources: schema/schema.gql [1207B] 
Connector "example": sources: example/queries.gql [455B] 
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"W0927 05:52:33.037386   14053 control.go:68] [/emulator/studio 83b3] connector \"example\" has errors: example/queries.gql:1:21: Unknown argument \"allow\" on directive \"@auth\".\nexample/queries.gql:9:56: Unknown argument \"allow\" on directive \"@auth\".\nexample/queries.gql:23:52: Unknown argument \"allow\" on directive \"@auth\".\n\nsources: example/queries.gql [455B] \nI0927 05:52:33.037552   14053 control.go:94] [/emulator/studio 83b3] UpdateResources(): done\nSchema: sources: schema/schema.gql [1207B] \nConnector \"example\": sources: example/queries.gql [455B] \n"}}
[debug] [2025-09-27T05:52:33.044Z] Ignoring unsupported arg: auto_download {"metadata":{"emulator":{"name":"dataconnect"},"message":"Ignoring unsupported arg: auto_download"}}
[debug] [2025-09-27T05:52:33.045Z] Ignoring unsupported arg: port {"metadata":{"emulator":{"name":"dataconnect"},"message":"Ignoring unsupported arg: port"}}
[debug] [2025-09-27T05:52:33.046Z] Starting Data Connect Emulator with command {"binary":"/home/user/.cache/firebase/emulators/dataconnect-emulator-2.13.0","args":["--logtostderr","-v=2","dev","--listen=localhost:50001","--config_dir=/home/user/studio/dataconnect","--enable_output_schema_extensions=true","--enable_output_generated_sdk=true"],"optionalArgs":["listen","config_dir","enable_output_schema_extensions","enable_output_generated_sdk"],"joinArgs":true,"shell":false,"port":9399} {"metadata":{"emulator":{"name":"dataconnect"},"message":"Starting Data Connect Emulator with command {\"binary\":\"/home/user/.cache/firebase/emulators/dataconnect-emulator-2.13.0\",\"args\":[\"--logtostderr\",\"-v=2\",\"dev\",\"--listen=localhost:50001\",\"--config_dir=/home/user/studio/dataconnect\",\"--enable_output_schema_extensions=true\",\"--enable_output_generated_sdk=true\"],\"optionalArgs\":[\"listen\",\"config_dir\",\"enable_output_schema_extensions\",\"enable_output_generated_sdk\"],\"joinArgs\":true,\"shell\":false,\"port\":9399}"}}
[info] i  dataconnect: Data Connect Emulator logging to dataconnect-debug.log {"metadata":{"emulator":{"name":"dataconnect"},"message":"Data Connect Emulator logging to dataconnect-debug.log"}}
[debug] [2025-09-27T05:52:33.133Z] I0927 05:52:33.105959   14058 load.go:31] Reloading schema and connectors...
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0927 05:52:33.105959   14058 load.go:31] Reloading schema and connectors...\n"}}
[debug] [2025-09-27T05:52:33.432Z] W0927 05:52:33.387546   14058 control.go:68] [/emulator/studio 9af8] connector "example" has errors: example/queries.gql:1:21: Unknown argument "allow" on directive "@auth".
example/queries.gql:9:56: Unknown argument "allow" on directive "@auth".
example/queries.gql:23:52: Unknown argument "allow" on directive "@auth".

sources: example/queries.gql [455B] 
I0927 05:52:33.387652   14058 control.go:94] [/emulator/studio 9af8] UpdateResources(): done
Schema: sources: schema/schema.gql [1207B] 
Connector "example": sources: example/queries.gql [455B] 
E0927 05:52:33.391964   14058 load.go:73] Could not load sources: 
Errors: example/queries.gql:1:21: Unknown argument "allow" on directive "@auth".
example/queries.gql:9:56: Unknown argument "allow" on directive "@auth".
example/queries.gql:23:52: Unknown argument "allow" on directive "@auth".
I0927 05:52:33.393481   14058 collector.go:107] schema extensions wrote into "/home/user/studio/dataconnect/.dataconnect/schema"
Generated sources: prelude.gql [73251B] main/implicit.gql [3592B] main/input.gql [68752B] main/relation.gql [33941B] main/query.gql [7604B] main/mutation.gql [20515B] 
I0927 05:52:33.398067   14058 load.go:74] Finished reloading
I0927 05:52:33.412606   14058 dev.go:102] Listening on address (HTTP + gRPC): 127.0.0.1:50001
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"W0927 05:52:33.387546   14058 control.go:68] [/emulator/studio 9af8] connector \"example\" has errors: example/queries.gql:1:21: Unknown argument \"allow\" on directive \"@auth\".\nexample/queries.gql:9:56: Unknown argument \"allow\" on directive \"@auth\".\nexample/queries.gql:23:52: Unknown argument \"allow\" on directive \"@auth\".\n\nsources: example/queries.gql [455B] \nI0927 05:52:33.387652   14058 control.go:94] [/emulator/studio 9af8] UpdateResources(): done\nSchema: sources: schema/schema.gql [1207B] \nConnector \"example\": sources: example/queries.gql [455B] \nE0927 05:52:33.391964   14058 load.go:73] Could not load sources: \nErrors: example/queries.gql:1:21: Unknown argument \"allow\" on directive \"@auth\".\nexample/queries.gql:9:56: Unknown argument \"allow\" on directive \"@auth\".\nexample/queries.gql:23:52: Unknown argument \"allow\" on directive \"@auth\".\nI0927 05:52:33.393481   14058 collector.go:107] schema extensions wrote into \"/home/user/studio/dataconnect/.dataconnect/schema\"\nGenerated sources: prelude.gql [73251B] main/implicit.gql [3592B] main/input.gql [68752B] main/relation.gql [33941B] main/query.gql [7604B] main/mutation.gql [20515B] \nI0927 05:52:33.398067   14058 load.go:74] Finished reloading\nI0927 05:52:33.412606   14058 dev.go:102] Listening on address (HTTP + gRPC): 127.0.0.1:50001\n"}}
[debug] [2025-09-27T05:52:33.576Z] I0927 05:52:33.576196   14058 engine.go:169] [/emulator/studio 9af8] ExecuteGraphqlRead IntrospectionQuery: succeeded. 
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0927 05:52:33.576196   14058 engine.go:169] [/emulator/studio 9af8] ExecuteGraphqlRead IntrospectionQuery: succeeded. \n"}}
[info] [Firebase Plugin] Stopping Data Connect toolkit
[warn] ⚠  Data Connect Emulator has exited upon receiving signal: SIGINT 
[debug] [2025-09-27T06:46:41.261Z] [Firebase Plugin] Activating Firebase extension.
[debug] [2025-09-27T06:47:14.579Z] [Firebase Plugin] requireAuthWrapper
[debug] [2025-09-27T06:47:14.582Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform","https://www.googleapis.com/auth/userinfo.email"]
[debug] [2025-09-27T06:47:14.583Z] > authorizing via signed-in user (innovativeenterprisesoman@gmail.com)
[debug] [2025-09-27T06:47:14.590Z] Checked if tokens are valid: false, expires at: 1758955949123
[debug] [2025-09-27T06:47:14.591Z] Checked if tokens are valid: false, expires at: 1758955949123
[debug] [2025-09-27T06:47:14.593Z] > refreshing access token with scopes: []
[debug] [2025-09-27T06:47:14.596Z] >>> [apiv2][query] POST https://www.googleapis.com/oauth2/v3/token [none]
[debug] [2025-09-27T06:47:14.597Z] >>> [apiv2][body] POST https://www.googleapis.com/oauth2/v3/token [omitted]
[debug] [2025-09-27T06:47:14.662Z] [Firebase Plugin] VSCode notification server listening on port 40001
[debug] [2025-09-27T06:47:14.766Z] <<< [apiv2][status] POST https://www.googleapis.com/oauth2/v3/token 200
[debug] [2025-09-27T06:47:14.767Z] <<< [apiv2][body] POST https://www.googleapis.com/oauth2/v3/token [omitted]
[debug] [2025-09-27T06:47:14.787Z] [Firebase Plugin] User found:  innovativeenterprisesoman@gmail.com
[info] [Firebase Plugin] (Core:Project) New user detected, fetching projects
[debug] [2025-09-27T06:47:14.789Z] Checked if tokens are valid: true, expires at: 1758959233768
[debug] [2025-09-27T06:47:14.790Z] >>> [apiv2][query] GET https://firebase.googleapis.com/v1beta1/projects pageSize=1000
[debug] [2025-09-27T06:47:15.304Z] <<< [apiv2][status] GET https://firebase.googleapis.com/v1beta1/projects 200
[debug] [2025-09-27T06:47:15.305Z] <<< [apiv2][body] GET https://firebase.googleapis.com/v1beta1/projects [omitted]
[info] [Firebase Plugin] Starting Data Connect toolkit (version 2.13.0) on port 50001
[debug] [2025-09-27T06:47:17.440Z] W0927 06:47:17.437424   18661 control.go:68] [/emulator/studio a17e] connector "example" has errors: example/queries.gql:1:21: Unknown argument "allow" on directive "@auth".
example/queries.gql:9:56: Unknown argument "allow" on directive "@auth".
example/queries.gql:23:52: Unknown argument "allow" on directive "@auth".

sources: example/queries.gql [455B] 
I0927 06:47:17.437581   18661 control.go:94] [/emulator/studio a17e] UpdateResources(): done
Schema: sources: schema/schema.gql [1207B] 
Connector "example": sources: example/queries.gql [455B] 
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"W0927 06:47:17.437424   18661 control.go:68] [/emulator/studio a17e] connector \"example\" has errors: example/queries.gql:1:21: Unknown argument \"allow\" on directive \"@auth\".\nexample/queries.gql:9:56: Unknown argument \"allow\" on directive \"@auth\".\nexample/queries.gql:23:52: Unknown argument \"allow\" on directive \"@auth\".\n\nsources: example/queries.gql [455B] \nI0927 06:47:17.437581   18661 control.go:94] [/emulator/studio a17e] UpdateResources(): done\nSchema: sources: schema/schema.gql [1207B] \nConnector \"example\": sources: example/queries.gql [455B] \n"}}
[debug] [2025-09-27T06:47:17.443Z] Ignoring unsupported arg: auto_download {"metadata":{"emulator":{"name":"dataconnect"},"message":"Ignoring unsupported arg: auto_download"}}
[debug] [2025-09-27T06:47:17.443Z] Ignoring unsupported arg: port {"metadata":{"emulator":{"name":"dataconnect"},"message":"Ignoring unsupported arg: port"}}
[debug] [2025-09-27T06:47:17.444Z] Starting Data Connect Emulator with command {"binary":"/home/user/.cache/firebase/emulators/dataconnect-emulator-2.13.0","args":["--logtostderr","-v=2","dev","--listen=localhost:50001","--config_dir=/home/user/studio/dataconnect","--enable_output_schema_extensions=true","--enable_output_generated_sdk=true"],"optionalArgs":["listen","config_dir","enable_output_schema_extensions","enable_output_generated_sdk"],"joinArgs":true,"shell":false,"port":9399} {"metadata":{"emulator":{"name":"dataconnect"},"message":"Starting Data Connect Emulator with command {\"binary\":\"/home/user/.cache/firebase/emulators/dataconnect-emulator-2.13.0\",\"args\":[\"--logtostderr\",\"-v=2\",\"dev\",\"--listen=localhost:50001\",\"--config_dir=/home/user/studio/dataconnect\",\"--enable_output_schema_extensions=true\",\"--enable_output_generated_sdk=true\"],\"optionalArgs\":[\"listen\",\"config_dir\",\"enable_output_schema_extensions\",\"enable_output_generated_sdk\"],\"joinArgs\":true,\"shell\":false,\"port\":9399}"}}
[info] i  dataconnect: Data Connect Emulator logging to dataconnect-debug.log {"metadata":{"emulator":{"name":"dataconnect"},"message":"Data Connect Emulator logging to dataconnect-debug.log"}}
[debug] [2025-09-27T06:47:17.518Z] I0927 06:47:17.509585   18666 load.go:31] Reloading schema and connectors...
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0927 06:47:17.509585   18666 load.go:31] Reloading schema and connectors...\n"}}
[debug] [2025-09-27T06:47:17.690Z] W0927 06:47:17.687453   18666 control.go:68] [/emulator/studio 9279] connector "example" has errors: example/queries.gql:1:21: Unknown argument "allow" on directive "@auth".
example/queries.gql:9:56: Unknown argument "allow" on directive "@auth".
example/queries.gql:23:52: Unknown argument "allow" on directive "@auth".

sources: example/queries.gql [455B] 
I0927 06:47:17.687560   18666 control.go:94] [/emulator/studio 9279] UpdateResources(): done
Schema: sources: schema/schema.gql [1207B] 
Connector "example": sources: example/queries.gql [455B] 
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"W0927 06:47:17.687453   18666 control.go:68] [/emulator/studio 9279] connector \"example\" has errors: example/queries.gql:1:21: Unknown argument \"allow\" on directive \"@auth\".\nexample/queries.gql:9:56: Unknown argument \"allow\" on directive \"@auth\".\nexample/queries.gql:23:52: Unknown argument \"allow\" on directive \"@auth\".\n\nsources: example/queries.gql [455B] \nI0927 06:47:17.687560   18666 control.go:94] [/emulator/studio 9279] UpdateResources(): done\nSchema: sources: schema/schema.gql [1207B] \nConnector \"example\": sources: example/queries.gql [455B] \n"}}
[debug] [2025-09-27T06:47:17.706Z] E0927 06:47:17.702742   18666 load.go:73] Could not load sources: 
Errors: example/queries.gql:1:21: Unknown argument "allow" on directive "@auth".
example/queries.gql:9:56: Unknown argument "allow" on directive "@auth".
example/queries.gql:23:52: Unknown argument "allow" on directive "@auth".
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0927 06:47:17.702742   18666 load.go:73] Could not load sources: \nErrors: example/queries.gql:1:21: Unknown argument \"allow\" on directive \"@auth\".\nexample/queries.gql:9:56: Unknown argument \"allow\" on directive \"@auth\".\nexample/queries.gql:23:52: Unknown argument \"allow\" on directive \"@auth\".\n"}}
[debug] [2025-09-27T06:47:17.708Z] I0927 06:47:17.708091   18666 collector.go:107] schema extensions wrote into "/home/user/studio/dataconnect/.dataconnect/schema"
Generated sources: prelude.gql [73251B] main/implicit.gql [3592B] main/input.gql [68752B] main/relation.gql [33941B] main/query.gql [7604B] main/mutation.gql [20515B] 
I0927 06:47:17.708270   18666 load.go:74] Finished reloading
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0927 06:47:17.708091   18666 collector.go:107] schema extensions wrote into \"/home/user/studio/dataconnect/.dataconnect/schema\"\nGenerated sources: prelude.gql [73251B] main/implicit.gql [3592B] main/input.gql [68752B] main/relation.gql [33941B] main/query.gql [7604B] main/mutation.gql [20515B] \nI0927 06:47:17.708270   18666 load.go:74] Finished reloading\n"}}
[debug] [2025-09-27T06:47:17.743Z] I0927 06:47:17.742391   18666 dev.go:102] Listening on address (HTTP + gRPC): 127.0.0.1:50001
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0927 06:47:17.742391   18666 dev.go:102] Listening on address (HTTP + gRPC): 127.0.0.1:50001\n"}}
[debug] [2025-09-27T06:47:18.944Z] I0927 06:47:18.078290   18666 engine.go:169] [/emulator/studio 9279] ExecuteGraphqlRead IntrospectionQuery: succeeded. 
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0927 06:47:18.078290   18666 engine.go:169] [/emulator/studio 9279] ExecuteGraphqlRead IntrospectionQuery: succeeded. \n"}}
[info] [Firebase Plugin] Stopping Data Connect toolkit
[warn] ⚠  Data Connect Emulator has exited upon receiving signal: SIGINT 
[debug] [2025-09-27T07:04:37.762Z] [Firebase Plugin] Activating Firebase extension.
[debug] [2025-09-27T07:05:36.869Z] [Firebase Plugin] requireAuthWrapper
[debug] [2025-09-27T07:05:36.872Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform","https://www.googleapis.com/auth/userinfo.email"]
[debug] [2025-09-27T07:05:36.874Z] > authorizing via signed-in user (innovativeenterprisesoman@gmail.com)
[debug] [2025-09-27T07:05:36.883Z] Checked if tokens are valid: true, expires at: 1758959233768
[debug] [2025-09-27T07:05:36.885Z] Checked if tokens are valid: true, expires at: 1758959233768
[debug] [2025-09-27T07:05:36.887Z] [Firebase Plugin] User found:  innovativeenterprisesoman@gmail.com
[info] [Firebase Plugin] (Core:Project) New user detected, fetching projects
[debug] [2025-09-27T07:05:36.892Z] Checked if tokens are valid: true, expires at: 1758959233768
[debug] [2025-09-27T07:05:36.895Z] >>> [apiv2][query] GET https://firebase.googleapis.com/v1beta1/projects pageSize=1000
[debug] [2025-09-27T07:05:36.979Z] [Firebase Plugin] VSCode notification server listening on port 40001
[debug] [2025-09-27T07:05:38.518Z] <<< [apiv2][status] GET https://firebase.googleapis.com/v1beta1/projects 200
[debug] [2025-09-27T07:05:38.521Z] <<< [apiv2][body] GET https://firebase.googleapis.com/v1beta1/projects [omitted]
[info] [Firebase Plugin] Starting Data Connect toolkit (version 2.13.0) on port 50001
[debug] [2025-09-27T07:05:46.204Z] W0927 07:05:46.195250   22174 control.go:68] [/emulator/studio 212e] connector "example" has errors: example/queries.gql:1:21: Unknown argument "allow" on directive "@auth".
example/queries.gql:9:56: Unknown argument "allow" on directive "@auth".
example/queries.gql:23:52: Unknown argument "allow" on directive "@auth".

sources: example/queries.gql [455B] 
I0927 07:05:46.195417   22174 control.go:94] [/emulator/studio 212e] UpdateResources(): done
Schema: sources: schema/schema.gql [1207B] 
Connector "example": sources: example/queries.gql [455B] 
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"W0927 07:05:46.195250   22174 control.go:68] [/emulator/studio 212e] connector \"example\" has errors: example/queries.gql:1:21: Unknown argument \"allow\" on directive \"@auth\".\nexample/queries.gql:9:56: Unknown argument \"allow\" on directive \"@auth\".\nexample/queries.gql:23:52: Unknown argument \"allow\" on directive \"@auth\".\n\nsources: example/queries.gql [455B] \nI0927 07:05:46.195417   22174 control.go:94] [/emulator/studio 212e] UpdateResources(): done\nSchema: sources: schema/schema.gql [1207B] \nConnector \"example\": sources: example/queries.gql [455B] \n"}}
[debug] [2025-09-27T07:05:46.208Z] Ignoring unsupported arg: auto_download {"metadata":{"emulator":{"name":"dataconnect"},"message":"Ignoring unsupported arg: auto_download"}}
[debug] [2025-09-27T07:05:46.213Z] Ignoring unsupported arg: port {"metadata":{"emulator":{"name":"dataconnect"},"message":"Ignoring unsupported arg: port"}}
[debug] [2025-09-27T07:05:46.215Z] Starting Data Connect Emulator with command {"binary":"/home/user/.cache/firebase/emulators/dataconnect-emulator-2.13.0","args":["--logtostderr","-v=2","dev","--listen=localhost:50001","--config_dir=/home/user/studio/dataconnect","--enable_output_schema_extensions=true","--enable_output_generated_sdk=true"],"optionalArgs":["listen","config_dir","enable_output_schema_extensions","enable_output_generated_sdk"],"joinArgs":true,"shell":false,"port":9399} {"metadata":{"emulator":{"name":"dataconnect"},"message":"Starting Data Connect Emulator with command {\"binary\":\"/home/user/.cache/firebase/emulators/dataconnect-emulator-2.13.0\",\"args\":[\"--logtostderr\",\"-v=2\",\"dev\",\"--listen=localhost:50001\",\"--config_dir=/home/user/studio/dataconnect\",\"--enable_output_schema_extensions=true\",\"--enable_output_generated_sdk=true\"],\"optionalArgs\":[\"listen\",\"config_dir\",\"enable_output_schema_extensions\",\"enable_output_generated_sdk\"],\"joinArgs\":true,\"shell\":false,\"port\":9399}"}}
[info] i  dataconnect: Data Connect Emulator logging to dataconnect-debug.log {"metadata":{"emulator":{"name":"dataconnect"},"message":"Data Connect Emulator logging to dataconnect-debug.log"}}
[debug] [2025-09-27T07:05:46.432Z] I0927 07:05:46.299305   22180 load.go:31] Reloading schema and connectors...
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0927 07:05:46.299305   22180 load.go:31] Reloading schema and connectors...\n"}}
[debug] [2025-09-27T07:05:46.972Z] W0927 07:05:46.742767   22180 control.go:68] [/emulator/studio 5b42] connector "example" has errors: example/queries.gql:1:21: Unknown argument "allow" on directive "@auth".
example/queries.gql:9:56: Unknown argument "allow" on directive "@auth".
example/queries.gql:23:52: Unknown argument "allow" on directive "@auth".

sources: example/queries.gql [455B] 
I0927 07:05:46.750761   22180 control.go:94] [/emulator/studio 5b42] UpdateResources(): done
Schema: sources: schema/schema.gql [1207B] 
Connector "example": sources: example/queries.gql [455B] 
E0927 07:05:46.801907   22180 load.go:73] Could not load sources: 
Errors: example/queries.gql:1:21: Unknown argument "allow" on directive "@auth".
example/queries.gql:9:56: Unknown argument "allow" on directive "@auth".
example/queries.gql:23:52: Unknown argument "allow" on directive "@auth".
I0927 07:05:46.806542   22180 collector.go:107] schema extensions wrote into "/home/user/studio/dataconnect/.dataconnect/schema"
Generated sources: main/implicit.gql [3592B] main/input.gql [68752B] prelude.gql [73251B] main/relation.gql [33941B] main/query.gql [7604B] main/mutation.gql [20515B] 
I0927 07:05:46.807299   22180 load.go:74] Finished reloading
I0927 07:05:46.875503   22180 dev.go:102] Listening on address (HTTP + gRPC): 127.0.0.1:50001
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"W0927 07:05:46.742767   22180 control.go:68] [/emulator/studio 5b42] connector \"example\" has errors: example/queries.gql:1:21: Unknown argument \"allow\" on directive \"@auth\".\nexample/queries.gql:9:56: Unknown argument \"allow\" on directive \"@auth\".\nexample/queries.gql:23:52: Unknown argument \"allow\" on directive \"@auth\".\n\nsources: example/queries.gql [455B] \nI0927 07:05:46.750761   22180 control.go:94] [/emulator/studio 5b42] UpdateResources(): done\nSchema: sources: schema/schema.gql [1207B] \nConnector \"example\": sources: example/queries.gql [455B] \nE0927 07:05:46.801907   22180 load.go:73] Could not load sources: \nErrors: example/queries.gql:1:21: Unknown argument \"allow\" on directive \"@auth\".\nexample/queries.gql:9:56: Unknown argument \"allow\" on directive \"@auth\".\nexample/queries.gql:23:52: Unknown argument \"allow\" on directive \"@auth\".\nI0927 07:05:46.806542   22180 collector.go:107] schema extensions wrote into \"/home/user/studio/dataconnect/.dataconnect/schema\"\nGenerated sources: main/implicit.gql [3592B] main/input.gql [68752B] prelude.gql [73251B] main/relation.gql [33941B] main/query.gql [7604B] main/mutation.gql [20515B] \nI0927 07:05:46.807299   22180 load.go:74] Finished reloading\nI0927 07:05:46.875503   22180 dev.go:102] Listening on address (HTTP + gRPC): 127.0.0.1:50001\n"}}
[debug] [2025-09-27T07:05:47.993Z] I0927 07:05:47.977986   22180 engine.go:169] [/emulator/studio 5b42] ExecuteGraphqlRead IntrospectionQuery: succeeded. 
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0927 07:05:47.977986   22180 engine.go:169] [/emulator/studio 5b42] ExecuteGraphqlRead IntrospectionQuery: succeeded. \n"}}
